{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "collapsed_sections": [
        "7wVOziG7ms0f",
        "xSLArzOom2u7",
        "76E5eMfiq2jc",
        "QZ_D78L_pyuN",
        "2-vFZhGsIEQA"
      ],
      "authorship_tag": "ABX9TyNebHscEujbnFy3eijWrmhI",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/etemadism/Courses/blob/main/simple_alignment.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "**Note:**\n",
        "\n",
        "This is just a simple code to perform pairwise alignment using BLAST+ software package. It is intended for use in a Bioinformatics class during the pairwise alignment session.\n",
        "It will be updated each session.\n",
        "\n",
        "**A. Etemadi**\n",
        "\n",
        "Tehran University of Medical Sciences"
      ],
      "metadata": {
        "id": "qnGx9fMW-rA7"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Step 1: Install BLAST+ software package"
      ],
      "metadata": {
        "id": "7wVOziG7ms0f"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "\n",
        "# Download the BLAST+ software package\n",
        "os.system('wget https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ncbi-blast-2.15.0+-x64-linux.tar.gz')\n",
        "\n",
        "# Extract the downloaded tar.gz file\n",
        "os.system('tar xvzf /content/ncbi-blast-2.15.0+-x64-linux.tar.gz')\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "3QG68Q1HA6vY",
        "outputId": "fe0c2710-d60d-4a47-b47c-a03acadcef82"
      },
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0"
            ]
          },
          "metadata": {},
          "execution_count": 1
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Test if BLAST+ is installed by checking the help message of the blastn executable\n",
        "os.system('/content/ncbi-blast-2.15.0+/bin/blastn -h')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "SGHFrXUUPOPf",
        "outputId": "5dc1a2b2-f0fe-41a4-9909-ded0f37e06e9"
      },
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0"
            ]
          },
          "metadata": {},
          "execution_count": 2
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Step 2: Install Entrez Direct (EDirect)"
      ],
      "metadata": {
        "id": "xSLArzOom2u7"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "Entrez Direct (EDirect) provides access to the NCBI's suite of interconnected databases (publication, sequence, structure, gene, variation, expression, etc.) from a Unix terminal window. Search terms are entered as command-line arguments. Individual operations are connected with Unix pipes to construct multi-step queries. Selected records can then be retrieved in a variety of formats."
      ],
      "metadata": {
        "id": "5biB382RiIhb"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "###You can find more info here: https://www.ncbi.nlm.nih.gov/books/NBK179288/\n",
        "import os\n",
        "\n",
        "# Install Entrez Direct\n",
        "os.system('sh -c \"$(curl -fsSL https://ftp.ncbi.nlm.nih.gov/entrez/entrezdirect/install-edirect.sh)\"')\n",
        "\n",
        "# Set PATH for the current session\n",
        "os.environ['PATH'] = f\"{os.environ['HOME']}/edirect:{os.environ['PATH']}\"\n"
      ],
      "metadata": {
        "id": "r287JwROdBEa"
      },
      "execution_count": 3,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Pairwise alignment\n"
      ],
      "metadata": {
        "id": "VyA-dlFjnBOl"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Perform protein pairwise alignment\n"
      ],
      "metadata": {
        "id": "76E5eMfiq2jc"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!efetch -db protein -format fasta -id NP_000509"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "DOuQugANplzs",
        "outputId": "09d8bf37-8956-4ca9-9a69-d2dc7723f0fa"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            ">NP_000509.1 hemoglobin subunit beta [Homo sapiens]\n",
            "MVHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHGKKVLG\n",
            "AFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEFTPPVQAAYQKVVAGVAN\n",
            "ALAHKYH\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "##This script fetches sequences from the NCBI database (here protein db) using their IDs\n",
        "## and saves them as FASTA files.\n",
        "!efetch -db protein -format fasta -id NP_000509 > NP_000509.fasta\n",
        "!efetch -db protein -format fasta -id NP_001188320 > NP_001188320.fasta\n"
      ],
      "metadata": {
        "id": "2v2NPs61fWBr"
      },
      "execution_count": 4,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "### This will provide information about the usage, options, and parameters available for the blastp command.\n",
        "!/content/ncbi-blast-2.15.0+/bin/blastp -h"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CcxAKJnEoBhn",
        "outputId": "3e1a452d-8f92-4deb-a3c8-2b7a14146257"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "USAGE\n",
            "  blastp [-h] [-help] [-import_search_strategy filename]\n",
            "    [-export_search_strategy filename] [-task task_name] [-db database_name]\n",
            "    [-dbsize num_letters] [-gilist filename] [-seqidlist filename]\n",
            "    [-negative_gilist filename] [-negative_seqidlist filename]\n",
            "    [-taxids taxids] [-negative_taxids taxids] [-taxidlist filename]\n",
            "    [-negative_taxidlist filename] [-no_taxid_expansion] [-ipglist filename]\n",
            "    [-negative_ipglist filename] [-entrez_query entrez_query]\n",
            "    [-db_soft_mask filtering_algorithm] [-db_hard_mask filtering_algorithm]\n",
            "    [-subject subject_input_file] [-subject_loc range] [-query input_file]\n",
            "    [-out output_file] [-evalue evalue] [-word_size int_value]\n",
            "    [-gapopen open_penalty] [-gapextend extend_penalty]\n",
            "    [-qcov_hsp_perc float_value] [-max_hsps int_value]\n",
            "    [-xdrop_ungap float_value] [-xdrop_gap float_value]\n",
            "    [-xdrop_gap_final float_value] [-searchsp int_value] [-seg SEG_options]\n",
            "    [-soft_masking soft_masking] [-matrix matrix_name]\n",
            "    [-threshold float_value] [-culling_limit int_value]\n",
            "    [-best_hit_overhang float_value] [-best_hit_score_edge float_value]\n",
            "    [-subject_besthit] [-window_size int_value] [-lcase_masking]\n",
            "    [-query_loc range] [-parse_deflines] [-outfmt format] [-show_gis]\n",
            "    [-num_descriptions int_value] [-num_alignments int_value]\n",
            "    [-line_length line_length] [-html] [-sorthits sort_hits]\n",
            "    [-sorthsps sort_hsps] [-max_target_seqs num_sequences]\n",
            "    [-num_threads int_value] [-mt_mode int_value] [-ungapped] [-remote]\n",
            "    [-comp_based_stats compo] [-use_sw_tback] [-version]\n",
            "\n",
            "DESCRIPTION\n",
            "   Protein-Protein BLAST 2.15.0+\n",
            "\n",
            "Use '-help' to print detailed descriptions of command line arguments\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# This will execute the blastp command with the specified subject\n",
        "# and query sequences in the Colab environment.\n",
        "# Make sure to replace \"NP_000509.fasta\" and \"NP_001188320.fasta\"\n",
        "# with the actual filenames or paths of your subject and query sequence files.\n",
        "\n",
        "\n",
        "\n",
        "!/content/ncbi-blast-2.15.0+/bin/blastp -subject NP_000509.fasta -query NP_001188320.fasta"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "gb4qM3LegkSN",
        "outputId": "a53d225d-aa33-4a6e-e462-0f0ff678131e"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "BLASTP 2.15.0+\n",
            "\n",
            "\n",
            "Reference: Stephen F. Altschul, Thomas L. Madden, Alejandro A.\n",
            "Schaffer, Jinghui Zhang, Zheng Zhang, Webb Miller, and David J.\n",
            "Lipman (1997), \"Gapped BLAST and PSI-BLAST: a new generation of\n",
            "protein database search programs\", Nucleic Acids Res. 25:3389-3402.\n",
            "\n",
            "\n",
            "Reference for composition-based statistics: Alejandro A. Schaffer,\n",
            "L. Aravind, Thomas L. Madden, Sergei Shavirin, John L. Spouge, Yuri\n",
            "I. Wolf, Eugene V. Koonin, and Stephen F. Altschul (2001),\n",
            "\"Improving the accuracy of PSI-BLAST protein database searches with\n",
            "composition-based statistics and other refinements\", Nucleic Acids\n",
            "Res. 29:2994-3005.\n",
            "\n",
            "\n",
            "\n",
            "Database: User specified sequence set (Input: NP_000509.fasta).\n",
            "           1 sequences; 147 total letters\n",
            "\n",
            "\n",
            "\n",
            "Query= NP_001188320.1 hemoglobin, beta adult s chain [Mus musculus]\n",
            "\n",
            "Length=147\n",
            "                                                                      Score     E\n",
            "Sequences producing significant alignments:                          (Bits)  Value\n",
            "\n",
            "NP_000509.1 hemoglobin subunit beta [Homo sapiens]                    251     2e-92\n",
            "\n",
            "\n",
            "> NP_000509.1 hemoglobin subunit beta [Homo sapiens]\n",
            "Length=147\n",
            "\n",
            " Score = 251 bits (642),  Expect = 2e-92, Method: Compositional matrix adjust.\n",
            " Identities = 118/147 (80%), Positives = 132/147 (90%), Gaps = 0/147 (0%)\n",
            "\n",
            "Query  1    MVHLTDAEKAAVSGLWGKVNADEVGGEALGRLLVVYPWTQRYFDSFGDLSSASAIMGNAK  60\n",
            "            MVHLT  EK+AV+ LWGKVN DEVGGEALGRLLVVYPWTQR+F+SFGDLS+  A+MGN K\n",
            "Sbjct  1    MVHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPK  60\n",
            "\n",
            "Query  61   VKAHGKKVITAFNDGLNHLDSLKGTFASLSELHCDKLHVDPENFRLLGNMIVIVLGHHLG  120\n",
            "            VKAHGKKV+ AF+DGL HLD+LKGTFA+LSELHCDKLHVDPENFRLLGN++V VL HH G\n",
            "Sbjct  61   VKAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG  120\n",
            "\n",
            "Query  121  KDFTPAAQAAFQKVVAGVAAALAHKYH  147\n",
            "            K+FTP  QAA+QKVVAGVA ALAHKYH\n",
            "Sbjct  121  KEFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "\n",
            "\n",
            "\n",
            "Lambda      K        H        a         alpha\n",
            "   0.320    0.135    0.404    0.792     4.96 \n",
            "\n",
            "Gapped\n",
            "Lambda      K        H        a         alpha    sigma\n",
            "   0.267   0.0410    0.140     1.90     42.6     43.6 \n",
            "\n",
            "Effective search space used: 17161\n",
            "\n",
            "\n",
            "  Database: User specified sequence set (Input: NP_000509.fasta).\n",
            "    Posted date:  Unknown\n",
            "  Number of letters in database: 147\n",
            "  Number of sequences in database:  1\n",
            "\n",
            "\n",
            "\n",
            "Matrix: BLOSUM62\n",
            "Gap Penalties: Existence: 11, Extension: 1\n",
            "Neighboring words threshold: 11\n",
            "Window for multiple hits: 40\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "## Perform NA pairwise alignment\n"
      ],
      "metadata": {
        "id": "QZ_D78L_pyuN"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!efetch -db nuccore -format fasta -id NM_001201391.1"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "2sR_nzx4qx7d",
        "outputId": "eb9bbf5e-1551-4133-da11-a237938f6b42"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            ">NM_001201391.1 Mus musculus hemoglobin, beta adult s chain (Hbb-bs), mRNA\n",
            "CACATTTGCTTCTGACATAGTTGTGTTGACTCACAACCCCAGAAACAGACATCATGGTGCACCTGACTGA\n",
            "TGCTGAGAAGGCTGCTGTCTCTGGCCTGTGGGGAAAGGTGAACGCCGATGAAGTTGGTGGTGAGGCCCTG\n",
            "GGCAGGCTGCTGGTTGTCTACCCTTGGACCCAGCGGTACTTTGATAGCTTTGGAGACCTATCCTCTGCCT\n",
            "CTGCTATCATGGGTAATGCCAAAGTGAAGGCCCATGGCAAGAAAGTGATAACTGCCTTTAACGATGGCCT\n",
            "GAATCACTTGGACAGCCTCAAGGGCACCTTTGCCAGCCTCAGTGAGCTCCACTGTGACAAGCTGCATGTG\n",
            "GATCCTGAGAACTTCAGGCTCCTGGGCAATATGATCGTGATTGTGCTGGGCCACCACCTGGGCAAGGATT\n",
            "TCACCCCCGCTGCACAGGCTGCCTTCCAGAAGGTGGTGGCTGGAGTGGCCGCTGCCCTGGCTCACAAGTA\n",
            "CCACTAAACCCCCTTTCCTGCTCTTGCCTGTGAACAATGGTTAATTGTTCCCAAGAGAGCATCTGTCAGT\n",
            "TGTTGGCAAAATGATAAAGACATTTGAAAATCTGTCTTCTGACAAATAAAAAGCATTTATTTCACTGCAA\n",
            "TGATGTTTT\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "##This script fetches sequences from the NCBI database (here nuccore or nucleotide db) using their IDs\n",
        "## and saves them as FASTA files.\n",
        "!efetch -db nuccore -format fasta -id NM_000518.5 > NM_000518.fasta\n",
        "!efetch -db nuccore -format fasta -id NM_001201391.1 > NM_001201391.fasta\n"
      ],
      "metadata": {
        "id": "tAd4Ol5mp6RL"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "### This will provide information about the usage, options, and parameters available for the blastn command.\n",
        "!/content/ncbi-blast-2.15.0+/bin/blastn -h"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "1bfcd6f9-dcb5-4002-bb28-6b31b412e9a7",
        "id": "yAjqaoerrnfY"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "USAGE\n",
            "  blastn [-h] [-help] [-import_search_strategy filename]\n",
            "    [-export_search_strategy filename] [-task task_name] [-db database_name]\n",
            "    [-dbsize num_letters] [-gilist filename] [-seqidlist filename]\n",
            "    [-negative_gilist filename] [-negative_seqidlist filename]\n",
            "    [-taxids taxids] [-negative_taxids taxids] [-taxidlist filename]\n",
            "    [-negative_taxidlist filename] [-no_taxid_expansion]\n",
            "    [-entrez_query entrez_query] [-db_soft_mask filtering_algorithm]\n",
            "    [-db_hard_mask filtering_algorithm] [-subject subject_input_file]\n",
            "    [-subject_loc range] [-query input_file] [-out output_file]\n",
            "    [-evalue evalue] [-word_size int_value] [-gapopen open_penalty]\n",
            "    [-gapextend extend_penalty] [-perc_identity float_value]\n",
            "    [-qcov_hsp_perc float_value] [-max_hsps int_value]\n",
            "    [-xdrop_ungap float_value] [-xdrop_gap float_value]\n",
            "    [-xdrop_gap_final float_value] [-searchsp int_value] [-penalty penalty]\n",
            "    [-reward reward] [-no_greedy] [-min_raw_gapped_score int_value]\n",
            "    [-template_type type] [-template_length int_value] [-dust DUST_options]\n",
            "    [-filtering_db filtering_database]\n",
            "    [-window_masker_taxid window_masker_taxid]\n",
            "    [-window_masker_db window_masker_db] [-soft_masking soft_masking]\n",
            "    [-ungapped] [-culling_limit int_value] [-best_hit_overhang float_value]\n",
            "    [-best_hit_score_edge float_value] [-subject_besthit]\n",
            "    [-window_size int_value] [-off_diagonal_range int_value]\n",
            "    [-use_index boolean] [-index_name string] [-lcase_masking]\n",
            "    [-query_loc range] [-strand strand] [-parse_deflines] [-outfmt format]\n",
            "    [-show_gis] [-num_descriptions int_value] [-num_alignments int_value]\n",
            "    [-line_length line_length] [-html] [-sorthits sort_hits]\n",
            "    [-sorthsps sort_hsps] [-max_target_seqs num_sequences]\n",
            "    [-num_threads int_value] [-mt_mode int_value] [-remote] [-version]\n",
            "\n",
            "DESCRIPTION\n",
            "   Nucleotide-Nucleotide BLAST 2.15.0+\n",
            "\n",
            "Use '-help' to print detailed descriptions of command line arguments\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# This will execute the blastn command with the specified subject\n",
        "# and query sequences in the Colab environment.\n",
        "\n",
        "!/content/ncbi-blast-2.15.0+/bin/blastn -subject NM_000518.fasta -query NM_001201391.fasta"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "fvZqKghZsBCw",
        "outputId": "6dabc243-1cd3-45fa-c28b-b5c4e543dc79"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "BLASTN 2.15.0+\n",
            "\n",
            "\n",
            "Reference: Zheng Zhang, Scott Schwartz, Lukas Wagner, and Webb\n",
            "Miller (2000), \"A greedy algorithm for aligning DNA sequences\", J\n",
            "Comput Biol 2000; 7(1-2):203-14.\n",
            "\n",
            "\n",
            "\n",
            "Database: User specified sequence set (Input: NM_000518.fasta).\n",
            "           1 sequences; 628 total letters\n",
            "\n",
            "\n",
            "\n",
            "Query= NM_001201391.1 Mus musculus hemoglobin, beta adult s chain (Hbb-bs),\n",
            "mRNA\n",
            "\n",
            "Length=639\n",
            "                                                                      Score     E\n",
            "Sequences producing significant alignments:                          (Bits)  Value\n",
            "\n",
            "NM_000518.5 Homo sapiens hemoglobin subunit beta (HBB), mRNA          444     7e-129\n",
            "\n",
            "\n",
            "> NM_000518.5 Homo sapiens hemoglobin subunit beta (HBB), mRNA\n",
            "Length=628\n",
            "\n",
            " Score = 444 bits (240),  Expect = 7e-129\n",
            " Identities = 422/512 (82%), Gaps = 4/512 (1%)\n",
            " Strand=Plus/Plus\n",
            "\n",
            "Query  2    ACATTTGCTTCTGACATAGTTGTGTTGACTCACAACCCCAGAAACAGACATCATGGTGCA  61\n",
            "            |||||||||||||||| |  |||||| |||  ||||| |  ||||||||| |||||||||\n",
            "Sbjct  1    ACATTTGCTTCTGACACAACTGTGTTCACTAGCAACCTC--AAACAGACACCATGGTGCA  58\n",
            "\n",
            "Query  62   CCTGACTGATGCTGAGAAGGCTGCTGTCTCTGGCCTGTGGGGAAAGGTGAACGCCGATGA  121\n",
            "             ||||||  ||  |||||| |||| ||  ||| ||||||||| ||||||||||  |||||\n",
            "Sbjct  59   TCTGACTCCTGAGGAGAAGTCTGCCGTTACTGCCCTGTGGGGCAAGGTGAACGTGGATGA  118\n",
            "\n",
            "Query  122  AGTTGGTGGTGAGGCCCTGGGCAGGCTGCTGGTTGTCTACCCTTGGACCCAGCGGTACTT  181\n",
            "            ||||||||||||||||||||||||||||||||| |||||||||||||||||| ||| |||\n",
            "Sbjct  119  AGTTGGTGGTGAGGCCCTGGGCAGGCTGCTGGTGGTCTACCCTTGGACCCAGAGGTTCTT  178\n",
            "\n",
            "Query  182  TGATAGCTTTGGAGACCTATCCTCTGCCTC-TGCTATCATGGGTAATGCCAAAGTGAAGG  240\n",
            "            |||   |||||| || || ||| || |||  |||| | ||||| ||  | || |||||||\n",
            "Sbjct  179  TGAGTCCTTTGGGGATCTGTCCACT-CCTGATGCTGTTATGGGCAACCCTAAGGTGAAGG  237\n",
            "\n",
            "Query  241  CCCATGGCAAGAAAGTGATAACTGCCTTTAACGATGGCCTGAATCACTTGGACAGCCTCA  300\n",
            "            | ||||||||||||||| |   ||||||||  |||||||||  |||| |||||| |||||\n",
            "Sbjct  238  CTCATGGCAAGAAAGTGCTCGGTGCCTTTAGTGATGGCCTGGCTCACCTGGACAACCTCA  297\n",
            "\n",
            "Query  301  AGGGCACCTTTGCCAGCCTCAGTGAGCTCCACTGTGACAAGCTGCATGTGGATCCTGAGA  360\n",
            "            |||||||||||||||  || |||||||| ||||||||||||||||| |||||||||||||\n",
            "Sbjct  298  AGGGCACCTTTGCCACACTGAGTGAGCTGCACTGTGACAAGCTGCACGTGGATCCTGAGA  357\n",
            "\n",
            "Query  361  ACTTCAGGCTCCTGGGCAATATGATCGTGATTGTGCTGGGCCACCACCTGGGCAAGGATT  420\n",
            "            |||||||||||||||||||  || | ||   |||||||| ||| ||| | ||||| || |\n",
            "Sbjct  358  ACTTCAGGCTCCTGGGCAACGTGCTGGTCTGTGTGCTGGCCCATCACTTTGGCAAAGAAT  417\n",
            "\n",
            "Query  421  TCACCCCCGCTGCACAGGCTGCCTTCCAGAAGGTGGTGGCTGGAGTGGCCGCTGCCCTGG  480\n",
            "            |||||||  | |  ||||||||||  ||||| ||||||||||| |||||   ||||||||\n",
            "Sbjct  418  TCACCCCACCAGTGCAGGCTGCCTATCAGAAAGTGGTGGCTGGTGTGGCTAATGCCCTGG  477\n",
            "\n",
            "Query  481  CTCACAAGTACCACTAAACCCCCTTTCCTGCT  512\n",
            "            | |||||||| |||||| | | ||||| ||||\n",
            "Sbjct  478  CCCACAAGTATCACTAAGCTCGCTTTCTTGCT  509\n",
            "\n",
            "\n",
            "\n",
            "Lambda      K        H\n",
            "    1.33    0.621     1.12 \n",
            "\n",
            "Gapped\n",
            "Lambda      K        H\n",
            "    1.28    0.460    0.850 \n",
            "\n",
            "Effective search space used: 386232\n",
            "\n",
            "\n",
            "  Database: User specified sequence set (Input: NM_000518.fasta).\n",
            "    Posted date:  Unknown\n",
            "  Number of letters in database: 628\n",
            "  Number of sequences in database:  1\n",
            "\n",
            "\n",
            "\n",
            "Matrix: blastn matrix 1 -2\n",
            "Gap Penalties: Existence: 0, Extension: 2.5\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#BLAST"
      ],
      "metadata": {
        "id": "EFT8hGdohJ5o"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "You can get all BLAST DBs from : https://ftp.ncbi.nlm.nih.gov/blast/db/"
      ],
      "metadata": {
        "id": "RW9AZJwWOeoJ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!wget https://ftp.ncbi.nlm.nih.gov/blast/db/pdbaa.tar.gz\n",
        "!tar -xvzf /content/pdbaa.tar.gz"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "id": "Q_7blzM4NGqy",
        "outputId": "dbdcda04-1831-4aab-fcb4-2d3de214c475"
      },
      "execution_count": 22,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-05-13 09:10:30--  https://ftp.ncbi.nlm.nih.gov/blast/db/pdbaa.tar.gz\n",
            "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.12, 130.14.250.13, 2607:f220:41e:250::10, ...\n",
            "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.12|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 107351900 (102M) [application/x-gzip]\n",
            "Saving to: ‘pdbaa.tar.gz.1’\n",
            "\n",
            "pdbaa.tar.gz.1      100%[===================>] 102.38M  41.8MB/s    in 2.4s    \n",
            "\n",
            "2024-05-13 09:10:33 (41.8 MB/s) - ‘pdbaa.tar.gz.1’ saved [107351900/107351900]\n",
            "\n",
            "taxdb.btd\n",
            "taxdb.bti\n",
            "taxonomy4blast.sqlite3\n",
            "pdbaa.pin\n",
            "pdbaa.phr\n",
            "pdbaa.psq\n",
            "pdbaa.ppi\n",
            "pdbaa.ppd\n",
            "pdbaa.pog\n",
            "pdbaa.pdb\n",
            "pdbaa.pos\n",
            "pdbaa.pot\n",
            "pdbaa.ptf\n",
            "pdbaa.pto\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!/content/ncbi-blast-2.15.0+/bin/blastp -db pdbaa -query NP_000509.fasta"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YOJjUiOrLyP1",
        "outputId": "ade54ad1-5f28-48fb-e775-fee9b106703e"
      },
      "execution_count": 25,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTPPVQAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPPVQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">1Y4B_B Chain B, Hemoglobin beta chain [Homo sapiens]\n",
            " 1Y4B_D Chain D, Hemoglobin beta chain [Homo sapiens]\n",
            " 1YG5_B Chain B, Hemoglobin beta chain [Homo sapiens]\n",
            " 1YG5_D Chain D, Hemoglobin beta chain [Homo sapiens]\n",
            "Length=146\n",
            "\n",
            " Score = 293 bits (751),  Expect = 1e-103, Method: Compositional matrix adjust.\n",
            " Identities = 144/146 (99%), Positives = 145/146 (99%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            +HLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYP TQRFFESFGDLSTPDAVMGNPKV\n",
            "Sbjct  1    MHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPHTQRFFESFGDLSTPDAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK\n",
            "Sbjct  61   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTPPVQAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPPVQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">1O1P_B Chain B, Hemoglobin beta chain [Homo sapiens]\n",
            " 1O1P_D Chain D, Hemoglobin beta chain [Homo sapiens]\n",
            "Length=146\n",
            "\n",
            " Score = 293 bits (749),  Expect = 3e-103, Method: Compositional matrix adjust.\n",
            " Identities = 143/146 (98%), Positives = 144/146 (99%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            +HLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV\n",
            "Sbjct  1    MHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVLGAFSDGLAHLDNLKGTFATLSELH DKLHVDPENFRLLG VLVCVLAHHFGK\n",
            "Sbjct  61   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHADKLHVDPENFRLLGKVLVCVLAHHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTPPVQAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPPVQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">6NBC_B Chain B, Hemoglobin subunit beta [Homo sapiens]\n",
            " 6NBC_D Chain D, Hemoglobin subunit beta [Homo sapiens]\n",
            " 6NBD_B Chain B, Hemoglobin subunit beta [Homo sapiens]\n",
            " 6NBD_D Chain D, Hemoglobin subunit beta [Homo sapiens]\n",
            "Length=143\n",
            "\n",
            " Score = 291 bits (746),  Expect = 6e-103, Method: Compositional matrix adjust.\n",
            " Identities = 143/143 (100%), Positives = 143/143 (100%), Gaps = 0/143 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV\n",
            "Sbjct  1    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK\n",
            "Sbjct  61   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAH  144\n",
            "            EFTPPVQAAYQKVVAGVANALAH\n",
            "Sbjct  121  EFTPPVQAAYQKVVAGVANALAH  143\n",
            "\n",
            "\n",
            ">1GBU_B Chain B, HEMOGLOBIN [Homo sapiens]\n",
            " 1GBU_D Chain D, HEMOGLOBIN [Homo sapiens]\n",
            "Length=146\n",
            "\n",
            " Score = 291 bits (746),  Expect = 6e-103, Method: Compositional matrix adjust.\n",
            " Identities = 144/146 (99%), Positives = 144/146 (99%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV\n",
            "Sbjct  1    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVLGAFSDGLAHLDNLKGTFATLSELH DKLHVDPENFRLLGNVLV VLAHHFGK\n",
            "Sbjct  61   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHADKLHVDPENFRLLGNVLVGVLAHHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTPPVQAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPPVQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">5HU6_B Chain B, Hemoglobin subunit beta [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 288 bits (736),  Expect = 2e-101, Method: Compositional matrix adjust.\n",
            " Identities = 141/141 (100%), Positives = 141/141 (100%), Gaps = 0/141 (0%)\n",
            "\n",
            "Query  3    HLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVK  62\n",
            "            HLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVK\n",
            "Sbjct  1    HLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVK  60\n",
            "\n",
            "Query  63   AHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE  122\n",
            "            AHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE\n",
            "Sbjct  61   AHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE  120\n",
            "\n",
            "Query  123  FTPPVQAAYQKVVAGVANALA  143\n",
            "            FTPPVQAAYQKVVAGVANALA\n",
            "Sbjct  121  FTPPVQAAYQKVVAGVANALA  141\n",
            "\n",
            "\n",
            ">1SHR_B Chain B, Hemoglobin delta chain [Homo sapiens]\n",
            " 1SHR_D Chain D, Hemoglobin delta chain [Homo sapiens]\n",
            " 1SI4_B Chain B, Hemoglobin delta chain [Homo sapiens]\n",
            " 1SI4_D Chain D, Hemoglobin delta chain [Homo sapiens]\n",
            "Length=146\n",
            "\n",
            " Score = 281 bits (719),  Expect = 8e-99, Method: Compositional matrix adjust.\n",
            " Identities = 136/146 (93%), Positives = 141/146 (97%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLTPEEK+AV ALWGKVNVD VGGEALGRLLVVYPWTQRFFESFGDLS+PDAVMGNPKV\n",
            "Sbjct  1    VHLTPEEKTAVNALWGKVNVDAVGGEALGRLLVVYPWTQRFFESFGDLSSPDAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVLGAFSDGLAHLDNLKGTF+ LSELHCDKLHVDPENFRLLGNVLVCVLA +FGK\n",
            "Sbjct  61   KAHGKKVLGAFSDGLAHLDNLKGTFSQLSELHCDKLHVDPENFRLLGNVLVCVLARNFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP +QAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPQMQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">3LQD_B Chain B, Hemoglobin subunit beta [Lepus europaeus]\n",
            " 3LQD_D Chain D, Hemoglobin subunit beta [Lepus europaeus]\n",
            "Length=146\n",
            "\n",
            " Score = 274 bits (700),  Expect = 7e-96, Method: Compositional matrix adjust.\n",
            " Identities = 133/146 (91%), Positives = 138/146 (95%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHL+ EEKSAVTALWGKVNV+EVGGE LGRLLVVYPWTQRFFESFGDLST  AVMGNPKV\n",
            "Sbjct  1    VHLSGEEKSAVTALWGKVNVEEVGGETLGRLLVVYPWTQRFFESFGDLSTASAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL AFS+GL+HLDNLKGTFA LSELHCDKLHVDPENFRLLGNVLV VL+HHFGK\n",
            "Sbjct  61   KAHGKKVLAAFSEGLSHLDNLKGTFAKLSELHCDKLHVDPENFRLLGNVLVIVLSHHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP VQAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPQVQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">2RAO_B Chain B, Hemoglobin subunit beta-1/2 [Oryctolagus cuniculus]\n",
            " 2RAO_D Chain D, Hemoglobin subunit beta-1/2 [Oryctolagus cuniculus]\n",
            "Length=146\n",
            "\n",
            " Score = 273 bits (699),  Expect = 8e-96, Method: Compositional matrix adjust.\n",
            " Identities = 132/146 (90%), Positives = 139/146 (95%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHL+ EEKSAVTALWGKVNV+EVGGEALGRLLVVYPWTQRFFESFGDLS+ +AVM NPKV\n",
            "Sbjct  1    VHLSSEEKSAVTALWGKVNVEEVGGEALGRLLVVYPWTQRFFESFGDLSSANAVMNNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL AFS+GL+HLDNLKGTFA LSELHCDKLHVDPENFRLLGNVLV VL+HHFGK\n",
            "Sbjct  61   KAHGKKVLAAFSEGLSHLDNLKGTFAKLSELHCDKLHVDPENFRLLGNVLVIVLSHHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP VQAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPQVQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">1FHJ_B Chain B, HEMOGLOBIN (BETA CHAIN) [Chrysocyon brachyurus]\n",
            " 1FHJ_D Chain D, HEMOGLOBIN (BETA CHAIN) [Chrysocyon brachyurus]\n",
            " 2QLS_B Chain B, Hemoglobin subunit beta [Canis lupus familiaris]\n",
            " 2QLS_D Chain D, Hemoglobin subunit beta [Canis lupus familiaris]\n",
            " 3GOU_B Chain B, Hemoglobin subunit beta [Canis lupus familiaris]\n",
            " 3GOU_D Chain D, Hemoglobin subunit beta [Canis lupus familiaris]\n",
            " 3PEL_B Chain B, Hemoglobin subunit beta [Canis lupus familiaris]\n",
            "Length=146\n",
            "\n",
            " Score = 272 bits (695),  Expect = 4e-95, Method: Compositional matrix adjust.\n",
            " Identities = 131/146 (90%), Positives = 137/146 (94%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT EEKS V+ LWGKVNVDEVGGEALGRLL+VYPWTQRFF+SFGDLSTPDAVM N KV\n",
            "Sbjct  1    VHLTAEEKSLVSGLWGKVNVDEVGGEALGRLLIVYPWTQRFFDSFGDLSTPDAVMSNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +FSDGL +LDNLKGTFA LSELHCDKLHVDPENF+LLGNVLVCVLAHHFGK\n",
            "Sbjct  61   KAHGKKVLNSFSDGLKNLDNLKGTFAKLSELHCDKLHVDPENFKLLGNVLVCVLAHHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP VQAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPQVQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">2B7H_B Chain B, hemoglobin beta chain [Cerdocyon thous]\n",
            " 2B7H_D Chain D, hemoglobin beta chain [Cerdocyon thous]\n",
            "Length=146\n",
            "\n",
            " Score = 269 bits (688),  Expect = 4e-94, Method: Compositional matrix adjust.\n",
            " Identities = 129/146 (88%), Positives = 136/146 (93%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT EEKS V+ LW KVNVDEVGGEALGRLL+VYPWTQRFF+SFGDLSTPD+VM N KV\n",
            "Sbjct  1    VHLTAEEKSLVSGLWAKVNVDEVGGEALGRLLIVYPWTQRFFDSFGDLSTPDSVMSNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +FSDGL +LDNLKGTFA LSELHCDKLHVDPENF+LLGNVLVCVLAHHFGK\n",
            "Sbjct  61   KAHGKKVLNSFSDGLKNLDNLKGTFAKLSELHCDKLHVDPENFKLLGNVLVCVLAHHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP VQAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPQVQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">3FH9_B Chain B, Hemoglobin beta chain [Pteropus giganteus]\n",
            "Length=146\n",
            "\n",
            " Score = 268 bits (685),  Expect = 1e-93, Method: Compositional matrix adjust.\n",
            " Identities = 130/146 (89%), Positives = 136/146 (93%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHL+ EEK+AVT LWGKV VDEVGGEALGRLLVVYPWTQRFF+SFGDLS+  AVMGNPKV\n",
            "Sbjct  1    VHLSGEEKAAVTGLWGKVKVDEVGGEALGRLLVVYPWTQRFFDSFGDLSSASAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +FS+GL HLDNLKGTFA LSELHCDKLHVDPENFRLLGNVLVCVLA HFGK\n",
            "Sbjct  61   KAHGKKVLDSFSEGLQHLDNLKGTFAKLSELHCDKLHVDPENFRLLGNVLVCVLARHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP VQAAYQKVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPQVQAAYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">4YU3_B Chain B, hemoglobin [Helogale parvula]\n",
            " 4YU4_B Chain B, hemoglobin [Helogale parvula]\n",
            " 4YU4_D Chain D, hemoglobin [Helogale parvula]\n",
            "Length=146\n",
            "\n",
            " Score = 265 bits (677),  Expect = 3e-92, Method: Compositional matrix adjust.\n",
            " Identities = 125/146 (86%), Positives = 136/146 (93%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT EEK+ V+ LWGKVN +EVGGEALGRLLVVYPWTQRFFE+FGDLS+ +A+M NPKV\n",
            "Sbjct  1    VHLTAEEKAHVSGLWGKVNTEEVGGEALGRLLVVYPWTQRFFETFGDLSSANAIMNNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +FSDGL +LDNLKGTFA LSELHCDKLHVDPENF+LLGNVLVCVLAHHFGK\n",
            "Sbjct  61   KAHGKKVLSSFSDGLKNLDNLKGTFAALSELHCDKLHVDPENFKLLGNVLVCVLAHHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP VQAAYQK+VAGVANALAHKYH\n",
            "Sbjct  121  EFTPQVQAAYQKIVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">1QPW_B Chain B, PORCINE HEMOGLOBIN (BETA SUBUNIT) [Sus scrofa]\n",
            " 1QPW_D Chain D, PORCINE HEMOGLOBIN (BETA SUBUNIT) [Sus scrofa]\n",
            " 2PGH_B Chain B, HEMOGLOBIN (AQUO MET) (BETA CHAIN) [Sus scrofa]\n",
            " 2PGH_D Chain D, HEMOGLOBIN (AQUO MET) (BETA CHAIN) [Sus scrofa]\n",
            "Length=146\n",
            "\n",
            " Score = 255 bits (651),  Expect = 2e-88, Method: Compositional matrix adjust.\n",
            " Identities = 124/146 (85%), Positives = 130/146 (89%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHL+ EEK AV  LWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLS  DAVMGNPKV\n",
            "Sbjct  1    VHLSAEEKEAVLGLWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSNADAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +FSDGL HLDNLKGTFA LSELHCD+LHVDPENFRLLGNV+V VLA   G \n",
            "Sbjct  61   KAHGKKVLQSFSDGLKHLDNLKGTFAKLSELHCDQLHVDPENFRLLGNVIVVVLARRLGH  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +F P VQAA+QKVVAGVANALAHKYH\n",
            "Sbjct  121  DFNPDVQAAFQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">4F4O_B Chain B, Hemoglobin subunit beta [Sus scrofa]\n",
            " 4F4O_E Chain E, Hemoglobin subunit beta [Sus scrofa]\n",
            " 4F4O_H Chain H, Hemoglobin subunit beta [Sus scrofa]\n",
            " 4F4O_K Chain K, Hemoglobin subunit beta [Sus scrofa]\n",
            "Length=146\n",
            "\n",
            " Score = 255 bits (651),  Expect = 2e-88, Method: Compositional matrix adjust.\n",
            " Identities = 124/146 (85%), Positives = 130/146 (89%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHL+ EEK AV  LWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLS  DAVMGNPKV\n",
            "Sbjct  1    VHLSAEEKEAVLGLWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSNADAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +FSDGL HLDNLKGTFA LSELHCD+LHVDPENFRLLGNV+V VLA   G \n",
            "Sbjct  61   KAHGKKVLQSFSDGLKHLDNLKGTFAKLSELHCDQLHVDPENFRLLGNVIVVVLARRLGH  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +F P VQAA+QKVVAGVANALAHKYH\n",
            "Sbjct  121  DFNPNVQAAFQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">3D4X_B Chain B, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3D4X_D Chain D, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GQP_B Chain B, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GQP_D Chain D, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GQR_B Chain B, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GQR_D Chain D, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GQR_F Chain F, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GQR_H Chain H, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GYS_B Chain B, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GYS_D Chain D, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GYS_F Chain F, Hemoglobin subunit beta-A/B [Felis catus]\n",
            " 3GYS_H Chain H, Hemoglobin subunit beta-A/B [Felis catus]\n",
            "Length=145\n",
            "\n",
            " Score = 253 bits (645),  Expect = 2e-87, Method: Compositional matrix adjust.\n",
            " Identities = 121/144 (84%), Positives = 129/144 (90%), Gaps = 0/144 (0%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKA  63\n",
            "            LT EEK  V  LWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLS+ DA+M N KVKA\n",
            "Sbjct  2    LTAEEKGLVNGLWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSSADAIMSNAKVKA  61\n",
            "\n",
            "Query  64   HGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEF  123\n",
            "            HGKKVL +FSDGL ++D+LKG FA LSELHCDKLHVDPENFRLLGNVLVCVLAHHFG +F\n",
            "Sbjct  62   HGKKVLNSFSDGLKNIDDLKGAFAKLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGHDF  121\n",
            "\n",
            "Query  124  TPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             P VQAA+QKVVAGVANALAHKYH\n",
            "Sbjct  122  NPQVQAAFQKVVAGVANALAHKYH  145\n",
            "\n",
            "\n",
            ">3DHT_B Chain B, Hemoglobin subunit beta-1 [Rattus norvegicus]\n",
            " 3HF4_B Chain B, Hemoglobin subunit beta-1 [Rattus norvegicus]\n",
            " 3HF4_F Chain F, Hemoglobin subunit beta-1 [Rattus norvegicus]\n",
            "Length=146\n",
            "\n",
            " Score = 249 bits (635),  Expect = 5e-86, Method: Compositional matrix adjust.\n",
            " Identities = 119/146 (82%), Positives = 131/146 (90%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT  EK+AV  LWGKVN D+VGGEALGRLLVVYPWTQR+F+SFGDLS+  A+MGNPKV\n",
            "Sbjct  1    VHLTDAEKAAVNGLWGKVNPDDVGGEALGRLLVVYPWTQRYFDSFGDLSSASAIMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+ AF+DGL HLDNLKGTFA LSELHCDKLHVDPENFRLLGN++V VL HH GK\n",
            "Sbjct  61   KAHGKKVINAFNDGLKHLDNLKGTFAHLSELHCDKLHVDPENFRLLGNMIVIVLGHHLGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP  QAA+QKVVAGVA+ALAHKYH\n",
            "Sbjct  121  EFTPCAQAAFQKVVAGVASALAHKYH  146\n",
            "\n",
            "\n",
            ">3HRW_B Chain B, Hemoglobin subunit beta-1 [Mus musculus]\n",
            " 3HRW_D Chain D, Hemoglobin subunit beta-1 [Mus musculus]\n",
            "Length=146\n",
            "\n",
            " Score = 249 bits (635),  Expect = 6e-86, Method: Compositional matrix adjust.\n",
            " Identities = 117/146 (80%), Positives = 131/146 (90%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT  EK+AV+ LWGKVN DEVGGEALGRLLVVYPWTQR+F+SFGDLS+  A+MGN KV\n",
            "Sbjct  1    VHLTDAEKAAVSCLWGKVNSDEVGGEALGRLLVVYPWTQRYFDSFGDLSSASAIMGNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+ AF+DGL HLD+LKGTFA+LSELHCDKLHVDPENFRLLGN++V VL HH GK\n",
            "Sbjct  61   KAHGKKVITAFNDGLNHLDSLKGTFASLSELHCDKLHVDPENFRLLGNMIVIVLGHHLGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAA+QKVVAGVA ALAHKYH\n",
            "Sbjct  121  DFTPAAQAAFQKVVAGVATALAHKYH  146\n",
            "\n",
            "\n",
            ">2DHB_B Chain B, HEMOGLOBIN (DEOXY) (BETA CHAIN) [Equus caballus]\n",
            "Length=146\n",
            "\n",
            " Score = 247 bits (630),  Expect = 3e-85, Method: Compositional matrix adjust.\n",
            " Identities = 120/146 (82%), Positives = 131/146 (90%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V L+ EEK+AV ALW KVN +EVGGEALGRLLVVYPWTQRFF+SFGDLS P AVMGNPKV\n",
            "Sbjct  1    VQLSGEEKAAVLALWDKVNEEEVGGEALGRLLVVYPWTQRFFDSFGDLSNPGAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +F +G+ HLDNLKGTFA LSELHCDKLHVDPENFRLLGNVL  V+A HFGK\n",
            "Sbjct  61   KAHGKKVLHSFGEGVHHLDNLKGTFAALSELHCDKLHVDPENFRLLGNVLALVVARHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP +QA+YQKVVAGVANALAHKYH\n",
            "Sbjct  121  DFTPELQASYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">1CH4_A Chain A, MODULE-SUBSTITUTED CHIMERA HEMOGLOBIN BETA-ALPHA [Homo \n",
            "sapiens]\n",
            " 1CH4_B Chain B, MODULE-SUBSTITUTED CHIMERA HEMOGLOBIN BETA-ALPHA [Homo \n",
            "sapiens]\n",
            " 1CH4_C Chain C, MODULE-SUBSTITUTED CHIMERA HEMOGLOBIN BETA-ALPHA [Homo \n",
            "sapiens]\n",
            " 1CH4_D Chain D, MODULE-SUBSTITUTED CHIMERA HEMOGLOBIN BETA-ALPHA [Homo \n",
            "sapiens]\n",
            "Length=146\n",
            "\n",
            " Score = 244 bits (623),  Expect = 4e-84, Method: Compositional matrix adjust.\n",
            " Identities = 120/146 (82%), Positives = 126/146 (86%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV\n",
            "Sbjct  1    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKL VDP NF+LL + L+  LA H   \n",
            "Sbjct  61   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLRVDPVNFKLLSHCLLVTLAAHLPA  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP V A+  KV+A V+  L  KY \n",
            "Sbjct  121  EFTPAVHASLDKVLASVSTVLTSKYR  146\n",
            "\n",
            "\n",
            ">4H2L_B Chain B, Beta globin [Peromyscus maniculatus]\n",
            "Length=146\n",
            "\n",
            " Score = 242 bits (618),  Expect = 2e-83, Method: Compositional matrix adjust.\n",
            " Identities = 112/146 (77%), Positives = 129/146 (88%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT  EK+ VT LWGKV  +E+GGEALGRLL VYPWTQRFF+SFGDLS+  A+MGN KV\n",
            "Sbjct  1    VHLTDAEKALVTGLWGKVKPEEIGGEALGRLLAVYPWTQRFFDSFGDLSSASAIMGNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+ +FS+GL HLDNLKGTFA+LSELHCDKLHVDPENF+LLGN++V V+AHH GK\n",
            "Sbjct  61   KAHGKKVIDSFSEGLKHLDNLKGTFASLSELHCDKLHVDPENFKLLGNMIVIVMAHHLGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  Q+AYQKVV+GVA ALAHKYH\n",
            "Sbjct  121  DFTPAAQSAYQKVVSGVATALAHKYH  146\n",
            "\n",
            "\n",
            ">5KER_B Chain B, Beta globin [Peromyscus maniculatus]\n",
            " 5KER_D Chain D, Beta globin [Peromyscus maniculatus]\n",
            " 5KER_F Chain F, Beta globin [Peromyscus maniculatus]\n",
            " 5KER_H Chain H, Beta globin [Peromyscus maniculatus]\n",
            "Length=146\n",
            "\n",
            " Score = 240 bits (613),  Expect = 1e-82, Method: Compositional matrix adjust.\n",
            " Identities = 112/146 (77%), Positives = 127/146 (87%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT  EK+ VT LWGKV  +E+GGEALGRLL VYPWTQRFF+SFGDLS+  A+MGN KV\n",
            "Sbjct  1    VHLTDAEKALVTGLWGKVKPEEIGGEALGRLLAVYPWTQRFFDSFGDLSSASAIMGNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV+ +F +GL HLDNLKGTFA+LSELHCDKLHVDPENF+LLGN++V V+AHH GK\n",
            "Sbjct  61   KGHGKKVIDSFGEGLKHLDNLKGTFASLSELHCDKLHVDPENFKLLGNMIVIVMAHHLGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAAYQKVVAGVA ALAHKYH\n",
            "Sbjct  121  DFTPAAQAAYQKVVAGVATALAHKYH  146\n",
            "\n",
            "\n",
            ">1A9W_E Chain E, HEMOGLOBIN (BETA CHAIN) [Homo sapiens]\n",
            " 1A9W_F Chain F, HEMOGLOBIN (BETA CHAIN) [Homo sapiens]\n",
            "Length=146\n",
            "\n",
            " Score = 238 bits (606),  Expect = 1e-81, Method: Compositional matrix adjust.\n",
            " Identities = 110/146 (75%), Positives = 129/146 (88%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH T EEK+AVT+LW K+NV+E GGEALGRLLVVYPWTQRFF+SFG+LS+P A++GNPKV\n",
            "Sbjct  1    VHFTAEEKAAVTSLWSKMNVEEAGGEALGRLLVVYPWTQRFFDSFGNLSSPSAILGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +F D + ++DNLK  FA LSELHCDKLHVDPENF+LLGNV+V +LA HFGK\n",
            "Sbjct  61   KAHGKKVLTSFGDAIKNMDNLKPAFAKLSELHCDKLHVDPENFKLLGNVMVIILATHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP VQAA+QK+V+ VA ALAHKYH\n",
            "Sbjct  121  EFTPEVQAAWQKLVSAVAIALAHKYH  146\n",
            "\n",
            "\n",
            ">3VRE_B Chain B, Hemoglobin subunit beta/delta hybrid [Mammuthus primigenius]\n",
            " 3VRE_D Chain D, Hemoglobin subunit beta/delta hybrid [Mammuthus primigenius]\n",
            " 3VRF_B Chain B, Hemoglobin subunit beta/delta hybrid [Mammuthus primigenius]\n",
            " 3VRG_B Chain B, Hemoglobin subunit beta/delta hybrid [Mammuthus primigenius]\n",
            "Length=146\n",
            "\n",
            " Score = 238 bits (606),  Expect = 2e-81, Method: Compositional matrix adjust.\n",
            " Identities = 116/146 (79%), Positives = 127/146 (87%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V+LT  EK+ V  LWGKVNV E+GGEAL RLLVVYPWT+RFFE FGDLST DAV+ N KV\n",
            "Sbjct  1    VNLTAAEKTQVANLWGKVNVKELGGEALSRLLVVYPWTRRFFEHFGDLSTADAVLHNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHG+KVL +F +GL HLDNLKGTF+ LSELHCDKLHVDP+NFRLLGNVLV VLA HFGK\n",
            "Sbjct  61   LAHGEKVLTSFGEGLKHLDNLKGTFSDLSELHCDKLHVDPQNFRLLGNVLVIVLARHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP VQAAY+KVVAGVANALAHKYH\n",
            "Sbjct  121  EFTPDVQAAYEKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">5EUI_B Chain B, HBB protein [Ochotona princeps]\n",
            "Length=146\n",
            "\n",
            " Score = 236 bits (602),  Expect = 6e-81, Method: Compositional matrix adjust.\n",
            " Identities = 123/146 (84%), Positives = 133/146 (91%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHL+ EEK+AVTALWGKVNVDEVGGE LGRLLVVYPWTQRFFE+FGDLS+  AVMGN KV\n",
            "Sbjct  1    VHLSGEEKAAVTALWGKVNVDEVGGETLGRLLVVYPWTQRFFETFGDLSSASAVMGNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+ AFS+GL HLDNLKGTFA LSELHCDKLHVDPENF+LLGNVLV VL+HH G \n",
            "Sbjct  61   KAHGKKVMNAFSEGLHHLDNLKGTFAKLSELHCDKLHVDPENFKLLGNVLVVVLSHHLGG  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP  QAA+QKVV+GVANALAHKYH\n",
            "Sbjct  121  EFTPQAQAAWQKVVSGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">4MQJ_B Chain B, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            " 4MQJ_D Chain D, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            " 4MQJ_F Chain F, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            " 4MQJ_H Chain H, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            "Length=146\n",
            "\n",
            " Score = 236 bits (601),  Expect = 8e-81, Method: Compositional matrix adjust.\n",
            " Identities = 108/146 (74%), Positives = 128/146 (88%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH T E+K+ +T+LWGKVNV++ GGE LGRLLVVYPWTQRFF+SFG+LS+  A+MGNPKV\n",
            "Sbjct  1    VHFTEEDKATITSLWGKVNVEDAGGETLGRLLVVYPWTQRFFDSFGNLSSASAIMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +  D + HLD+LKGTFA LSELHCDKLHVDPENF+LLGNVLV VLA HFGK\n",
            "Sbjct  61   KAHGKKVLTSLGDAIKHLDDLKGTFAQLSELHCDKLHVDPENFKLLGNVLVTVLAIHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP VQA++QK+V GVA+AL+ +YH\n",
            "Sbjct  121  EFTPEVQASWQKMVTGVASALSSRYH  146\n",
            "\n",
            "\n",
            ">3CY5_B Chain B, Hemoglobin subunit beta [Bubalus bubalis]\n",
            " 3CY5_D Chain D, Hemoglobin subunit beta [Bubalus bubalis]\n",
            "Length=145\n",
            "\n",
            " Score = 236 bits (601),  Expect = 8e-81, Method: Compositional matrix adjust.\n",
            " Identities = 123/144 (85%), Positives = 133/144 (92%), Gaps = 0/144 (0%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKA  63\n",
            "            LT EEK+AVTA WGKV+VDEVGGEALGRLLVVYPWTQRFFESFGDLST DAVM NPKVKA\n",
            "Sbjct  2    LTAEEKAAVTAFWGKVHVDEVGGEALGRLLVVYPWTQRFFESFGDLSTADAVMNNPKVKA  61\n",
            "\n",
            "Query  64   HGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEF  123\n",
            "            HGKKVL +FS+G+ HLD+LKGTFA LSELHCDKLHVDPENF+LLGNVLV VLA HFGKEF\n",
            "Sbjct  62   HGKKVLDSFSNGMKHLDDLKGTFAALSELHCDKLHVDPENFKLLGNVLVVVLARHFGKEF  121\n",
            "\n",
            "Query  124  TPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            TP +QA +QKVVAGVANALAH+YH\n",
            "Sbjct  122  TPVLQADFQKVVAGVANALAHRYH  145\n",
            "\n",
            "\n",
            ">7QU4_G Chain G, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            " 7QU4_H Chain H, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            "Length=147\n",
            "\n",
            " Score = 235 bits (600),  Expect = 1e-80, Method: Compositional matrix adjust.\n",
            " Identities = 108/147 (73%), Positives = 128/147 (87%), Gaps = 0/147 (0%)\n",
            "\n",
            "Query  1    MVHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPK  60\n",
            "            M H T E+K+ +T+LWGKVNV++ GGE LGRLLVVYPWTQRFF+SFG+LS+  A+MGNPK\n",
            "Sbjct  1    MGHFTEEDKATITSLWGKVNVEDAGGETLGRLLVVYPWTQRFFDSFGNLSSASAIMGNPK  60\n",
            "\n",
            "Query  61   VKAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG  120\n",
            "            VKAHGKKVL +  D + HLD+LKGTFA LSELHCDKLHVDPENF+LLGNVLV VLA HFG\n",
            "Sbjct  61   VKAHGKKVLTSLGDAIKHLDDLKGTFAQLSELHCDKLHVDPENFKLLGNVLVTVLAIHFG  120\n",
            "\n",
            "Query  121  KEFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            KEFTP VQA++QK+V GVA+AL+ +YH\n",
            "Sbjct  121  KEFTPEVQASWQKMVTGVASALSSRYH  147\n",
            "\n",
            "\n",
            ">3GDJ_B Chain B, Hemoglobin subunit beta [Camelus dromedarius]\n",
            " 3GDJ_D Chain D, Hemoglobin subunit beta [Camelus dromedarius]\n",
            "Length=146\n",
            "\n",
            " Score = 234 bits (597),  Expect = 3e-80, Method: Compositional matrix adjust.\n",
            " Identities = 123/146 (84%), Positives = 131/146 (90%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHL+ +EK+AV  LW KV VDEVGGEALGRLLVVYPWT+RFFESFGDLST DAVM NPKV\n",
            "Sbjct  1    VHLSGDEKNAVHGLWSKVKVDEVGGEALGRLLVVYPWTRRFFESFGDLSTADAVMNNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHG KVL +F DGL HLDNLKGT+A LSELHCDKLHVDPENFRLLGNVLV VLA HFGK\n",
            "Sbjct  61   KAHGSKVLNSFGDGLNHLDNLKGTYAKLSELHCDKLHVDPENFRLLGNVLVVVLARHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP +QAAYQKVVAGVANALAH+YH\n",
            "Sbjct  121  EFTPDLQAAYQKVVAGVANALAHRYH  146\n",
            "\n",
            "\n",
            ">1FSX_B Chain B, HEMOGLOBIN BETA CHAIN [Bos taurus]\n",
            " 1FSX_D Chain D, HEMOGLOBIN BETA CHAIN [Bos taurus]\n",
            " 1G08_B Chain B, HEMOGLOBIN BETA CHAIN [Bos taurus]\n",
            " 1G08_D Chain D, HEMOGLOBIN BETA CHAIN [Bos taurus]\n",
            " 1G09_B Chain B, HEMOGLOBIN BETA CHAIN [Bos taurus]\n",
            " 1G09_D Chain D, HEMOGLOBIN BETA CHAIN [Bos taurus]\n",
            " 1G0A_B Chain B, HEMOGLOBIN BETA CHAIN [Bos taurus]\n",
            " 1G0A_D Chain D, HEMOGLOBIN BETA CHAIN [Bos taurus]\n",
            " 1HDA_B Chain B, HEMOGLOBIN (DEOXY) (BETA CHAIN) [Bos taurus]\n",
            " 1HDA_D Chain D, HEMOGLOBIN (DEOXY) (BETA CHAIN) [Bos taurus]\n",
            " 2QSP_B Chain B, Hemoglobin subunit beta [Bos taurus]\n",
            " 2QSP_D Chain D, Hemoglobin subunit beta [Bos taurus]\n",
            " 2QSS_B Chain B, Hemoglobin subunit beta [Bos taurus]\n",
            " 2QSS_D Chain D, Hemoglobin subunit beta [Bos taurus]\n",
            " 3CIU_B Chain B, Hemoglobin subunit beta [Bos taurus]\n",
            " 3CIU_D Chain D, Hemoglobin subunit beta [Bos taurus]\n",
            " 3PI8_B Chain B, Hemoglobin subunit beta [Bos taurus]\n",
            " 3PI8_D Chain D, Hemoglobin subunit beta [Bos taurus]\n",
            " 3PI9_B Chain B, Hemoglobin subunit beta [Bos taurus]\n",
            " 3PI9_D Chain D, Hemoglobin subunit beta [Bos taurus]\n",
            " 3PIA_B Chain B, Hemoglobin subunit beta [Bos taurus]\n",
            " 3PIA_D Chain D, Hemoglobin subunit beta [Bos taurus]\n",
            " 6II1_B Chain B, Hemoglobin subunit beta [Bos taurus]\n",
            " 6II1_D Chain D, Hemoglobin subunit beta [Bos taurus]\n",
            "Length=145\n",
            "\n",
            " Score = 234 bits (596),  Expect = 4e-80, Method: Compositional matrix adjust.\n",
            " Identities = 122/144 (85%), Positives = 132/144 (92%), Gaps = 0/144 (0%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKA  63\n",
            "            LT EEK+AVTA WGKV VDEVGGEALGRLLVVYPWTQRFFESFGDLST DAVM NPKVKA\n",
            "Sbjct  2    LTAEEKAAVTAFWGKVKVDEVGGEALGRLLVVYPWTQRFFESFGDLSTADAVMNNPKVKA  61\n",
            "\n",
            "Query  64   HGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEF  123\n",
            "            HGKKVL +FS+G+ HLD+LKGTFA LSELHCDKLHVDPENF+LLGNVLV VLA +FGKEF\n",
            "Sbjct  62   HGKKVLDSFSNGMKHLDDLKGTFAALSELHCDKLHVDPENFKLLGNVLVVVLARNFGKEF  121\n",
            "\n",
            "Query  124  TPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            TP +QA +QKVVAGVANALAH+YH\n",
            "Sbjct  122  TPVLQADFQKVVAGVANALAHRYH  145\n",
            "\n",
            "\n",
            ">1FDH_G Chain G, HEMOGLOBIN F (DEOXY) (GAMMA CHAIN) [Homo sapiens]\n",
            " 1FDH_H Chain H, HEMOGLOBIN F (DEOXY) (GAMMA CHAIN) [Homo sapiens]\n",
            "Length=147\n",
            "\n",
            " Score = 234 bits (596),  Expect = 4e-80, Method: Compositional matrix adjust.\n",
            " Identities = 107/145 (74%), Positives = 127/145 (88%), Gaps = 0/145 (0%)\n",
            "\n",
            "Query  3    HLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVK  62\n",
            "            H T E+K+ +T+LWGKVNV++ GGE LGRLLVVYPWTQRFF+SFG+LS+  A+MGNPKVK\n",
            "Sbjct  3    HFTEEDKATITSLWGKVNVEDAGGETLGRLLVVYPWTQRFFDSFGNLSSASAIMGNPKVK  62\n",
            "\n",
            "Query  63   AHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE  122\n",
            "            AHGKKVL +  D + HLD+LKGTFA LSELHCDKLHVDPENF+LLGNVLV VLA HFGKE\n",
            "Sbjct  63   AHGKKVLTSLGDAIKHLDDLKGTFAQLSELHCDKLHVDPENFKLLGNVLVTVLAIHFGKE  122\n",
            "\n",
            "Query  123  FTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            FTP VQA++QK+V GVA+AL+ +YH\n",
            "Sbjct  123  FTPEVQASWQKMVTGVASALSSRYH  147\n",
            "\n",
            "\n",
            ">4MQK_B Chain B, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            " 4MQK_D Chain D, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            " 4MQK_F Chain F, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            " 4MQK_H Chain H, Hemoglobin subunit gamma-2 [Homo sapiens]\n",
            "Length=146\n",
            "\n",
            " Score = 233 bits (594),  Expect = 1e-79, Method: Compositional matrix adjust.\n",
            " Identities = 106/145 (73%), Positives = 127/145 (88%), Gaps = 0/145 (0%)\n",
            "\n",
            "Query  3    HLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVK  62\n",
            "            H T E+K+ +T+LWGKVNV++ GGE LGRLLVVYPWTQRFF+SFG+LS+  A+MGNPKVK\n",
            "Sbjct  2    HFTEEDKATITSLWGKVNVEDAGGETLGRLLVVYPWTQRFFDSFGNLSSASAIMGNPKVK  61\n",
            "\n",
            "Query  63   AHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE  122\n",
            "            AHGKK+L +  D + HLD+LKGTFA LSELHCDKLHVDPENF+LLGNVLV VLA HFGKE\n",
            "Sbjct  62   AHGKKMLTSLGDAIKHLDDLKGTFAQLSELHCDKLHVDPENFKLLGNVLVTVLAIHFGKE  121\n",
            "\n",
            "Query  123  FTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            FTP VQA++QK+V GVA+AL+ +YH\n",
            "Sbjct  122  FTPEVQASWQKMVTGVASALSSRYH  146\n",
            "\n",
            "\n",
            ">1I3D_A Chain A, HEMOGLOBIN GAMMA CHAINS [Homo sapiens]\n",
            " 1I3D_B Chain B, HEMOGLOBIN GAMMA CHAINS [Homo sapiens]\n",
            " 1I3E_A Chain A, HEMOGLOBIN GAMMA CHAINS [Homo sapiens]\n",
            " 1I3E_B Chain B, HEMOGLOBIN GAMMA CHAINS [Homo sapiens]\n",
            "Length=146\n",
            "\n",
            " Score = 232 bits (591),  Expect = 3e-79, Method: Compositional matrix adjust.\n",
            " Identities = 106/145 (73%), Positives = 126/145 (87%), Gaps = 0/145 (0%)\n",
            "\n",
            "Query  3    HLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVK  62\n",
            "            H T E+K+ +T+LWGKVNV++ GGE LGRLLVVYPWTQRFF+SFG+LS+  A+MGNPKVK\n",
            "Sbjct  2    HFTEEDKATITSLWGKVNVEDAGGETLGRLLVVYPWTQRFFDSFGNLSSASAIMGNPKVK  61\n",
            "\n",
            "Query  63   AHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE  122\n",
            "            AHGKKVL +  D + HLD+LKGTFA LSELHCDKLHVDPENF+LLGNVLV VLA HFGKE\n",
            "Sbjct  62   AHGKKVLTSLGDAIKHLDDLKGTFAQLSELHCDKLHVDPENFKLLGNVLVTVLAIHFGKE  121\n",
            "\n",
            "Query  123  FTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            FTP VQA++QK+V  VA+AL+ +YH\n",
            "Sbjct  122  FTPEVQASWQKMVTAVASALSSRYH  146\n",
            "\n",
            "\n",
            ">6IHX_B Chain B, Hemoglobin subunit beta [Bos taurus]\n",
            " 6IHX_D Chain D, Hemoglobin subunit beta [Bos taurus]\n",
            "Length=144\n",
            "\n",
            " Score = 231 bits (590),  Expect = 3e-79, Method: Compositional matrix adjust.\n",
            " Identities = 121/143 (85%), Positives = 131/143 (92%), Gaps = 0/143 (0%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKA  63\n",
            "            LT EEK+AVTA WGKV VDEVGGEALGRLLVVYPWTQRFFESFGDLST DAVM NPKVKA\n",
            "Sbjct  2    LTAEEKAAVTAFWGKVKVDEVGGEALGRLLVVYPWTQRFFESFGDLSTADAVMNNPKVKA  61\n",
            "\n",
            "Query  64   HGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEF  123\n",
            "            HGKKVL +FS+G+ HLD+LKGTFA LSELHCDKLHVDPENF+LLGNVLV VLA +FGKEF\n",
            "Sbjct  62   HGKKVLDSFSNGMKHLDDLKGTFAALSELHCDKLHVDPENFKLLGNVLVVVLARNFGKEF  121\n",
            "\n",
            "Query  124  TPPVQAAYQKVVAGVANALAHKY  146\n",
            "            TP +QA +QKVVAGVANALAH+Y\n",
            "Sbjct  122  TPVLQADFQKVVAGVANALAHRY  144\n",
            "\n",
            "\n",
            ">1JEB_B Chain B, HEMOGLOBIN BETA-SINGLE CHAIN [Mus musculus]\n",
            " 1JEB_D Chain D, HEMOGLOBIN BETA-SINGLE CHAIN [Mus musculus]\n",
            "Length=146\n",
            "\n",
            " Score = 231 bits (589),  Expect = 6e-79, Method: Compositional matrix adjust.\n",
            " Identities = 117/146 (80%), Positives = 131/146 (90%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT  EK+AV+ LWGKVN DEVGGEALGRLLVVYPWTQR+F+SFGDLS+  A+MGN KV\n",
            "Sbjct  1    VHLTDAEKAAVSGLWGKVNADEVGGEALGRLLVVYPWTQRYFDSFGDLSSASAIMGNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+ AF+DGL HLD+LKGTFA+LSELHCDKLHVDPENFRLLGN++V VL HH GK\n",
            "Sbjct  61   KAHGKKVITAFNDGLNHLDSLKGTFASLSELHCDKLHVDPENFRLLGNMIVIVLGHHLGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAA+QKVVAGVA ALAHKYH\n",
            "Sbjct  121  DFTPAAQAAFQKVVAGVAAALAHKYH  146\n",
            "\n",
            "\n",
            ">3HYU_B Chain B, Hemoglobin subunit beta [Cavia porcellus]\n",
            "Length=146\n",
            "\n",
            " Score = 231 bits (589),  Expect = 6e-79, Method: Compositional matrix adjust.\n",
            " Identities = 110/146 (75%), Positives = 121/146 (83%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT  EKSA+  LWGKVNV E+G EALGRLLVVYPWTQRFFE FGDLS+  A+M N  V\n",
            "Sbjct  1    VHLTAAEKSAILDLWGKVNVGEIGAEALGRLLVVYPWTQRFFEKFGDLSSASAIMSNAHV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K+HG KVL +FS+GL HL +LKGTFA LSELHCDKLHVDPENFRLLGN++V  LAHH   \n",
            "Sbjct  61   KSHGAKVLASFSEGLKHLQDLKGTFAKLSELHCDKLHVDPENFRLLGNMIVIALAHHHPS  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP  QAA+QKV AGVANALAHKYH\n",
            "Sbjct  121  EFTPCTQAAFQKVTAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">1G0B_B Chain B, HEMOGLOBIN BETA CHAIN [Equus caballus]\n",
            " 1IBE_B Chain B, HEMOGLOBIN (DEOXY) [Equus caballus]\n",
            " 1IWH_B Chain B, Hemoglobin beta chain [Equus caballus]\n",
            " 1NS6_B Chain B, Hemoglobin beta subunit [Equus caballus]\n",
            " 1NS9_B Chain B, Hemoglobin beta subunit [Equus caballus]\n",
            " 1S0H_B Chain B, Hemoglobin beta chain [Equus asinus]\n",
            " 1Y8H_B Chain B, Hemoglobin beta chain [Equus caballus]\n",
            " 1Y8H_D Chain D, Hemoglobin beta chain [Equus caballus]\n",
            " 1Y8I_B Chain B, Hemoglobin beta chain [Equus caballus]\n",
            " 1Y8I_D Chain D, Hemoglobin beta chain [Equus caballus]\n",
            " 1Y8K_B Chain B, Hemoglobin beta chain [Equus caballus]\n",
            " 1Y8K_D Chain D, Hemoglobin beta chain [Equus caballus]\n",
            " 2D5X_B Chain B, Hemoglobin beta subunit [Equus caballus]\n",
            " 2MHB_B Chain B, Hemoglobin (aquo Met) (beta Chain) [Equus caballus]\n",
            " 2ZLT_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            " 2ZLU_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            " 2ZLV_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            " 2ZLW_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            " 2ZLW_D Chain D, Hemoglobin subunit beta [Equus caballus]\n",
            " 2ZLX_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            " 2ZLX_D Chain D, Hemoglobin subunit beta [Equus caballus]\n",
            " 5C6E_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            " 6SVA_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            " 8PUQ_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            " 8PUR_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            "Length=146\n",
            "\n",
            " Score = 230 bits (587),  Expect = 1e-78, Method: Compositional matrix adjust.\n",
            " Identities = 122/146 (84%), Positives = 132/146 (90%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V L+ EEK+AV ALW KVN +EVGGEALGRLLVVYPWTQRFF+SFGDLS P AVMGNPKV\n",
            "Sbjct  1    VQLSGEEKAAVLALWDKVNEEEVGGEALGRLLVVYPWTQRFFDSFGDLSNPGAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +F +G+ HLDNLKGTFA LSELHCDKLHVDPENFRLLGNVLV VLA HFGK\n",
            "Sbjct  61   KAHGKKVLHSFGEGVHHLDNLKGTFAALSELHCDKLHVDPENFRLLGNVLVVVLARHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP +QA+YQKVVAGVANALAHKYH\n",
            "Sbjct  121  DFTPELQASYQKVVAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">3A0G_B Chain B, Hemoglobin subunit beta [Cavia porcellus]\n",
            "Length=146\n",
            "\n",
            " Score = 229 bits (585),  Expect = 2e-78, Method: Compositional matrix adjust.\n",
            " Identities = 109/146 (75%), Positives = 120/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VHLT  EKSA+  LWGKVNV E+G EALGRLLVVYPWTQRFFE FGDLS+  A+M N  V\n",
            "Sbjct  1    VHLTAAEKSAILDLWGKVNVGEIGAEALGRLLVVYPWTQRFFEKFGDLSSASAIMSNAHV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K+HG KVL +FS+GL HL +LKGTFA LSELHCDKLHVDPENFRLLGN+L   +AHH   \n",
            "Sbjct  61   KSHGAKVLASFSEGLKHLQDLKGTFAKLSELHCDKLHVDPENFRLLGNMLTIAIAHHHPS  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP  QAA+QKV AGVANALAHKYH\n",
            "Sbjct  121  EFTPCTQAAFQKVTAGVANALAHKYH  146\n",
            "\n",
            "\n",
            ">6R2O_B Chain B, Hemoglobin subunit beta [Equus caballus]\n",
            " 6R2O_D Chain D, Hemoglobin subunit beta [Equus caballus]\n",
            "Length=145\n",
            "\n",
            " Score = 228 bits (581),  Expect = 1e-77, Method: Compositional matrix adjust.\n",
            " Identities = 121/145 (83%), Positives = 131/145 (90%), Gaps = 0/145 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V L+ EEK+AV ALW KVN +EVGGEALGRLLVVYPWTQRFF+SFGDLS P AVMGNPKV\n",
            "Sbjct  1    VQLSGEEKAAVLALWDKVNEEEVGGEALGRLLVVYPWTQRFFDSFGDLSNPGAVMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +F +G+ HLDNLKGTFA LSELHCDKLHVDPENFRLLGNVLV VLA HFGK\n",
            "Sbjct  61   KAHGKKVLHSFGEGVHHLDNLKGTFAALSELHCDKLHVDPENFRLLGNVLVVVLARHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP +QA+YQKVVAGVANALAHKY\n",
            "Sbjct  121  DFTPELQASYQKVVAGVANALAHKY  145\n",
            "\n",
            "\n",
            ">2QU0_B Chain B, Hemoglobin subunit beta [Ovis aries]\n",
            " 2QU0_D Chain D, Hemoglobin subunit beta [Ovis aries]\n",
            "Length=145\n",
            "\n",
            " Score = 224 bits (572),  Expect = 2e-76, Method: Compositional matrix adjust.\n",
            " Identities = 119/144 (83%), Positives = 126/144 (88%), Gaps = 0/144 (0%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKA  63\n",
            "            LT EEK+AVT  WGKV VDEVG EALGRLLVVYPWTQRFFE FGDLS  DAVM NPKVKA\n",
            "Sbjct  2    LTAEEKAAVTGFWGKVKVDEVGAEALGRLLVVYPWTQRFFEHFGDLSNADAVMNNPKVKA  61\n",
            "\n",
            "Query  64   HGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEF  123\n",
            "            HGKKVL +FS+G+ HLD+LKGTFA LSELHCDKLHVDPENFRLLGNVLV VLA H G EF\n",
            "Sbjct  62   HGKKVLDSFSNGMKHLDDLKGTFAQLSELHCDKLHVDPENFRLLGNVLVVVLARHHGNEF  121\n",
            "\n",
            "Query  124  TPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            TP +QA +QKVVAGVANALAHKYH\n",
            "Sbjct  122  TPVLQADFQKVVAGVANALAHKYH  145\n",
            "\n",
            "\n",
            ">1HDS_B Chain B, HEMOGLOBIN S (DEOXY) (BETA CHAIN) [Odocoileus virginianus]\n",
            " 1HDS_D Chain D, HEMOGLOBIN S (DEOXY) (BETA CHAIN) [Odocoileus virginianus]\n",
            "Length=145\n",
            "\n",
            " Score = 224 bits (570),  Expect = 4e-76, Method: Compositional matrix adjust.\n",
            " Identities = 107/144 (74%), Positives = 123/144 (85%), Gaps = 0/144 (0%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKA  63\n",
            "            LT EEK+AVT  WGKV+VD VG +ALGRLLVVYPWTQRFF+ FG+LS+  AVM NPKVKA\n",
            "Sbjct  2    LTAEEKAAVTGFWGKVDVDVVGAQALGRLLVVYPWTQRFFQHFGNLSSAGAVMNNPKVKA  61\n",
            "\n",
            "Query  64   HGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEF  123\n",
            "            HGK+VL AF+ GL HLD+LKG FA LS LHC+KLHV+P+NFRLLGNVL  V+A +FG +F\n",
            "Sbjct  62   HGKRVLDAFTQGLKHLDDLKGAFAQLSGLHCNKLHVNPQNFRLLGNVLALVVARNFGGQF  121\n",
            "\n",
            "Query  124  TPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            TP VQA +QKVVAGVANALAHKYH\n",
            "Sbjct  122  TPNVQALFQKVVAGVANALAHKYH  145\n",
            "\n",
            "\n",
            ">2RI4_B Chain B, Hemoglobin subunit beta-A [Capra hircus]\n",
            " 2RI4_D Chain D, Hemoglobin subunit beta-A [Capra hircus]\n",
            " 2RI4_J Chain J, Hemoglobin subunit beta-A [Capra hircus]\n",
            " 2RI4_L Chain L, Hemoglobin subunit beta-A [Capra hircus]\n",
            " 3D1A_B Chain B, Hemoglobin subunit beta-A [Capra hircus]\n",
            " 3D1A_D Chain D, Hemoglobin subunit beta-A [Capra hircus]\n",
            " 3EU1_B Chain B, Hemoglobin subunit beta-A [Capra hircus]\n",
            " 3EU1_D Chain D, Hemoglobin subunit beta-A [Capra hircus]\n",
            "Length=145\n",
            "\n",
            " Score = 221 bits (563),  Expect = 5e-75, Method: Compositional matrix adjust.\n",
            " Identities = 116/144 (81%), Positives = 126/144 (88%), Gaps = 0/144 (0%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKA  63\n",
            "            LT EEK+AVT  WGKV VDEVG EALGRLLVVYPWTQRFFE FGDLS+ DAVM N KVKA\n",
            "Sbjct  2    LTAEEKAAVTGFWGKVKVDEVGAEALGRLLVVYPWTQRFFEHFGDLSSADAVMNNAKVKA  61\n",
            "\n",
            "Query  64   HGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEF  123\n",
            "            HGKKVL +FS+G+ HLD+LKGTFA LSELHCDKLHVDPENF+LLGNVLV VLA H G EF\n",
            "Sbjct  62   HGKKVLDSFSNGMKHLDDLKGTFAQLSELHCDKLHVDPENFKLLGNVLVVVLARHHGSEF  121\n",
            "\n",
            "Query  124  TPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            TP +QA +QKVVAGVANALAH+YH\n",
            "Sbjct  122  TPLLQAEFQKVVAGVANALAHRYH  145\n",
            "\n",
            "\n",
            ">1FAW_B Chain B, HEMOGLOBIN (BETA SUBUNIT) [Anser anser]\n",
            " 1FAW_D Chain D, HEMOGLOBIN (BETA SUBUNIT) [Anser anser]\n",
            "Length=146\n",
            "\n",
            " Score = 219 bits (558),  Expect = 3e-74, Method: Compositional matrix adjust.\n",
            " Identities = 101/146 (69%), Positives = 120/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH + EEK  +T LWGKVNV + G EAL RLL+VYPWTQRFF SFG+LS+P A++GNP V\n",
            "Sbjct  1    VHWSAEEKQLITGLWGKVNVADCGAEALARLLIVYPWTQRFFSSFGNLSSPTAILGNPMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F D + +LDN+K TFA LSELHCDKLHVDPENFRLLG++L+ VLA HF K\n",
            "Sbjct  61   RAHGKKVLTSFGDAVKNLDNIKNTFAQLSELHCDKLHVDPENFRLLGDILIIVLAAHFAK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  EFTPECQAAWQKLVRVVAHALARKYH  146\n",
            "\n",
            "\n",
            ">1A4F_B Chain B, HEMOGLOBIN (BETA CHAIN) [Anser indicus]\n",
            " 1C40_B Chain B, PROTEIN (HEMOGLOBIN (BETA CHAIN)) [Anser indicus]\n",
            " 1HV4_B Chain B, HEMOGLOBIN BETA CHAIN [Anser indicus]\n",
            " 1HV4_D Chain D, HEMOGLOBIN BETA CHAIN [Anser indicus]\n",
            " 1HV4_F Chain F, HEMOGLOBIN BETA CHAIN [Anser indicus]\n",
            " 1HV4_H Chain H, HEMOGLOBIN BETA CHAIN [Anser indicus]\n",
            "Length=146\n",
            "\n",
            " Score = 219 bits (558),  Expect = 3e-74, Method: Compositional matrix adjust.\n",
            " Identities = 101/146 (69%), Positives = 120/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH + EEK  +T LWGKVNV + G EAL RLL+VYPWTQRFF SFG+LS+P A++GNP V\n",
            "Sbjct  1    VHWSAEEKQLITGLWGKVNVADCGAEALARLLIVYPWTQRFFSSFGNLSSPTAILGNPMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F D + +LDN+K TFA LSELHCDKLHVDPENFRLLG++L+ VLA HF K\n",
            "Sbjct  61   RAHGKKVLTSFGDAVKNLDNIKNTFAQLSELHCDKLHVDPENFRLLGDILIIVLAAHFAK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  EFTPDCQAAWQKLVRVVAHALARKYH  146\n",
            "\n",
            "\n",
            ">1HBR_B Chain B, PROTEIN (HEMOGLOBIN D) [Gallus gallus]\n",
            " 1HBR_D Chain D, PROTEIN (HEMOGLOBIN D) [Gallus gallus]\n",
            "Length=146\n",
            "\n",
            " Score = 219 bits (558),  Expect = 3e-74, Method: Compositional matrix adjust.\n",
            " Identities = 101/146 (69%), Positives = 120/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH T EEK  +T LWGKVNV E G EAL RLL+VYPWTQRFF SFG+LS+P A++GNP V\n",
            "Sbjct  1    VHWTAEEKQLITGLWGKVNVAECGAEALARLLIVYPWTQRFFASFGNLSSPTAILGNPMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F D + +LDN+K TF+ LSELHCDKLHVDPENFRLLG++L+ VLA HF K\n",
            "Sbjct  61   RAHGKKVLTSFGDAVKNLDNIKNTFSQLSELHCDKLHVDPENFRLLGDILIIVLAAHFSK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  DFTPECQAAWQKLVRVVAHALARKYH  146\n",
            "\n",
            "\n",
            ">2R80_B Chain B, Hemoglobin subunit beta [Columba livia]\n",
            " 2R80_D Chain D, Hemoglobin subunit beta [Columba livia]\n",
            " 3DHR_B Chain B, Hemoglobin subunit beta [Columba livia]\n",
            " 3DHR_D Chain D, Hemoglobin subunit beta [Columba livia]\n",
            " 3DHR_F Chain F, Hemoglobin subunit beta [Columba livia]\n",
            " 3DHR_H Chain H, Hemoglobin subunit beta [Columba livia]\n",
            " 3MJU_B Chain B, Hemoglobin subunit beta [Columba livia]\n",
            "Length=146\n",
            "\n",
            " Score = 219 bits (558),  Expect = 4e-74, Method: Compositional matrix adjust.\n",
            " Identities = 101/146 (69%), Positives = 121/146 (83%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH + EEK  +T++WGKVNV + G EAL RLL+VYPWTQRFF SFG+LS+  A+ GNP V\n",
            "Sbjct  1    VHWSAEEKQLITSIWGKVNVADCGAEALARLLIVYPWTQRFFSSFGNLSSATAISGNPNV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKVL +F D + +LDN+KGTFA LSELHCDKLHVDPENFRLLG++LV +LA HFGK\n",
            "Sbjct  61   KAHGKKVLTSFGDAVKNLDNIKGTFAQLSELHCDKLHVDPENFRLLGDILVIILAAHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  DFTPECQAAWQKLVRVVAHALARKYH  146\n",
            "\n",
            "\n",
            ">3EOK_B Chain B, Hemoglobin subunit beta [Anas platyrhynchos]\n",
            "Length=146\n",
            "\n",
            " Score = 218 bits (555),  Expect = 8e-74, Method: Compositional matrix adjust.\n",
            " Identities = 101/146 (69%), Positives = 120/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH T EEK  +T LWGKVNV + G EAL RLL+VYPWTQRFF SFG+LS+P A++GNP V\n",
            "Sbjct  1    VHWTAEEKQLITGLWGKVNVADCGAEALARLLIVYPWTQRFFASFGNLSSPTAILGNPMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F D + +LDN+K TFA LSELHCDKLHVDPENFRLLG++L+ VLA HF K\n",
            "Sbjct  61   RAHGKKVLTSFGDAVKNLDNIKNTFAQLSELHCDKLHVDPENFRLLGDILIIVLAAHFTK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  DFTPECQAAWQKLVRVVAHALARKYH  146\n",
            "\n",
            "\n",
            ">2ZFB_B Chain B, Hemoglobin subunit beta [Psittacula krameri]\n",
            "Length=146\n",
            "\n",
            " Score = 218 bits (555),  Expect = 1e-73, Method: Compositional matrix adjust.\n",
            " Identities = 101/146 (69%), Positives = 120/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH + EEK  +T LWGKVNV E G EAL RLL+VYPWTQRFF SFG+LS+  AV+GNP V\n",
            "Sbjct  1    VHWSAEEKQLITGLWGKVNVAECGAEALARLLIVYPWTQRFFTSFGNLSSASAVLGNPNV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F + + +LDN+K TFA LSELHCDKLHVDPENFRLLG++L+ VLA HFGK\n",
            "Sbjct  61   RAHGKKVLTSFGEAVKNLDNIKNTFAQLSELHCDKLHVDPENFRLLGDILIIVLAGHFGK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  DFTPDCQAAWQKLVRAVAHALARKYH  146\n",
            "\n",
            "\n",
            ">3MJP_B Chain B, Hemoglobin subunit beta [Coturnix japonica]\n",
            " 3MJP_D Chain D, Hemoglobin subunit beta [Coturnix japonica]\n",
            "Length=146\n",
            "\n",
            " Score = 217 bits (553),  Expect = 2e-73, Method: Compositional matrix adjust.\n",
            " Identities = 100/146 (68%), Positives = 120/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH + EEK  +T LWGKVNV E G EAL RLL+VYPWTQRFF SFG+LS+P A++GNP V\n",
            "Sbjct  1    VHWSAEEKQLITGLWGKVNVAECGAEALARLLIVYPWTQRFFASFGNLSSPTAILGNPMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F D + +LDN+K TF+ LSELHCDKLHVDPENFRLLG++L+ VLA HF K\n",
            "Sbjct  61   RAHGKKVLTSFGDAVKNLDNIKNTFSQLSELHCDKLHVDPENFRLLGDILIIVLAAHFTK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  DFTPECQAAWQKLVRVVAHALARKYH  146\n",
            "\n",
            "\n",
            ">3WR1_B Chain B, Hemoglobin subunit beta [Phalacrocorax carbo]\n",
            "Length=146\n",
            "\n",
            " Score = 216 bits (551),  Expect = 3e-73, Method: Compositional matrix adjust.\n",
            " Identities = 100/146 (68%), Positives = 118/146 (81%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH T EEK  +T LWGKVNV E G EAL RLL+VYPWTQRFF SFG+LS+  A+ GNP V\n",
            "Sbjct  1    VHWTAEEKQLITGLWGKVNVAECGAEALARLLIVYPWTQRFFASFGNLSSATAITGNPMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F + + +LDN+K TFA LSELHCDKLHVDPENFRLLG++L+ VLA HF K\n",
            "Sbjct  61   RAHGKKVLTSFGEAVKNLDNIKATFAQLSELHCDKLHVDPENFRLLGDILIIVLAAHFAK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  DFTPECQAAWQKLVGAVAHALARKYH  146\n",
            "\n",
            "\n",
            ">3WTG_B Chain B, Hemoglobin [Dromaius novaehollandiae]\n",
            " 3WTG_D Chain D, Hemoglobin [Dromaius novaehollandiae]\n",
            "Length=146\n",
            "\n",
            " Score = 216 bits (551),  Expect = 4e-73, Method: Compositional matrix adjust.\n",
            " Identities = 99/146 (68%), Positives = 119/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  + EEK  +++LWGKVNV E G EAL RLL+VYPWTQRFF SFG+LS+  A++GNP V\n",
            "Sbjct  1    VQWSAEEKQLISSLWGKVNVAECGAEALARLLIVYPWTQRFFTSFGNLSSASAIIGNPMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F D + +LDN+K TFA LSELHCDKLHVDPENFRLLG++L+ VLA HF K\n",
            "Sbjct  61   RAHGKKVLTSFGDAVKNLDNIKNTFAQLSELHCDKLHVDPENFRLLGDILIIVLAAHFAK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  EFTPECQAAWQKLVRVVAHALARKYH  146\n",
            "\n",
            "\n",
            ">2QMB_B Chain B, Hemoglobin beta chain [Meleagris gallopavo]\n",
            " 2QMB_D Chain D, Hemoglobin beta chain [Meleagris gallopavo]\n",
            " 3K8B_B Chain B, Hemoglobin beta chain [Meleagris gallopavo]\n",
            " 3K8B_D Chain D, Hemoglobin beta chain [Meleagris gallopavo]\n",
            " 6ZMX_B Chain B, Hemoglobin beta chain [Meleagris gallopavo]\n",
            " 6ZMX_D Chain D, Hemoglobin beta chain [Meleagris gallopavo]\n",
            " 6ZMY_B Chain B, Hemoglobin beta chain [Meleagris gallopavo]\n",
            " 6ZMY_D Chain D, Hemoglobin beta chain [Meleagris gallopavo]\n",
            "Length=146\n",
            "\n",
            " Score = 216 bits (550),  Expect = 4e-73, Method: Compositional matrix adjust.\n",
            " Identities = 99/146 (68%), Positives = 120/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH + EEK  +T LWGKVNV + G EAL RLL+VYPWTQRFF SFG+LS+P A++GNP V\n",
            "Sbjct  1    VHWSAEEKQLITGLWGKVNVADCGAEALARLLIVYPWTQRFFASFGNLSSPTAILGNPMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F D + +LDN+K TF+ LSELHCDKLHVDPENFRLLG++L+ VLA HF K\n",
            "Sbjct  61   RAHGKKVLTSFGDAVKNLDNIKNTFSQLSELHCDKLHVDPENFRLLGDILIIVLAAHFSK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  DFTPECQAAWQKLVRVVAHALARKYH  146\n",
            "\n",
            "\n",
            ">3A59_B Chain B, Hemoglobin subunit beta [Struthio camelus]\n",
            " 3A59_D Chain D, Hemoglobin subunit beta [Struthio camelus]\n",
            " 3A59_F Chain F, Hemoglobin subunit beta [Struthio camelus]\n",
            " 3A59_H Chain H, Hemoglobin subunit beta [Struthio camelus]\n",
            " 3FS4_B Chain B, Hemoglobin subunit beta [Struthio camelus]\n",
            " 3FS4_D Chain D, Hemoglobin subunit beta [Struthio camelus]\n",
            "Length=146\n",
            "\n",
            " Score = 215 bits (547),  Expect = 1e-72, Method: Compositional matrix adjust.\n",
            " Identities = 99/146 (68%), Positives = 119/146 (82%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  + EEK  ++ LWGKVNV + G EAL RLL+VYPWTQRFF SFG+LS+P A++GNP V\n",
            "Sbjct  1    VQWSAEEKQLISGLWGKVNVADCGAEALARLLIVYPWTQRFFASFGNLSSPTAILGNPMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHGKKVL +F D + +LDN+K TFA LSELHCDKLHVDPENFRLLG++L+ VLA HF K\n",
            "Sbjct  61   RAHGKKVLTSFGDAVKNLDNIKNTFAQLSELHCDKLHVDPENFRLLGDILIIVLAAHFTK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP  QAA+QK+V  VA+ALA KYH\n",
            "Sbjct  121  EFTPECQAAWQKLVRVVAHALARKYH  146\n",
            "\n",
            "\n",
            ">1V75_B Chain B, Hemoglobin A and D beta chain [Aldabrachelys gigantea]\n",
            " 1WMU_B Chain B, Hemoglobin A and D beta chain [Aldabrachelys gigantea]\n",
            " 2Z6N_B Chain B, Hemoglobin A/D subunit beta [Aldabrachelys gigantea]\n",
            "Length=146\n",
            "\n",
            " Score = 206 bits (523),  Expect = 6e-69, Method: Compositional matrix adjust.\n",
            " Identities = 97/146 (66%), Positives = 118/146 (81%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH T EEK  +T+LW KVNV EVGGEAL RLL+VYPWTQRFF SFG+LS+ +A++ N KV\n",
            "Sbjct  1    VHWTSEEKQYITSLWAKVNVGEVGGEALARLLIVYPWTQRFFASFGNLSSANAILHNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHG+KVL +F + + +LDN+K TFA LSELHC+KLHVDPENF+LLGN+L+ VLA HF K\n",
            "Sbjct  61   LAHGQKVLTSFGEAVKNLDNIKKTFAQLSELHCEKLHVDPENFKLLGNILIIVLATHFPK  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP  QAA+ K+V  VA+ALA  YH\n",
            "Sbjct  121  EFTPASQAAWTKLVNAVAHALALGYH  146\n",
            "\n",
            "\n",
            ">3AT5_B Chain B, Beta-globin [Podocnemis unifilis]\n",
            " 3AT6_B Chain B, Beta-globin [Podocnemis unifilis]\n",
            "Length=146\n",
            "\n",
            " Score = 203 bits (517),  Expect = 5e-68, Method: Compositional matrix adjust.\n",
            " Identities = 91/144 (63%), Positives = 117/144 (81%), Gaps = 0/144 (0%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKA  63\n",
            "             T EE+  +  LWG+V+V+++G EAL RLL+VYPWTQRFF SFG+LS+P A++ N KV A\n",
            "Sbjct  3    FTQEERQFIVNLWGRVDVEQIGAEALARLLIVYPWTQRFFSSFGNLSSPSAILHNAKVHA  62\n",
            "\n",
            "Query  64   HGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEF  123\n",
            "            HGKKVL +F + + +LD +K TFA LSELH DKLHVDPENF+LLGN+L+ VLA HFGK+F\n",
            "Sbjct  63   HGKKVLTSFGEAVKNLDQIKQTFAQLSELHSDKLHVDPENFKLLGNILIIVLAAHFGKDF  122\n",
            "\n",
            "Query  124  TPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            TP  QAA+QK+V+ VA+ALA +YH\n",
            "Sbjct  123  TPASQAAWQKLVSAVAHALALRYH  146\n",
            "\n",
            "\n",
            ">1OUT_B Chain B, Hemoglobin I [Oncorhynchus mykiss]\n",
            " 1OUU_B Chain B, Hemoglobin I [Oncorhynchus mykiss]\n",
            " 1OUU_D Chain D, Hemoglobin I [Oncorhynchus mykiss]\n",
            "Length=146\n",
            "\n",
            " Score = 172 bits (437),  Expect = 8e-56, Method: Compositional matrix adjust.\n",
            " Identities = 77/145 (53%), Positives = 104/145 (72%), Gaps = 0/145 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T  EKS ++A+WGKVN+DE+G  AL R+L+VYPWTQR+F SFG++STP A+MGNPKV\n",
            "Sbjct  1    VEWTDAEKSTISAVWGKVNIDEIGPLALARVLIVYPWTQRYFGSFGNVSTPAAIMGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHGK V GA    + ++ N+  T+ +LSE H +KL VDP+NFR+L +VL  V+A  FG \n",
            "Sbjct  61   AAHGKVVCGALDKAVKNMGNILATYKSLSETHANKLFVDPDNFRVLADVLTIVIAAKFGA  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "             FTP +QA +QK +  V  A+  +Y\n",
            "Sbjct  121  SFTPEIQATWQKFMKVVVAAMGSRY  145\n",
            "\n",
            "\n",
            ">3BCQ_B Chain B, Beta-chain hemoglobin [Brycon cephalus]\n",
            " 3BCQ_D Chain D, Beta-chain hemoglobin [Brycon cephalus]\n",
            "Length=146\n",
            "\n",
            " Score = 161 bits (408),  Expect = 2e-51, Method: Compositional matrix adjust.\n",
            " Identities = 75/147 (51%), Positives = 105/147 (71%), Gaps = 2/147 (1%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  +  E+SA+  LWGK++VDE+G +AL RLL+VYPWTQR F +FG+LS+P A+ GNPKV\n",
            "Sbjct  1    VEWSTAERSAIAGLWGKISVDEIGPQALSRLLIVYPWTQRHFAAFGNLSSPAAINGNPKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG-  120\n",
            "              HGK V+G     + ++DN+K  +++LS +H +KLHVDP+NFRLL + +   +A  FG \n",
            "Sbjct  61   AHHGKVVMGGLERAIKNMDNIKAAYSSLSVMHSEKLHVDPDNFRLLADCITVCVAMKFGP  120\n",
            "\n",
            "Query  121  KEFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "              FTP VQ A+QK +A V  AL+ +YH\n",
            "Sbjct  121  SAFTPDVQEAWQKFLAVVVAALS-RYH  146\n",
            "\n",
            "\n",
            ">6IYH_B Chain B, Beta chain [Acipenser persicus]\n",
            "Length=145\n",
            "\n",
            " Score = 161 bits (407),  Expect = 3e-51, Method: Compositional matrix adjust.\n",
            " Identities = 73/143 (51%), Positives = 101/143 (71%), Gaps = 0/143 (0%)\n",
            "\n",
            "Query  5    TPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAH  64\n",
            "            T  E+ A+T LW KVNV+ VG +AL RLLVVYPWTQR+F +FG++S   A+ GN +V AH\n",
            "Sbjct  3    TDSERFAITTLWAKVNVESVGAQALVRLLVVYPWTQRYFGAFGNISDAAAIAGNAQVHAH  62\n",
            "\n",
            "Query  65   GKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEFT  124\n",
            "            GK VL +  + +AH+D++   F  LS  H + LHVDP+NF+  G+ L  VLA  FG  +T\n",
            "Sbjct  63   GKTVLDSVGNAIAHMDDVADAFTALSTFHSETLHVDPDNFQHFGDCLSIVLAATFGTAYT  122\n",
            "\n",
            "Query  125  PPVQAAYQKVVAGVANALAHKYH  147\n",
            "            P VQAA+QK++A + +AL+ +YH\n",
            "Sbjct  123  PDVQAAWQKMIAVIISALSKEYH  145\n",
            "\n",
            "\n",
            ">6IYI_B Chain B, Beta chain [Acipenser stellatus]\n",
            "Length=146\n",
            "\n",
            " Score = 160 bits (406),  Expect = 4e-51, Method: Compositional matrix adjust.\n",
            " Identities = 75/146 (51%), Positives = 100/146 (68%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T  E+ A+T LW KV+V+ VG +AL RLLVVYPWTQR+F +FG++S   A+ GN KV\n",
            "Sbjct  1    VKWTDSERFAITTLWAKVDVERVGAQALVRLLVVYPWTQRYFGAFGNISDAAAIAGNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHGK VL +    +AH+D+L G F  LS  H + LHVDP+NF   G+ L  VLA  FG \n",
            "Sbjct  61   HAHGKTVLSSVGIAIAHMDDLAGAFTALSAFHSETLHVDPDNFEHFGDCLSIVLAATFGT  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             +TP V AA+QK++A + +AL+ +YH\n",
            "Sbjct  121  AYTPDVHAAWQKMIAVIISALSKEYH  146\n",
            "\n",
            "\n",
            ">1V4U_B Chain B, hemoglobin beta chain [Thunnus thynnus]\n",
            " 1V4U_D Chain D, hemoglobin beta chain [Thunnus thynnus]\n",
            " 1V4W_B Chain B, hemoglobin beta chain [Thunnus thynnus]\n",
            " 1V4W_D Chain D, hemoglobin beta chain [Thunnus thynnus]\n",
            " 1V4X_B Chain B, hemoglobin beta chain [Thunnus thynnus]\n",
            " 1V4X_D Chain D, hemoglobin beta chain [Thunnus thynnus]\n",
            "Length=146\n",
            "\n",
            " Score = 155 bits (391),  Expect = 8e-49, Method: Compositional matrix adjust.\n",
            " Identities = 70/146 (48%), Positives = 99/146 (68%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T +E+S +  ++  +N +++G +AL R L+VYPWTQR+F ++GDLSTPDA+ GN K+\n",
            "Sbjct  1    VEWTQQERSIIAGIFANLNYEDIGPKALARCLIVYPWTQRYFGAYGDLSTPDAIKGNAKI  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHG KVL      + ++DN+   ++ LS LH DKLHVDP+NFR+LG+ L  V+A + G \n",
            "Sbjct  61   AAHGVKVLHGLDRAVKNMDNINEAYSELSVLHSDKLHVDPDNFRILGDCLTVVIAANLGD  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             FT   Q A+QK +A V  AL  KYH\n",
            "Sbjct  121  AFTVETQCAFQKFLAVVVFALGRKYH  146\n",
            "\n",
            "\n",
            ">2R1H_B Chain B, Hemoglobin subunit beta-4 [Oncorhynchus mykiss]\n",
            " 2R1H_D Chain D, Hemoglobin subunit beta-4 [Oncorhynchus mykiss]\n",
            " 3BOM_B Chain B, Hemoglobin subunit beta-4 [Oncorhynchus mykiss]\n",
            " 3BOM_D Chain D, Hemoglobin subunit beta-4 [Oncorhynchus mykiss]\n",
            "Length=147\n",
            "\n",
            " Score = 154 bits (390),  Expect = 1e-48, Method: Compositional matrix adjust.\n",
            " Identities = 74/147 (50%), Positives = 100/147 (68%), Gaps = 1/147 (1%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T  E+SA+  LWGK++VDE+G +AL RLL+V PWTQR F +FG+LSTP A+MGNP V\n",
            "Sbjct  1    VDWTDAERSAIVGLWGKISVDEIGPQALARLLIVSPWTQRHFSTFGNLSTPAAIMGNPAV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGN-VLVCVLAHHFG  120\n",
            "              HGK V+      + +LD++K T+ TLS +H +KL VDP+NFRLL + + VCV A    \n",
            "Sbjct  61   AKHGKTVMHGLDRAVQNLDDIKNTYVTLSVMHSEKLFVDPDNFRLLADCITVCVAAKLGP  120\n",
            "\n",
            "Query  121  KEFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "              F+   Q A+QK +A V +AL  +YH\n",
            "Sbjct  121  AVFSADTQEAFQKFLAVVVSALGRQYH  147\n",
            "\n",
            "\n",
            ">1SPG_B Chain B, Hemoglobin [Leiostomus xanthurus]\n",
            "Length=147\n",
            "\n",
            " Score = 154 bits (389),  Expect = 2e-48, Method: Compositional matrix adjust.\n",
            " Identities = 70/147 (48%), Positives = 100/147 (68%), Gaps = 1/147 (1%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T  E++A+ ALWGK++V E+G +AL RLL+VYPWTQR F+ FG++ST  A++GN KV\n",
            "Sbjct  1    VDWTDAERAAIKALWGKIDVGEIGPQALSRLLIVYPWTQRHFKGFGNISTNAAILGNAKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG-  120\n",
            "              HGK V+G     + ++DN+K  +  LS  H +K+HVDP+NFRLLG ++   +   FG \n",
            "Sbjct  61   AEHGKTVMGGLDRAVQNMDNIKNVYKQLSIKHSEKIHVDPDNFRLLGEIITMCVGAKFGP  120\n",
            "\n",
            "Query  121  KEFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "              FTP +  A+QK +A V +AL  +YH\n",
            "Sbjct  121  SAFTPEIHEAWQKFLAVVVSALGRQYH  147\n",
            "\n",
            "\n",
            ">4ESA_B Chain B, Hemoglobin beta chain [Eleginops maclovinus]\n",
            " 4ESA_D Chain D, Hemoglobin beta chain [Eleginops maclovinus]\n",
            "Length=146\n",
            "\n",
            " Score = 146 bits (368),  Expect = 3e-45, Method: Compositional matrix adjust.\n",
            " Identities = 67/146 (46%), Positives = 99/146 (68%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T +E++ +++++G ++ D++G +AL R L+VYPWTQR F SFG+L   +A++GN KV\n",
            "Sbjct  1    VEWTDQERATISSIFGSLDYDDIGPKALSRCLIVYPWTQRHFGSFGNLYNAEAIIGNQKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHG KVL      + ++DN+K  +A LS LH +KLHVDP+NF+LL + L  V+A   G \n",
            "Sbjct  61   AAHGIKVLHGLDRAVKNMDNIKEIYAELSILHSEKLHVDPDNFKLLADCLTIVVAAKMGS  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             F P  QA +QK +A V +AL  +YH\n",
            "Sbjct  121  GFNPGTQATFQKFLAVVVSALGKQYH  146\n",
            "\n",
            "\n",
            ">1XQ5_B Chain B, Hemoglobin beta-2 chain [Perca flavescens]\n",
            " 1XQ5_D Chain D, Hemoglobin beta-2 chain [Perca flavescens]\n",
            " 3BJ1_B Chain B, hemoglobin beta [Perca flavescens]\n",
            " 3BJ1_D Chain D, hemoglobin beta [Perca flavescens]\n",
            " 3BJ2_B Chain B, hemoglobin beta [Perca flavescens]\n",
            " 3BJ2_D Chain D, hemoglobin beta [Perca flavescens]\n",
            " 3BJ3_B Chain B, hemoglobin beta [Perca flavescens]\n",
            " 3BJ3_D Chain D, hemoglobin beta [Perca flavescens]\n",
            "Length=146\n",
            "\n",
            " Score = 143 bits (361),  Expect = 2e-44, Method: Compositional matrix adjust.\n",
            " Identities = 64/143 (45%), Positives = 95/143 (66%), Gaps = 0/143 (0%)\n",
            "\n",
            "Query  5    TPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAH  64\n",
            "            T  E++ +  ++ K++ + VGG  L R L+VYPWTQR+F +FG+L    A+MGNP +  H\n",
            "Sbjct  4    TDFERATIADIFSKLDYEAVGGATLARCLIVYPWTQRYFGNFGNLYNAAAIMGNPMIAKH  63\n",
            "\n",
            "Query  65   GKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEFT  124\n",
            "            G  +L      + ++DN+K T+A LS LH +KLHVDP+NF+LL + L  V+A   GK F+\n",
            "Sbjct  64   GTTILHGLDRAVKNMDNIKATYAELSVLHSEKLHVDPDNFKLLSDCLTIVVAAQLGKAFS  123\n",
            "\n",
            "Query  125  PPVQAAYQKVVAGVANALAHKYH  147\n",
            "              VQAA+QK ++ V +AL  +YH\n",
            "Sbjct  124  GEVQAAFQKFLSVVVSALGKQYH  146\n",
            "\n",
            "\n",
            ">1HBH_B Chain B, HEMOGLOBIN (DEOXY) (BETA CHAIN) [Trematomus bernacchii]\n",
            " 1HBH_D Chain D, HEMOGLOBIN (DEOXY) (BETA CHAIN) [Trematomus bernacchii]\n",
            " 1PBX_B Chain B, Hemoglobin (carbonmonoxy) (beta Chain) [Trematomus bernacchii]\n",
            " 1S5X_B Chain B, Hemoglobin beta chain [Trematomus bernacchii]\n",
            " 1S5Y_B Chain B, Hemoglobin beta chain [Trematomus bernacchii]\n",
            " 1S5Y_D Chain D, Hemoglobin beta chain [Trematomus bernacchii]\n",
            " 2H8D_B Chain B, Hemoglobin beta subunit [Trematomus bernacchii]\n",
            " 2H8D_D Chain D, Hemoglobin beta subunit [Trematomus bernacchii]\n",
            " 2H8F_B Chain B, Hemoglobin beta subunit [Trematomus bernacchii]\n",
            " 2H8F_D Chain D, Hemoglobin beta subunit [Trematomus bernacchii]\n",
            " 2PEG_B Chain B, Hemoglobin subunit beta [Trematomus bernacchii]\n",
            " 3GKV_B Chain B, Hemoglobin subunit beta [Trematomus bernacchii]\n",
            " 3GQG_B Chain B, Hemoglobin subunit beta [Trematomus bernacchii]\n",
            " 3GQG_D Chain D, Hemoglobin subunit beta [Trematomus bernacchii]\n",
            " 4G51_B Chain B, Hemoglobin subunit beta [Trematomus bernacchii]\n",
            " 4G51_D Chain D, Hemoglobin subunit beta [Trematomus bernacchii]\n",
            " 4IRO_B Crystal structure of T-state carbonmonoxy hemoglobin from Trematomus \n",
            "bernacchii at pH 8.4 [Trematomus bernacchii]\n",
            " 4IRO_D Crystal structure of T-state carbonmonoxy hemoglobin from Trematomus \n",
            "bernacchii at pH 8.4 [Trematomus bernacchii]\n",
            " 4ODC_B Chain B, Hemoglobin subunit beta [Trematomus bernacchii]\n",
            "Length=146\n",
            "\n",
            " Score = 143 bits (360),  Expect = 3e-44, Method: Compositional matrix adjust.\n",
            " Identities = 66/146 (45%), Positives = 96/146 (66%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T +E+S ++ ++  ++ D++G +AL R L+VYPWTQR F  FG+L   +A++GN  V\n",
            "Sbjct  1    VEWTDKERSIISDIFSHMDYDDIGPKALSRCLIVYPWTQRHFSGFGNLYNAEAIIGNANV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHG KVL     G+ ++DN+  T+A LS LH +KLHVDP+NF+LL + +  VLA   G \n",
            "Sbjct  61   AAHGIKVLHGLDRGVKNMDNIAATYADLSTLHSEKLHVDPDNFKLLSDCITIVLAAKMGH  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             FT   Q A+QK +A V +AL  +YH\n",
            "Sbjct  121  AFTAETQGAFQKFLAVVVSALGKQYH  146\n",
            "\n",
            "\n",
            ">6HIT_B Chain B, Hemoglobin beta 4 chain [Gadus morhua]\n",
            " 6HIT_D Chain D, Hemoglobin beta 4 chain [Gadus morhua]\n",
            " 6HIT_F Chain F, Hemoglobin beta 4 chain [Gadus morhua]\n",
            " 6HIT_H Chain H, Hemoglobin beta 4 chain [Gadus morhua]\n",
            "Length=145\n",
            "\n",
            " Score = 143 bits (360),  Expect = 4e-44, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 96/145 (66%), Gaps = 0/145 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T  E++ +T+++  ++ +E+G ++L R L+VYPWTQR+F +FG+L   + ++ NP +\n",
            "Sbjct  1    VEWTDSERAIITSIFSNLDYEEIGRKSLCRCLIVYPWTQRYFGAFGNLYNAETILANPLI  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHG K+L      L ++D++K T+A LS LH DKLHVDP+NFRLL + L  V+A   G \n",
            "Sbjct  61   AAHGTKILHGLDRALKNMDDIKNTYAELSLLHSDKLHVDPDNFRLLADCLTVVIAAKMGS  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "             FT   Q A+QK ++ V +AL  +Y\n",
            "Sbjct  121  AFTVDTQVAWQKFLSVVVSALGRQY  145\n",
            "\n",
            "\n",
            ">1LA6_B Chain B, Hemoglobin beta-1/2 chain [Trematomus newnesi]\n",
            " 1T1N_B Chain B, PROTEIN (HEMOGLOBIN) [Trematomus newnesi]\n",
            " 3D1K_B Chain B, Hemoglobin subunit beta-1/2 [Trematomus newnesi]\n",
            " 3NFE_B Chain B, Hemoglobin subunit beta-1/2 [Trematomus newnesi]\n",
            " 3NFE_D Chain D, Hemoglobin subunit beta-1/2 [Trematomus newnesi]\n",
            " 3NG6_B Chain B, Hemoglobin subunit beta-1/2 [Trematomus newnesi]\n",
            " 3NG6_D Chain D, Hemoglobin subunit beta-1/2 [Trematomus newnesi]\n",
            " 5LFG_B Chain B, Hemoglobin subunit beta-1/2 [Trematomus newnesi]\n",
            " 5LFG_D Chain D, Hemoglobin subunit beta-1/2 [Trematomus newnesi]\n",
            "Length=146\n",
            "\n",
            " Score = 142 bits (357),  Expect = 1e-43, Method: Compositional matrix adjust.\n",
            " Identities = 64/146 (44%), Positives = 93/146 (64%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T +E+S ++ ++  ++ D++G +AL R LVVYPWTQR+F  FG+L   + +M N  V\n",
            "Sbjct  1    VEWTDKERSIISDIFSHMDYDDIGPKALSRCLVVYPWTQRYFSGFGNLYNAEGIMSNANV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHG KVL     G+ ++DN+   +  LS LH +KLHVDP+NF+LL + +  VLA   G \n",
            "Sbjct  61   AAHGIKVLHGLDRGMKNMDNIADAYTDLSTLHSEKLHVDPDNFKLLSDCITIVLAAKMGH  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             FT   Q A+QK +A V +AL  +YH\n",
            "Sbjct  121  AFTAETQGAFQKFLAAVVSALGKQYH  146\n",
            "\n",
            "\n",
            ">6RP5_B Chain B, Hemoglobin subunit beta-1 [Eleginops maclovinus]\n",
            "Length=144\n",
            "\n",
            " Score = 140 bits (354),  Expect = 3e-43, Method: Compositional matrix adjust.\n",
            " Identities = 65/144 (45%), Positives = 97/144 (67%), Gaps = 0/144 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T +E++ +++++G ++ D++G +AL R L+VYPWTQR F SFG+L   +A++GN KV\n",
            "Sbjct  1    VEWTDQERATISSIFGSLDYDDIGPKALSRCLIVYPWTQRHFGSFGNLYNAEAIIGNQKV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "             AHG KVL      + ++DN+K  +A LS LH +KLHVDP+NF+LL + L  V+A   G \n",
            "Sbjct  61   AAHGIKVLHGLDRAVKNMDNIKEIYAELSILHSEKLHVDPDNFKLLADCLTIVVAAKMGS  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHK  145\n",
            "             F P  QA +QK +A V +AL  +\n",
            "Sbjct  121  GFNPGTQATFQKFLAVVVSALGKQ  144\n",
            "\n",
            "\n",
            ">2AA1_B Chain B, Hemoglobin beta-C chain [Trematomus newnesi]\n",
            " 2AA1_D Chain D, Hemoglobin beta-C chain [Trematomus newnesi]\n",
            "Length=146\n",
            "\n",
            " Score = 130 bits (327),  Expect = 5e-39, Method: Compositional matrix adjust.\n",
            " Identities = 62/146 (42%), Positives = 88/146 (60%), Gaps = 0/146 (0%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V  T  E++ +  ++ K+  D VG   L R LVVYPWTQR+F  FG+L    A+  N  V\n",
            "Sbjct  1    VEWTDFERATIKDIFSKLEYDVVGPATLARCLVVYPWTQRYFGKFGNLYNAAAIAQNAMV  60\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "              HG  +L      + ++D++  T+A LS LH +KLHVDP+NF+LL + L  V+A  FG \n",
            "Sbjct  61   SKHGTTILNGLDRAVKNMDDITNTYAELSVLHSEKLHVDPDNFKLLADCLTIVVAARFGS  120\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             FT  VQAA+QK +A V ++L  +Y \n",
            "Sbjct  121  AFTGEVQAAFQKFMAVVVSSLGKQYR  146\n",
            "\n",
            "\n",
            ">1HBH_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Trematomus bernacchii]\n",
            " 1HBH_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Trematomus bernacchii]\n",
            " 1PBX_A Chain A, Hemoglobin (carbonmonoxy) (alpha Chain) [Trematomus \n",
            "bernacchii]\n",
            " 1S5X_A Chain A, Hemoglobin alpha chain [Trematomus bernacchii]\n",
            " 1S5Y_A Chain A, Hemoglobin alpha chain [Trematomus bernacchii]\n",
            " 1S5Y_C Chain C, Hemoglobin alpha chain [Trematomus bernacchii]\n",
            " 2H8D_A Chain A, Hemoglobin alpha subunit [Trematomus bernacchii]\n",
            " 2H8D_C Chain C, Hemoglobin alpha subunit [Trematomus bernacchii]\n",
            " 2H8F_A Chain A, Hemoglobin alpha subunit [Trematomus bernacchii]\n",
            " 2H8F_C Chain C, Hemoglobin alpha subunit [Trematomus bernacchii]\n",
            " 2PEG_A Chain A, Hemoglobin subunit alpha [Trematomus bernacchii]\n",
            " 3GKV_A Chain A, Hemoglobin subunit alpha [Trematomus bernacchii]\n",
            " 3GQG_A Chain A, Hemoglobin subunit alpha [Trematomus bernacchii]\n",
            " 3GQG_C Chain C, Hemoglobin subunit alpha [Trematomus bernacchii]\n",
            " 4G51_A Chain A, Hemoglobin subunit alpha [Trematomus bernacchii]\n",
            " 4G51_C Chain C, Hemoglobin subunit alpha [Trematomus bernacchii]\n",
            " 4IRO_A Crystal structure of T-state carbonmonoxy hemoglobin from Trematomus \n",
            "bernacchii at pH 8.4 [Trematomus bernacchii]\n",
            " 4IRO_C Crystal structure of T-state carbonmonoxy hemoglobin from Trematomus \n",
            "bernacchii at pH 8.4 [Trematomus bernacchii]\n",
            " 4ODC_A Chain A, Hemoglobin subunit alpha [Trematomus bernacchii]\n",
            "Length=143\n",
            "\n",
            " Score = 122 bits (307),  Expect = 3e-36, Method: Compositional matrix adjust.\n",
            " Identities = 61/146 (42%), Positives = 93/146 (64%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+AV ALW K+  + D +G +AL R++VVYP T+ +F  +     PD   G+P +\n",
            "Sbjct  3    LSDKDKAAVRALWSKIGKSADAIGNDALSRMIVVYPQTKTYFSHW-----PDVTPGSPHI  57\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+G  +  ++ +D+LK     LSE H  KL VDP NF++L + ++ V++  F K\n",
            "Sbjct  58   KAHGKKVMGGIALAVSKIDDLKTGLMELSEQHAYKLRVDPANFKILNHCILVVISTMFPK  117\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP    +  K ++GVA ALA +Y \n",
            "Sbjct  118  EFTPEAHVSLDKFLSGVALALAERYR  143\n",
            "\n",
            "\n",
            ">3D1K_A Chain A, Hemoglobin subunit alpha-1 [Trematomus newnesi]\n",
            "Length=142\n",
            "\n",
            " Score = 122 bits (307),  Expect = 4e-36, Method: Compositional matrix adjust.\n",
            " Identities = 61/146 (42%), Positives = 93/146 (64%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+AV ALW K+  + D +G +AL R++VVYP T+ +F  +     PD   G+P +\n",
            "Sbjct  2    LSDKDKAAVRALWSKIGKSSDAIGNDALSRMIVVYPQTKIYFSHW-----PDVTPGSPNI  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+G  +  ++ +D+LK     LSE H  KL VDP NF++L + ++ V++  F K\n",
            "Sbjct  57   KAHGKKVMGGIALAVSKIDDLKTGLMELSEQHAYKLRVDPSNFKILNHCILVVISTMFPK  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP    +  K ++GVA ALA +Y \n",
            "Sbjct  117  EFTPEAHVSLDKFLSGVALALAERYR  142\n",
            "\n",
            "\n",
            ">1LA6_A Chain A, Hemoglobin alpha-1 chain [Trematomus newnesi]\n",
            " 1T1N_A Chain A, PROTEIN (HEMOGLOBIN) [Trematomus newnesi]\n",
            " 2AA1_A Chain A, Hemoglobin alpha-1 chain [Trematomus newnesi]\n",
            " 2AA1_C Chain C, Hemoglobin alpha-1 chain [Trematomus newnesi]\n",
            " 3NFE_A Chain A, Hemoglobin subunit alpha-1 [Trematomus newnesi]\n",
            " 3NFE_C Chain C, Hemoglobin subunit alpha-1 [Trematomus newnesi]\n",
            " 3NG6_A Chain A, Hemoglobin subunit alpha-1 [Trematomus newnesi]\n",
            " 3NG6_C Chain C, Hemoglobin subunit alpha-1 [Trematomus newnesi]\n",
            " 5LFG_A Chain A, Hemoglobin subunit alpha-1 [Trematomus newnesi]\n",
            " 5LFG_C Chain C, Hemoglobin subunit alpha-1 [Trematomus newnesi]\n",
            "Length=143\n",
            "\n",
            " Score = 122 bits (307),  Expect = 4e-36, Method: Compositional matrix adjust.\n",
            " Identities = 61/146 (42%), Positives = 93/146 (64%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+AV ALW K+  + D +G +AL R++VVYP T+ +F  +     PD   G+P +\n",
            "Sbjct  3    LSDKDKAAVRALWSKIGKSSDAIGNDALSRMIVVYPQTKIYFSHW-----PDVTPGSPNI  57\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+G  +  ++ +D+LK     LSE H  KL VDP NF++L + ++ V++  F K\n",
            "Sbjct  58   KAHGKKVMGGIALAVSKIDDLKTGLMELSEQHAYKLRVDPSNFKILNHCILVVISTMFPK  117\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP    +  K ++GVA ALA +Y \n",
            "Sbjct  118  EFTPEAHVSLDKFLSGVALALAERYR  143\n",
            "\n",
            "\n",
            ">3BJ1_A Chain A, hemoglobin alpha [Perca flavescens]\n",
            " 3BJ1_C Chain C, hemoglobin alpha [Perca flavescens]\n",
            " 3BJ2_A Chain A, hemoglobin alpha [Perca flavescens]\n",
            " 3BJ2_C Chain C, hemoglobin alpha [Perca flavescens]\n",
            " 3BJ3_A Chain A, hemoglobin alpha [Perca flavescens]\n",
            " 3BJ3_C Chain C, hemoglobin alpha [Perca flavescens]\n",
            "Length=142\n",
            "\n",
            " Score = 122 bits (305),  Expect = 9e-36, Method: Compositional matrix adjust.\n",
            " Identities = 63/146 (43%), Positives = 88/146 (60%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K AV ALWGK+    +E+G +ALGR+L VYP T+ +F  + DLS      G+  V\n",
            "Sbjct  2    LSSKDKDAVKALWGKIADKAEEIGADALGRMLAVYPQTKTYFSHWKDLSP-----GSAPV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "              HGK ++G   D +A +D+L      LSELH   L VDP NF++L + ++  LA  F K\n",
            "Sbjct  57   NKHGKTIMGGLVDAVASIDDLNAGLLALSELHAFTLRVDPANFKILSHCILVQLAVKFPK  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP V  +Y K  + VA ALA KY \n",
            "Sbjct  117  DFTPEVHLSYDKFFSAVARALAEKYR  142\n",
            "\n",
            "\n",
            ">1V4U_A Chain A, hemoglobin alpha chain [Thunnus thynnus]\n",
            " 1V4U_C Chain C, hemoglobin alpha chain [Thunnus thynnus]\n",
            " 1V4W_A Chain A, hemoglobin alpha chain [Thunnus thynnus]\n",
            " 1V4W_C Chain C, hemoglobin alpha chain [Thunnus thynnus]\n",
            " 1V4X_A Chain A, hemoglobin alpha chain [Thunnus thynnus]\n",
            " 1V4X_C Chain C, hemoglobin alpha chain [Thunnus thynnus]\n",
            "Length=144\n",
            "\n",
            " Score = 122 bits (305),  Expect = 9e-36, Method: Compositional matrix adjust.\n",
            " Identities = 63/146 (43%), Positives = 92/146 (63%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVN--VDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++KS V ALWGK++   D +G +ALGR+L VYP T+ +F  + D+S      G+  V\n",
            "Sbjct  4    LSDKDKSTVKALWGKISKSADAIGADALGRMLAVYPQTKTYFSHWPDMSP-----GSGPV  58\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+G  +  ++ +D+L      LSELH  K+ VDP NF++L + ++ V+A  F K\n",
            "Sbjct  59   KAHGKKVMGGVALAVSKIDDLTTGLGDLSELHAFKMRVDPSNFKILSHCILVVVAKMFPK  118\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP    +  K +A VA ALA +Y \n",
            "Sbjct  119  EFTPDAHVSLDKFLASVALALAERYR  144\n",
            "\n",
            "\n",
            ">1GLI_A Chain A, DEOXYHEMOGLOBIN [Homo sapiens]\n",
            " 1GLI_C Chain C, DEOXYHEMOGLOBIN [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 119 bits (299),  Expect = 7e-35, Method: Compositional matrix adjust.\n",
            " Identities = 64/145 (44%), Positives = 89/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +PWT+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPWTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1V75_A Chain A, Hemoglobin D alpha chain [Aldabrachelys gigantea]\n",
            " 1WMU_A Chain A, Hemoglobin D alpha chain [Aldabrachelys gigantea]\n",
            " 2Z6N_A Chain A, Hemoglobin D subunit alpha [Aldabrachelys gigantea]\n",
            "Length=141\n",
            "\n",
            " Score = 118 bits (296),  Expect = 1e-34, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            LT ++K  +  +W KV  + ++ G EAL R+ +VYP T+ +F  F      D    + ++\n",
            "Sbjct  2    LTEDDKQLIQHVWEKVLEHQEDFGAEALERMFIVYPSTKTYFPHF------DLHHDSEQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            + HGKKV+GA  D + H+DNL  T + LS LH   L VDP NF+LL +    VL  H G+\n",
            "Sbjct  56   RHHGKKVVGALGDAVKHIDNLSATLSELSNLHAYNLRVDPVNFKLLSHCFQVVLGAHLGR  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            E+TP VQ AY K +A V+  LA KY\n",
            "Sbjct  116  EYTPQVQVAYDKFLAAVSAVLAEKY  140\n",
            "\n",
            "\n",
            ">1XQ5_A Chain A, Hemoglobin alpha-1 chain [Perca flavescens]\n",
            " 1XQ5_C Chain C, Hemoglobin alpha-1 chain [Perca flavescens]\n",
            "Length=143\n",
            "\n",
            " Score = 118 bits (296),  Expect = 2e-34, Method: Compositional matrix adjust.\n",
            " Identities = 60/146 (41%), Positives = 87/146 (60%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K  V ALWGK+    +E+G +AL R+L VYP T+ +F  + DLS      G+  V\n",
            "Sbjct  3    LSSKDKDTVKALWGKIADKAEEIGSDALSRMLAVYPQTKTYFSHWKDLSP-----GSAPV  57\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "              HGK ++G   D +A +D+L      LSELH   L VDP NF++L + ++ +LA  F K\n",
            "Sbjct  58   NKHGKTIMGGIVDAVASIDDLNAGLLALSELHAFTLRVDPANFKILSHCILVLLAVKFPK  117\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP V  +Y K  + +A ALA KY \n",
            "Sbjct  118  DFTPEVHISYDKFFSALARALAEKYR  143\n",
            "\n",
            "\n",
            ">6IYH_A Chain A, Alpha chain [Acipenser persicus]\n",
            "Length=142\n",
            "\n",
            " Score = 118 bits (296),  Expect = 2e-34, Method: Compositional matrix adjust.\n",
            " Identities = 60/145 (41%), Positives = 90/145 (62%), Gaps = 7/145 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVN--VDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            LT  +KS V ++W K +   +E+G EALGR+L V+P T+ +F  + DLS     + + +V\n",
            "Sbjct  2    LTSADKSHVKSIWSKASGKAEELGAEALGRMLEVFPNTKTYFSHYADLS-----VSSGQV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "              HGKK+L A +  + H+D++ GT   LS LH   L VDP NF++L + ++ VLA +F  \n",
            "Sbjct  57   HTHGKKILDAITTAVNHIDDITGTMTALSTLHAKTLRVDPANFKILSHTILVVLALYFPA  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V  A  K +A V++ LA KY\n",
            "Sbjct  117  DFTPEVHLACDKFLASVSHTLATKY  141\n",
            "\n",
            "\n",
            ">5EUI_A Chain A, HBA protein [Ochotona princeps]\n",
            "Length=141\n",
            "\n",
            " Score = 118 bits (295),  Expect = 2e-34, Method: Compositional matrix adjust.\n",
            " Identities = 64/145 (44%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F      D   G+ +V\n",
            "Sbjct  2    LSPADKANVKAAWGKVGGHAGEYGAEALDRMFLSFPTTKTYFPHF------DMSHGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA+H   \n",
            "Sbjct  56   KAHGKKVADALTQAVDHLDDLPGALSALSDLHAHKLRVDPVNFKLLAHCLLVTLANHHPN  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1OUT_A Chain A, Hemoglobin I [Oncorhynchus mykiss]\n",
            " 1OUU_A Chain A, Hemoglobin I [Oncorhynchus mykiss]\n",
            " 1OUU_C Chain C, Hemoglobin I [Oncorhynchus mykiss]\n",
            "Length=143\n",
            "\n",
            " Score = 118 bits (295),  Expect = 3e-34, Method: Compositional matrix adjust.\n",
            " Identities = 64/146 (44%), Positives = 88/146 (60%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVN--VDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            LT ++KS V A WGK++   D VG EALGR+L  YP T+ +F  + DLS      G+  V\n",
            "Sbjct  3    LTAKDKSVVKAFWGKISGKADVVGAEALGRMLTAYPQTKTYFSHWADLSP-----GSGPV  57\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  ++GA    +  +D+L G  + LS+LH  KL VDP NF++L + ++  LA HF  \n",
            "Sbjct  58   KKHGGIIMGAIGKAVGLMDDLVGGMSALSDLHAFKLRVDPGNFKILSHNILVTLAIHFPS  117\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP V  A  K +A V+ ALA KY \n",
            "Sbjct  118  DFTPEVHIAVDKFLAAVSAALADKYR  143\n",
            "\n",
            "\n",
            ">6IYI_A Chain A, Alpha chain [Acipenser stellatus]\n",
            "Length=142\n",
            "\n",
            " Score = 117 bits (293),  Expect = 6e-34, Method: Compositional matrix adjust.\n",
            " Identities = 58/145 (40%), Positives = 90/145 (62%), Gaps = 7/145 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            LT  +KS V ++W K   + +E+G EALGR+L  +P T+ +F+ + DLS   A     +V\n",
            "Sbjct  2    LTSADKSHVRSIWSKAGGSAEEIGAEALGRMLESFPNTKTYFDHYADLSVSSA-----QV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "              HGKK++ A +  + H+D++ G  ++LS LH   L VDP NF++L + ++ VLA +F  \n",
            "Sbjct  57   HTHGKKIIDALTTAVNHIDDITGALSSLSTLHAQTLRVDPANFKILSHTILVVLALYFPA  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V  A  K +A V++ALA  Y\n",
            "Sbjct  117  DFTPEVHLACDKFLANVSHALADNY  141\n",
            "\n",
            "\n",
            ">4ESA_A Chain A, Hemoglobin alpha chain [Eleginops maclovinus]\n",
            " 4ESA_C Chain C, Hemoglobin alpha chain [Eleginops maclovinus]\n",
            " 6RP5_A Chain A, Hemoglobin subunit alpha 1 [Eleginops maclovinus]\n",
            "Length=143\n",
            "\n",
            " Score = 116 bits (291),  Expect = 1e-33, Method: Compositional matrix adjust.\n",
            " Identities = 58/146 (40%), Positives = 92/146 (63%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVN--VDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+AV  LW K++   D +G +AL R++VVYP T+ +F  + DLS      G+P V\n",
            "Sbjct  3    LSDKDKAAVKLLWSKISKSSDAIGNDALSRMIVVYPQTKTYFAHWPDLSP-----GSPHV  57\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGK V+G  +  ++ +D+L+     LSE H  KL VDP NF++L + ++ V++  F K\n",
            "Sbjct  58   KAHGKTVMGGIALAVSKIDDLRAGLLDLSEQHAYKLRVDPANFKILSHCILVVISMMFPK  117\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP    +  K ++GV+ AL+ +Y \n",
            "Sbjct  118  EFTPEAHVSLDKFLSGVSLALSERYR  143\n",
            "\n",
            "\n",
            ">3FH9_A Chain A, Hemoglobin alpha chain [Pteropus giganteus]\n",
            "Length=141\n",
            "\n",
            " Score = 115 bits (288),  Expect = 2e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +KS V A W KV  NV E G EAL R+ + +P T+ +F  F      D   G+ +V\n",
            "Sbjct  2    LSSTDKSNVKAAWDKVGGNVGEYGAEALERMFLSFPTTKTYFPHF------DLAHGSSQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A ++ + H+D+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVGDALTNAVGHIDDLPGALSALSDLHAYKLRVDPVNFKLLSHCLLVTLASHLPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1AJ9_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 115 bits (287),  Expect = 5e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSSQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1A00_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A00_C Chain C, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A01_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A01_C Chain C, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A0U_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A0U_C Chain C, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A0Z_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A0Z_C Chain C, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A3N_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A3N_C Chain C, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A9W_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A9W_C Chain C, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1B86_A Chain A, PROTEIN (HEMOGLOBIN; ALPHA CHAIN) [Homo sapiens]\n",
            " 1B86_C Chain C, PROTEIN (HEMOGLOBIN; ALPHA CHAIN) [Homo sapiens]\n",
            " 1BBB_A Chain A, HEMOGLOBIN A (CARBONMONOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1BBB_C Chain C, HEMOGLOBIN A (CARBONMONOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1BIJ_A Chain A, HEMOGLOBIN A [Homo sapiens]\n",
            " 1BIJ_C Chain C, HEMOGLOBIN A [Homo sapiens]\n",
            " 1BUW_A Chain A, PROTEIN (HEMOGLOBIN) [Homo sapiens]\n",
            " 1BUW_C Chain C, PROTEIN (HEMOGLOBIN) [Homo sapiens]\n",
            " 1BZ0_A Chain A, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1BZ0_C Chain C, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1CLS_A Chain A, HEMOGLOBIN [Homo sapiens]\n",
            " 1CLS_C Chain C, HEMOGLOBIN [Homo sapiens]\n",
            " 1CMY_A Chain A, HEMOGLOBIN YPSILANTI (CARBONMONOXY) (ALPHA CHAIN) [Homo \n",
            "sapiens]\n",
            " 1CMY_C Chain C, HEMOGLOBIN YPSILANTI (CARBONMONOXY) (ALPHA CHAIN) [Homo \n",
            "sapiens]\n",
            " 1COH_A Chain A, HEMOGLOBIN (FERROUS CARBONMONOXY) (ALPHA CHAIN) [Homo \n",
            "sapiens]\n",
            " 1COH_C Chain C, HEMOGLOBIN (FERROUS CARBONMONOXY) (ALPHA CHAIN) [Homo \n",
            "sapiens]\n",
            " 1DKE_A Chain A, HEMOGLOBIN: ALPHA CHAIN [Homo sapiens]\n",
            " 1DKE_C Chain C, HEMOGLOBIN: ALPHA CHAIN [Homo sapiens]\n",
            " 1DXT_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1DXT_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1DXU_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1DXU_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1DXV_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1DXV_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1FDH_A Chain A, HEMOGLOBIN F (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1FDH_B Chain B, HEMOGLOBIN F (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1FN3_A Chain A, HEMOGLOBIN ALPHA CHAIN [Homo sapiens]\n",
            " 1FN3_C Chain C, HEMOGLOBIN ALPHA CHAIN [Homo sapiens]\n",
            " 1G9V_A Chain A, HEMOGLOBIN ALPHA CHAIN [Homo sapiens]\n",
            " 1G9V_C Chain C, HEMOGLOBIN ALPHA CHAIN [Homo sapiens]\n",
            " 1GBU_A Chain A, HEMOGLOBIN [Homo sapiens]\n",
            " 1GBU_C Chain C, HEMOGLOBIN [Homo sapiens]\n",
            " 1GBV_A Chain A, HEMOGLOBIN [Homo sapiens]\n",
            " 1GBV_C Chain C, HEMOGLOBIN [Homo sapiens]\n",
            " 1GZX_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 1GZX_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 1HAB_A Chain A, HEMOGLOBIN A [Homo sapiens]\n",
            " 1HAB_C Chain C, HEMOGLOBIN A [Homo sapiens]\n",
            " 1HAC_A Chain A, HEMOGLOBIN A [Homo sapiens]\n",
            " 1HAC_C Chain C, HEMOGLOBIN A [Homo sapiens]\n",
            " 1HBA_A Chain A, HEMOGLOBIN ROTHSCHILD (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HBA_C Chain C, HEMOGLOBIN ROTHSCHILD (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HBB_A Chain A, HEMOGLOBIN A (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HBB_C Chain C, HEMOGLOBIN A (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HBS_A Chain A, HEMOGLOBIN S (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HBS_C Chain C, HEMOGLOBIN S (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HBS_E Chain E, HEMOGLOBIN S (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HBS_G Chain G, HEMOGLOBIN S (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HCO_A Chain A, HEMOGLOBIN (CARBONMONOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HDB_A Chain A, HEMOGLOBIN (DEOXY) BETA-V67T [Homo sapiens]\n",
            " 1HDB_C Chain C, HEMOGLOBIN (DEOXY) BETA-V67T [Homo sapiens]\n",
            " 1HGA_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HGA_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HGB_A Chain A, HEMOGLOBIN (AQUO MET) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HGB_C Chain C, HEMOGLOBIN (AQUO MET) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HGC_A Chain A, HEMOGLOBIN (OXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HGC_C Chain C, HEMOGLOBIN (OXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1HHO_A Chain A, HEMOGLOBIN A (OXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1IRD_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1J3Y_A Chain A, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J3Y_C Chain C, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J3Y_E Chain E, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J3Y_G Chain G, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J3Z_A Chain A, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J3Z_C Chain C, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J3Z_E Chain E, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J3Z_G Chain G, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J40_A Chain A, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J40_C Chain C, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J40_E Chain E, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J40_G Chain G, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J41_A Chain A, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J41_C Chain C, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J41_E Chain E, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1J41_G Chain G, Hemoglobin alpha Chain [Homo sapiens]\n",
            " 1JY7_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1JY7_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1JY7_P Chain P, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1JY7_R Chain R, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1JY7_U Chain U, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1JY7_W Chain W, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1K0Y_A Chain A, hemoglobin alpha chain [Homo sapiens]\n",
            " 1K0Y_C Chain C, hemoglobin alpha chain [Homo sapiens]\n",
            " 1K1K_A Chain A, HEMOGLOBIN ALPHA CHAIN [Homo sapiens]\n",
            " 1KD2_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1KD2_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LFL_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LFL_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LFL_P Chain P, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LFL_R Chain R, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LFQ_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LFT_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LFV_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LFY_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LFZ_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1LJW_A Chain A, hemoglobin alpha chain [Homo sapiens]\n",
            " 1M9P_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1M9P_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1MKO_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1MKO_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1NEJ_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1NEJ_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1NIH_A Chain A, HEMOGLOBIN (NICKELOUS DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1NIH_C Chain C, HEMOGLOBIN (NICKELOUS DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1NQP_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1NQP_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1QSH_A Chain A, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1QSH_C Chain C, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1QSI_A Chain A, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1QSI_C Chain C, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1QXD_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1QXD_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1QXE_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1QXE_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1RPS_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1RPS_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1RQ3_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1RQ3_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1RQ4_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1RQ4_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1RQA_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1RQA_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1SDK_A Chain A, Hemoglobin A [Homo sapiens]\n",
            " 1SDK_C Chain C, Hemoglobin A [Homo sapiens]\n",
            " 1SDL_A Chain A, Hemoglobin A [Homo sapiens]\n",
            " 1SDL_C Chain C, Hemoglobin A [Homo sapiens]\n",
            " 1SHR_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1SHR_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1SI4_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1SI4_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1THB_A Chain A, HEMOGLOBIN A (OXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1THB_C Chain C, HEMOGLOBIN A (OXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1UIW_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1UIW_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1UIW_E Chain E, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1UIW_G Chain G, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XXT_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XXT_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XZ2_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XZ2_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y0T_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y0T_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y0W_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y0W_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y22_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y22_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y2Z_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y2Z_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y31_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y31_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y35_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y35_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y45_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y45_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y46_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y46_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4B_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4B_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4F_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4F_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4G_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4G_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4P_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4P_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4Q_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4Q_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4R_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4R_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4V_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y4V_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y5F_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y5F_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y5J_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y5J_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y5K_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y5K_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y7C_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y7C_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y7D_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y7D_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y7G_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y7G_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y7Z_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y7Z_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y83_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y83_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y85_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y85_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YE0_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YE0_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YE1_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YE1_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YE2_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YE2_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEN_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEN_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEO_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEO_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEQ_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEQ_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEU_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEU_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEV_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YEV_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YFF_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YFF_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YFF_E Chain E, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YFF_G Chain G, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YG5_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YG5_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YGD_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YGD_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YGF_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YGF_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YH9_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YH9_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YHE_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YHE_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YHR_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YHR_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YIE_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YIE_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YIH_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YIH_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YVQ_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YVQ_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YVT_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YZI_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 2D5Z_A Chain A, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2D5Z_C Chain C, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2D60_A Chain A, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2D60_C Chain C, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2DN1_A Chain A, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2DN2_A Chain A, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2DN2_C Chain C, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2DN3_A Chain A, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2DXM_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 2DXM_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 2H35_A Chain A, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2H35_C Chain C, Hemoglobin alpha subunit [Homo sapiens]\n",
            " 2HBC_A Chain A, HEMOGLOBIN A (ETHYL ISOCYANIDE) (ALPHA CHAIN) [Homo \n",
            "sapiens]\n",
            " 2HBD_A Chain A, HEMOGLOBIN A (METHYL ISOCYANIDE) (ALPHA CHAIN) [Homo \n",
            "sapiens]\n",
            " 2HBE_A Chain A, HEMOGLOBIN A (N-BUTYL ISOCYANIDE) (ALPHA CHAIN) [Homo \n",
            "sapiens]\n",
            " 2HBF_A Chain A, HEMOGLOBIN A (N-PROPYL ISOCYANIDE) (ALPHA CHAIN) [Homo \n",
            "sapiens]\n",
            " 2HBS_A Chain A, HEMOGLOBIN S (DEOXY), ALPHA CHAIN [Homo sapiens]\n",
            " 2HBS_C Chain C, HEMOGLOBIN S (DEOXY), ALPHA CHAIN [Homo sapiens]\n",
            " 2HBS_E Chain E, HEMOGLOBIN S (DEOXY), ALPHA CHAIN [Homo sapiens]\n",
            " 2HBS_G Chain G, HEMOGLOBIN S (DEOXY), ALPHA CHAIN [Homo sapiens]\n",
            " 2HCO_A Chain A, HEMOGLOBIN (CARBONMONOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 2HHB_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 2HHB_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 2HHD_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 2HHD_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 2HHE_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 2HHE_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 2M6Z_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 2M6Z_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 2W6V_A Chain A, HEMOGLOBIN SUBUNIT ALPHA [Homo sapiens]\n",
            " 2W6V_C Chain C, HEMOGLOBIN SUBUNIT ALPHA [Homo sapiens]\n",
            " 2YRS_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 2YRS_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 2YRS_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 2YRS_M Chain M, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3B75_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3B75_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3B75_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3B75_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3B75_S Chain S, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3D17_A Chain A, Hemoglobin subunit alpha [unidentified]\n",
            " 3D17_C Chain C, Hemoglobin subunit alpha [unidentified]\n",
            " 3D7O_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3DUT_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3DUT_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3HHB_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 3HHB_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 3HXN_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3HXN_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3IC0_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3IC0_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3IC2_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3IC2_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3KMF_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3KMF_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3NL7_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3ODQ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3ODQ_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3ONZ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3OO4_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3OO5_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3OVU_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3P5Q_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3QJC_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3QJE_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3QJE_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3R5I_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3R5I_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3S48_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3S48_D Chain D, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3S65_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3S65_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3S66_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3SZK_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3SZK_D Chain D, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3WCP_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3WCP_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3WHM_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3WHM_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4FC3_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4HHB_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4HHB_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4IJ2_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4IJ2_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4L7Y_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4L7Y_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4M4A_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4M4B_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4MQG_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4MQJ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4MQJ_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4MQJ_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4MQJ_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4MQK_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4MQK_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4MQK_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4MQK_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7N_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7N_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7N_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7N_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7N_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7N_K Chain K, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7O_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7O_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7O_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7O_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7O_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7O_K Chain K, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7P_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7P_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7P_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7P_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7P_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N7P_K Chain K, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4N8T_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4NI0_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4NI1_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4ROL_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4ROL_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4ROM_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4ROM_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4WJG_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4WJG_F Chain F, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4WJG_K Chain K, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4WJG_P Chain P, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4WJG_U Chain U, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4WJG_Z Chain Z, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4X0L_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 4XS0_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5E29_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5E29_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5E6E_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5E83_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5E83_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5EE4_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5EE4_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5HU6_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5HY8_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5HY8_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5HY8_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5HY8_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5HY8_S Chain S, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5JDO_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5KDQ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5KDQ_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5KSI_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5KSI_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5KSJ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5KSJ_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5NI1_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5NI1_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5U3I_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5U3I_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5UCU_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5UFJ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5UFJ_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5URC_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5URC_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5VMM_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5VMM_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2R_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2R_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2R_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2R_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2R_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2R_K Chain K, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2S_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2S_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2S_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2S_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2S_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2S_K Chain K, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2T_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2T_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2T_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2T_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2T_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2T_K Chain K, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2U_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2U_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2U_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2U_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2U_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5X2U_K Chain K, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6BNR_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6BNR_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6BWP_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6BWP_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6BWU_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6DI4_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6DI4_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6HBW_A Chain A, PROTEIN (HEMOGLOBIN ALPHA 1) [Homo sapiens]\n",
            " 6HBW_C Chain C, PROTEIN (HEMOGLOBIN ALPHA 1) [Homo sapiens]\n",
            " 6HK2_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6HK2_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KA9_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KA9_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KA9_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KA9_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAE_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAE_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAE_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAE_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAH_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAH_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAH_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAH_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAI_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAI_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAI_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAI_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAO_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAP_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAQ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAR_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAS_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAS_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAT_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAT_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAU_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAU_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAV_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KAV_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6L5V_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6L5W_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6L5X_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6L5X_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6L5Y_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6L5Y_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6LCW_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6LCW_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6LCW_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6LCW_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6LCX_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6LCX_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6LCX_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6LCX_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6NQ5_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6TB2_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6XD9_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6XD9_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6XDT_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6XDT_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6XE7_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6XE7_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JJQ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JJQ_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JXZ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JXZ_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JY0_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JY0_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JY1_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JY1_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JY3_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7JY3_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7PCF_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7PCH_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7PCH_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7PCQ_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7PCQ_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UVB_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UVB_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8DOV_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8DOV_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8DOV_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8DOV_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8FDK_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8FDK_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8FDL_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8FDL_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8FDM_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8FDM_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8FDN_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8FDN_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 114 bits (286),  Expect = 5e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1BZZ_A Chain A, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1BZZ_C Chain C, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1C7B_A Chain A, PROTEIN (DEOXYHEMOGLOBIN (ALPHA CHAIN)) [Homo sapiens]\n",
            " 1C7B_C Chain C, PROTEIN (DEOXYHEMOGLOBIN (ALPHA CHAIN)) [Homo sapiens]\n",
            " 1J7W_A Chain A, hemoglobin [Homo sapiens]\n",
            " 1J7W_C Chain C, hemoglobin [Homo sapiens]\n",
            " 1O1K_A Chain A, Hemoglobin Alpha chain [Homo sapiens]\n",
            " 1O1K_C Chain C, Hemoglobin Alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 114 bits (286),  Expect = 5e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1BZ1_A Chain A, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1BZ1_C Chain C, PROTEIN (HEMOGLOBIN ALPHA CHAIN) [Homo sapiens]\n",
            " 1Y01_B Chain B, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Z8U_B Chain B, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Z8U_D Chain D, Hemoglobin alpha chain [Homo sapiens]\n",
            " 6KYE_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KYE_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KYE_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KYE_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KYE_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6KYE_K Chain K, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7CUE_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7CUE_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7K4M_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7K4M_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7K4M_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7K4M_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7K4M_I Chain I, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UD7_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UD7_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UD8_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UD8_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UF6_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UF6_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UF7_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7UF7_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7VDE_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7VDE_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7XGY_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7XGY_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8EGI_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 8EGI_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=142\n",
            "\n",
            " Score = 114 bits (286),  Expect = 5e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  3    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  57   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  117  EFTPAVHASLDKFLASVSTVLTSKY  141\n",
            "\n",
            "\n",
            ">3IA3_B Chain B, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3IA3_D Chain D, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=145\n",
            "\n",
            " Score = 114 bits (286),  Expect = 6e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  6    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  59\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  60   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  119\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  120  EFTPAVHASLDKFLASVSTVLTSKY  144\n",
            "\n",
            "\n",
            ">1BAB_A Chain A, HEMOGLOBIN THIONVILLE (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            " 1BAB_C Chain C, HEMOGLOBIN THIONVILLE (DEOXY) (ALPHA CHAIN) [Homo sapiens]\n",
            "Length=143\n",
            "\n",
            " Score = 114 bits (286),  Expect = 6e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/147 (43%), Positives = 89/147 (61%), Gaps = 8/147 (5%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNP  59\n",
            "            + L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ \n",
            "Sbjct  2    MELSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSA  55\n",
            "\n",
            "Query  60   KVKAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHF  119\n",
            "            +VK HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H \n",
            "Sbjct  56   QVKGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHL  115\n",
            "\n",
            "Query  120  GKEFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  PAEFTPAVHASLDKFLASVSTVLTSKY  142\n",
            "\n",
            "\n",
            ">1Y0D_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y0D_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 4MQI_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5JDO_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6NBC_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6NBC_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6NBD_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6NBD_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=140\n",
            "\n",
            " Score = 114 bits (286),  Expect = 6e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">6BB5_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6HAL_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 6HAL_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7AET_AAA Chain AAA, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7AET_CCC Chain CCC, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7AEU_AAA Chain AAA, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7AEU_CCC Chain CCC, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7AEV_AAA Chain AAA, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7AEV_CCC Chain CCC, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=139\n",
            "\n",
            " Score = 114 bits (286),  Expect = 6e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  1    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  54\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  55   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  114\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  115  EFTPAVHASLDKFLASVSTVLTSKY  139\n",
            "\n",
            "\n",
            ">5SW7_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=142\n",
            "\n",
            " Score = 114 bits (286),  Expect = 6e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1HBR_A Chain A, PROTEIN (HEMOGLOBIN D) [Gallus gallus]\n",
            " 1HBR_C Chain C, PROTEIN (HEMOGLOBIN D) [Gallus gallus]\n",
            "Length=141\n",
            "\n",
            " Score = 114 bits (285),  Expect = 8e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 85/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            LT E+K  +   W K   + +E G EAL R+   YP T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LTAEDKKLIQQAWEKAASHQEEFGAEALTRMFTTYPQTKTYFPHF-DLSP-----GSDQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            + HGKKVLGA  + + ++DNL    A LS LH   L VDP NF+LL   +  VLA H GK\n",
            "Sbjct  56   RGHGKKVLGALGNAVKNVDNLSQAMAELSNLHAYNLRVDPVNFKLLSQCIQVVLAVHMGK  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            ++TP V AA+ K ++ V+  LA KY\n",
            "Sbjct  116  DYTPEVHAAFDKFLSAVSAVLAEKY  140\n",
            "\n",
            "\n",
            ">1RVW_A Chain A, HEMOGLOBIN [Homo sapiens]\n",
            " 1VWT_A Chain A, HEMOGLOBIN [Homo sapiens]\n",
            " 1VWT_C Chain C, HEMOGLOBIN [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 114 bits (285),  Expect = 9e-33, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPWNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1XZ7_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XZ7_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y8W_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y8W_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 114 bits (284),  Expect = 1e-32, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLAVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">2R1H_A Chain A, Hemoglobin subunit alpha-4 [Oncorhynchus mykiss]\n",
            " 2R1H_C Chain C, Hemoglobin subunit alpha-4 [Oncorhynchus mykiss]\n",
            " 3BOM_A Chain A, Hemoglobin subunit alpha-4 [Oncorhynchus mykiss]\n",
            " 3BOM_C Chain C, Hemoglobin subunit alpha-4 [Oncorhynchus mykiss]\n",
            "Length=143\n",
            "\n",
            " Score = 114 bits (284),  Expect = 1e-32, Method: Compositional matrix adjust.\n",
            " Identities = 58/146 (40%), Positives = 88/146 (60%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+ V A+WGK+    DE+G +AL R+LVVYP T+ +F  +  ++      G+  V\n",
            "Sbjct  3    LSAKDKANVKAIWGKILPKSDEIGEQALSRMLVVYPQTKAYFSHWASVAP-----GSAPV  57\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  ++    D + H+D+L G    LSELH  KL VDP NF++L + L+ V+A +F  \n",
            "Sbjct  58   KKHGITIMNQIDDCVGHMDDLFGFLTKLSELHATKLRVDPTNFKILAHNLIVVIAAYFPA  117\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP +  +  K +  +A ALA KY \n",
            "Sbjct  118  EFTPEIHLSVDKFLQQLALALAEKYR  143\n",
            "\n",
            "\n",
            ">4MQC_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 113 bits (283),  Expect = 2e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKK+  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKMADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1O1O_A Chain A, Hemoglobin Alpha chain [Homo sapiens]\n",
            " 1O1O_C Chain C, Hemoglobin Alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 113 bits (283),  Expect = 2e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKK+  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKLADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1XY0_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XY0_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 113 bits (282),  Expect = 2e-32, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T  +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTGTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1Y0C_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y0C_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YDZ_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1YDZ_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 113 bits (282),  Expect = 2e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  K+\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKF  140\n",
            "\n",
            "\n",
            ">1A3O_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            " 1A3O_C Chain C, HEMOGLOBIN (ALPHA CHAIN) [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 112 bits (281),  Expect = 3e-32, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+  F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTHFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1XZ5_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XZ5_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 112 bits (281),  Expect = 3e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  K  VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKARVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1XYE_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XYE_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XZ4_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XZ4_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 112 bits (281),  Expect = 3e-32, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+  F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTAFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">7DY4_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7DY4_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7DY4_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7DY4_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 112 bits (281),  Expect = 4e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K +GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGYGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">7DY3_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7DY3_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7DY3_E Chain E, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7DY3_G Chain G, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 112 bits (281),  Expect = 4e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+L+  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLYAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">3D1A_A Chain A, Hemoglobin subunit alpha-1/2 [Capra hircus]\n",
            " 3D1A_C Chain C, Hemoglobin subunit alpha-1/2 [Capra hircus]\n",
            "Length=142\n",
            "\n",
            " Score = 112 bits (280),  Expect = 4e-32, Method: Compositional matrix adjust.\n",
            " Identities = 64/145 (44%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +KS V A WGKV  N    G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  3    LSAADKSNVKAAWGKVGGNAGAYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG+KV  A +  + HLD+L GT + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  57   KGHGEKVAAALTKAVGHLDDLPGTLSDLSDLHAHKLRVDPVNFKLLSHSLLVTLACHLPN  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K +A V+  L  KY\n",
            "Sbjct  117  DFTPAVHASLDKFLANVSTVLTSKY  141\n",
            "\n",
            "\n",
            ">2QU0_A Chain A, Hemoglobin subunit alpha-1/2 [Ovis aries]\n",
            " 2QU0_C Chain C, Hemoglobin subunit alpha-1/2 [Ovis aries]\n",
            " 2RI4_A Chain A, Hemoglobin subunit alpha-1/2 [Capra hircus]\n",
            " 2RI4_C Chain C, Hemoglobin subunit alpha-1/2 [Capra hircus]\n",
            " 2RI4_I Chain I, Hemoglobin subunit alpha-1/2 [Capra hircus]\n",
            " 2RI4_K Chain K, Hemoglobin subunit alpha-1/2 [Capra hircus]\n",
            " 3EU1_A Chain A, Hemoglobin subunit alpha-1/2 [Capra hircus]\n",
            " 3EU1_C Chain C, Hemoglobin subunit alpha-1/2 [Capra hircus]\n",
            "Length=141\n",
            "\n",
            " Score = 112 bits (280),  Expect = 5e-32, Method: Compositional matrix adjust.\n",
            " Identities = 64/145 (44%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +KS V A WGKV  N    G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKSNVKAAWGKVGGNAGAYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG+KV  A +  + HLD+L GT + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGEKVAAALTKAVGHLDDLPGTLSDLSDLHAHKLRVDPVNFKLLSHSLLVTLACHLPN  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLANVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1Y0A_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y0A_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 112 bits (279),  Expect = 6e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/144 (43%), Positives = 87/144 (60%), Gaps = 8/144 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHK  145\n",
            "            EFTP V A+  K +A V+  L  K\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSK  139\n",
            "\n",
            "\n",
            ">2QLS_A Chain A, Hemoglobin subunit alpha [Canis lupus familiaris]\n",
            " 2QLS_C Chain C, Hemoglobin subunit alpha [Canis lupus familiaris]\n",
            " 3GOU_A Chain A, Hemoglobin subunit alpha [Canis lupus familiaris]\n",
            " 3GOU_C Chain C, Hemoglobin subunit alpha [Canis lupus familiaris]\n",
            " 3PEL_A Chain A, Hemoglobin subunit alpha [Canis lupus familiaris]\n",
            "Length=141\n",
            "\n",
            " Score = 112 bits (279),  Expect = 6e-32, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ + + W K+  +  + GGEAL R    +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNIKSTWDKIGGHAGDYGGEALDRTFQSFPTTKTYFPHF-DLSP-----GSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  +AHLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVADALTTAVAHLDDLPGALSALSDLHAYKLRVDPVNFKLLSHCLLVTLACHHPT  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K  A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFFAAVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1Y09_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1Y09_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 112 bits (279),  Expect = 7e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP  F+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVAFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1R1X_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1R1Y_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1R1Y_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 111 bits (278),  Expect = 9e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL V P NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVAPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1XZU_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XZU_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 111 bits (278),  Expect = 9e-32, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL V P NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVGPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1XZV_A Chain A, Hemoglobin alpha chain [Homo sapiens]\n",
            " 1XZV_C Chain C, Hemoglobin alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 111 bits (278),  Expect = 1e-31, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VD  NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDAVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">7QU4_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 7QU4_B Chain B, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=142\n",
            "\n",
            " Score = 111 bits (278),  Expect = 1e-31, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  3    LSPADKTNVEAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            + HGKKV  A ++ +AH+D++    + LS+LH  +L VDP NF+LL + L+  LA H   \n",
            "Sbjct  57   EGHGKKVADALTNAVAHVDDMPDALSALSDLHAHELRVDPVNFKLLSHCLLVTLAAHLPA  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  117  EFTPAVHASLDKFLASVSTVLTSKY  141\n",
            "\n",
            "\n",
            ">2RAO_A Chain A, Hemoglobin subunit alpha-1/2 [Oryctolagus cuniculus]\n",
            " 2RAO_C Chain C, Hemoglobin subunit alpha-1/2 [Oryctolagus cuniculus]\n",
            "Length=141\n",
            "\n",
            " Score = 111 bits (277),  Expect = 1e-31, Method: Compositional matrix adjust.\n",
            " Identities = 59/145 (41%), Positives = 85/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ +   W K+     E G EA+ R+ + +P T+ +F  F      D   G+ ++\n",
            "Sbjct  2    LSPADKTNIKTAWEKIGSHGGEYGAEAVERMFLGFPTTKTYFPHF------DFTHGSEQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  + HLD+L G  +TLS+LH  KL VDP NF+LL + L+  LA+H   \n",
            "Sbjct  56   KAHGKKVSEALTKAVGHLDDLPGALSTLSDLHAHKLRVDPVNFKLLSHCLLVTLANHHPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLANVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">4YU3_A Chain A, hemoglobin [Helogale parvula]\n",
            " 4YU4_A Chain A, hemoglobin [Helogale parvula]\n",
            " 4YU4_C Chain C, hemoglobin [Helogale parvula]\n",
            "Length=141\n",
            "\n",
            " Score = 111 bits (277),  Expect = 1e-31, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ + A W K+     E G EAL R  + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNIKASWEKIGSHGGEYGAEALERTFLCFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A ++ + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVADALTNAVGHLDDLPGALSALSDLHAYKLRVDPVNFKLLSHCLLVTLASHHPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K ++ V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLSSVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">3HRW_A Chain A, Hemoglobin subunit alpha [Mus musculus]\n",
            " 3HRW_C Chain C, Hemoglobin subunit alpha [Mus musculus]\n",
            "Length=141\n",
            "\n",
            " Score = 111 bits (277),  Expect = 1e-31, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 83/145 (57%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ E+KS + A WGK+  +  E G EAL R+   +P T+ +F  F      D   G+ +V\n",
            "Sbjct  2    LSGEDKSNIKAAWGKIGGHGAEYGAEALERMFASFPTTKTYFPHF------DVSHGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A +    HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALASAAGHLDDLPGALSALSDLHAHKLRVDPVNFKLLSHCLLVTLASHHPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1ABW_A Chain A, HEMOGLOBIN-BASED BLOOD SUBSTITUTE [Homo sapiens]\n",
            " 1ABY_A Chain A, HEMOGLOBIN [Homo sapiens]\n",
            " 1C7C_A Chain A, PROTEIN (DEOXYHEMOGLOBIN (ALPHA CHAIN)) [Homo sapiens]\n",
            " 1O1P_A Chain A, Hemoglobin Alpha chain [Homo sapiens]\n",
            "Length=283\n",
            "\n",
            " Score = 114 bits (286),  Expect = 2e-31, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            " Score = 114 bits (286),  Expect = 2e-31, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  144  LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  197\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  198  KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  257\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  258  EFTPAVHASLDKFLASVSTVLTSKY  282\n",
            "\n",
            "\n",
            ">5WOG_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5WOG_B Chain B, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5WOH_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 5WOH_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=137\n",
            "\n",
            " Score = 110 bits (276),  Expect = 2e-31, Method: Compositional matrix adjust.\n",
            " Identities = 61/141 (43%), Positives = 86/141 (61%), Gaps = 8/141 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  1    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  54\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  55   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  114\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANAL  142\n",
            "            EFTP V A+  K +A V+  L\n",
            "Sbjct  115  EFTPAVHASLDKFLASVSTVL  135\n",
            "\n",
            "\n",
            ">1C7D_A Chain A, PROTEIN (DEOXYHEMOGLOBIN (ALPHA CHAIN)) [Homo sapiens]\n",
            "Length=284\n",
            "\n",
            " Score = 114 bits (286),  Expect = 2e-31, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            " Score = 114 bits (286),  Expect = 2e-31, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 88/145 (61%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  145  LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  198\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  199  KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  258\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  259  EFTPAVHASLDKFLASVSTVLTSKY  283\n",
            "\n",
            "\n",
            ">4MQH_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=139\n",
            "\n",
            " Score = 110 bits (276),  Expect = 2e-31, Method: Compositional matrix adjust.\n",
            " Identities = 61/144 (42%), Positives = 87/144 (60%), Gaps = 8/144 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKK+  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKMADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHK  145\n",
            "            EFTP V A+  K +A V+  L  K\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSK  139\n",
            "\n",
            "\n",
            ">3QJB_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3QJD_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3QJD_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 110 bits (276),  Expect = 2e-31, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGLGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1FHJ_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Chrysocyon brachyurus]\n",
            " 1FHJ_C Chain C, HEMOGLOBIN (ALPHA CHAIN) [Chrysocyon brachyurus]\n",
            " 2B7H_A Chain A, hemoglobin alpha chain [Cerdocyon thous]\n",
            " 2B7H_C Chain C, hemoglobin alpha chain [Cerdocyon thous]\n",
            "Length=141\n",
            "\n",
            " Score = 110 bits (275),  Expect = 2e-31, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 85/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ + + W K+  +  + GGEAL R    +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNIKSTWDKIGGHAGDYGGEALDRTFQSFPTTKTYFPHF-DLSP-----GSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  +AHLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVADALTTAVAHLDDLPGALSALSDLHAYKLRVDPVNFKLLSHCLLVTLACHHPT  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K    V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFFTAVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1O1I_A Chain A, Hemoglobin Alpha chain [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 110 bits (275),  Expect = 2e-31, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEAFERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGQGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">3NMM_A Chain A, Hemoglobin subunit alpha [Homo sapiens]\n",
            " 3NMM_C Chain C, Hemoglobin subunit alpha [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 110 bits (275),  Expect = 3e-31, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGWGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1J7S_A Chain A, Hemoglobin [Homo sapiens]\n",
            " 1J7S_C Chain C, Hemoglobin [Homo sapiens]\n",
            " 1J7Y_A Chain A, Hemoglobin [Homo sapiens]\n",
            " 1J7Y_C Chain C, Hemoglobin [Homo sapiens]\n",
            " 1QI8_A Chain A, HEMOGLOBIN [Homo sapiens]\n",
            " 1QI8_C Chain C, HEMOGLOBIN [Homo sapiens]\n",
            " 2W72_C Chain C, HUMAN HEMOGLOBIN A [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 110 bits (274),  Expect = 3e-31, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEAYERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGQGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">2W72_A Chain A, HUMAN HEMOGLOBIN A [Homo sapiens]\n",
            "Length=141\n",
            "\n",
            " Score = 110 bits (274),  Expect = 3e-31, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEAYERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGQGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">3CY5_A Chain A, Hemoglobin subunit alpha-2 [Bubalus bubalis]\n",
            " 3CY5_C Chain C, Hemoglobin subunit alpha-2 [Bubalus bubalis]\n",
            "Length=141\n",
            "\n",
            " Score = 109 bits (272),  Expect = 6e-31, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 85/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +KS + A WGKV  +  + G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKSNIQAAWGKVGGHAADYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG KV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGAKVANALTKAVGHLDDLPGALSELSDLHAHKLRVDPVNFKLLSHSLLVTLASHLPN  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">3LQD_A Chain A, Hemoglobin subunit alpha [Lepus europaeus]\n",
            " 3LQD_C Chain C, Hemoglobin subunit alpha [Lepus europaeus]\n",
            "Length=141\n",
            "\n",
            " Score = 109 bits (272),  Expect = 8e-31, Method: Compositional matrix adjust.\n",
            " Identities = 58/145 (40%), Positives = 84/145 (58%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ +   W K+     E G EA+ R+ + +P T+ +F  F      D   G+ ++\n",
            "Sbjct  2    LSPADKTNIKTAWEKIGSHGGEYGAEAVERMFLGFPTTKTYFPHF------DFTHGSEQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA+H   \n",
            "Sbjct  56   KAHGKKVSEALTKAVGHLDDLPGALSALSDLHAHKLRVDPVNFKLLSHCLLVTLANHHPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLANVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1S0H_A Chain A, Hemoglobin alpha chain [Equus asinus]\n",
            "Length=141\n",
            "\n",
            " Score = 108 bits (271),  Expect = 1e-30, Method: Compositional matrix adjust.\n",
            " Identities = 63/145 (43%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V A W KV  N  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKTNVKAAWSKVGGNAGEFGAEALERMFLGFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVGDALTLAVGHLDDLPGALSNLSDLHAHKLRVDPVNFKLLSHCLLSTLAVHLPN  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K ++ V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLSSVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1FSX_A Chain A, HEMOGLOBIN ALPHA CHAIN [Bos taurus]\n",
            " 1FSX_C Chain C, HEMOGLOBIN ALPHA CHAIN [Bos taurus]\n",
            " 1G08_A Chain A, HEMOGLOBIN ALPHA CHAIN [Bos taurus]\n",
            " 1G08_C Chain C, HEMOGLOBIN ALPHA CHAIN [Bos taurus]\n",
            " 1G09_A Chain A, HEMOGLOBIN ALPHA CHAIN [Bos taurus]\n",
            " 1G09_C Chain C, HEMOGLOBIN ALPHA CHAIN [Bos taurus]\n",
            " 1G0A_A Chain A, HEMOGLOBIN ALPHA CHAIN [Bos taurus]\n",
            " 1G0A_C Chain C, HEMOGLOBIN ALPHA CHAIN [Bos taurus]\n",
            " 1HDA_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Bos taurus]\n",
            " 1HDA_C Chain C, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Bos taurus]\n",
            " 2QSP_A Chain A, Hemoglobin subunit alpha [Bos taurus]\n",
            " 2QSP_C Chain C, Hemoglobin subunit alpha [Bos taurus]\n",
            " 2QSS_A Chain A, Hemoglobin subunit alpha [Bos taurus]\n",
            " 2QSS_C Chain C, Hemoglobin subunit alpha [Bos taurus]\n",
            " 3CIU_A Chain A, Hemoglobin subunit alpha [Bos taurus]\n",
            " 3CIU_C Chain C, Hemoglobin subunit alpha [Bos taurus]\n",
            " 3PI8_A Chain A, Hemoglobin subunit alpha [Bos taurus]\n",
            " 3PI8_C Chain C, Hemoglobin subunit alpha [Bos taurus]\n",
            " 3PI9_A Chain A, Hemoglobin subunit alpha [Bos taurus]\n",
            " 3PI9_C Chain C, Hemoglobin subunit alpha [Bos taurus]\n",
            " 3PIA_A Chain A, Hemoglobin subunit alpha [Bos taurus]\n",
            " 3PIA_C Chain C, Hemoglobin subunit alpha [Bos taurus]\n",
            "Length=141\n",
            "\n",
            " Score = 108 bits (271),  Expect = 1e-30, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 84/145 (58%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K  V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKGNVKAAWGKVGGHAAEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG KV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGAKVAAALTKAVEHLDDLPGALSELSDLHAHKLRVDPVNFKLLSHSLLVTLASHLPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLANVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">6IHX_A Chain A, Hemoglobin subunit alpha [Bos taurus]\n",
            " 6IHX_C Chain C, Hemoglobin subunit alpha [Bos taurus]\n",
            "Length=140\n",
            "\n",
            " Score = 108 bits (271),  Expect = 1e-30, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 84/145 (58%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K  V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKGNVKAAWGKVGGHAAEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG KV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGAKVAAALTKAVEHLDDLPGALSELSDLHAHKLRVDPVNFKLLSHSLLVTLASHLPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLANVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">5KER_A Chain A, Alpha-globin [Peromyscus maniculatus]\n",
            " 5KER_C Chain C, Alpha-globin [Peromyscus maniculatus]\n",
            " 5KER_E Chain E, Alpha-globin [Peromyscus maniculatus]\n",
            " 5KER_G Chain G, Alpha-globin [Peromyscus maniculatus]\n",
            "Length=141\n",
            "\n",
            " Score = 108 bits (270),  Expect = 2e-30, Method: Compositional matrix adjust.\n",
            " Identities = 59/145 (41%), Positives = 83/145 (57%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+ + A WGK+  +  E G EAL R+   +P T+ +F  F      D   G+ +V\n",
            "Sbjct  2    LSADDKANIKAAWGKIGGHGAEYGAEALERMFCSFPTTKTYFPHF------DVSPGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG KV GA +   +HLD+L    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGAKVAGALATAASHLDDLPAALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHHPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1O1N_A Chain A, Hemoglobin Alpha chain [Homo sapiens]\n",
            "Length=285\n",
            "\n",
            " Score = 112 bits (279),  Expect = 2e-30, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEAWERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            " Score = 112 bits (279),  Expect = 2e-30, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 87/145 (60%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  146  LSPADKTNVKAAWGKVGAHAGEYGAEAWERMFLSFPTTKTYFPHF-DLS-----HGSAQV  199\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  200  KGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  259\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  260  EFTPAVHASLDKFLASVSTVLTSKY  284\n",
            "\n",
            "\n",
            ">3D4X_A Chain A, Hemoglobin subunit alpha [Felis catus]\n",
            " 3D4X_C Chain C, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GQP_A Chain A, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GQP_C Chain C, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GQR_A Chain A, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GQR_C Chain C, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GQR_E Chain E, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GQR_G Chain G, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GYS_A Chain A, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GYS_C Chain C, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GYS_E Chain E, Hemoglobin subunit alpha [Felis catus]\n",
            " 3GYS_G Chain G, Hemoglobin subunit alpha [Felis catus]\n",
            "Length=141\n",
            "\n",
            " Score = 107 bits (266),  Expect = 6e-30, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 83/145 (57%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +KS V A WGK+  +  E G EAL R    +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKSNVKACWGKIGSHAGEYGAEALERTFCSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHG+KV  A +  +AH+D+L    + LS+LH  KL VDP NF+ L + L+  LA H   \n",
            "Sbjct  56   KAHGQKVADALTQAVAHMDDLPTAMSALSDLHAYKLRVDPVNFKFLSHCLLVTLACHHPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K  + V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFFSAVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1O1J_A Chain A, Hemoglobin Alpha chain [Homo sapiens]\n",
            "Length=283\n",
            "\n",
            " Score = 110 bits (276),  Expect = 6e-30, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEAFERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGQGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            " Score = 110 bits (276),  Expect = 6e-30, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  144  LSPADKTNVKAAWGKVGAHAGEYGAEAFERMFLSFPTTKTYFPHF-DLS-----HGSAQV  197\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  198  KGQGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  257\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  258  EFTPAVHASLDKFLASVSTVLTSKY  282\n",
            "\n",
            "\n",
            ">3BCQ_A Chain A, Alpha-chain hemoglobin [Brycon cephalus]\n",
            " 3BCQ_C Chain C, Alpha-chain hemoglobin [Brycon cephalus]\n",
            "Length=142\n",
            "\n",
            " Score = 107 bits (266),  Expect = 7e-30, Method: Compositional matrix adjust.\n",
            " Identities = 54/146 (37%), Positives = 91/146 (62%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVN--VDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K ++ A W K++   +++G +AL R+L VYP T+ +F  + DLS      G+  V\n",
            "Sbjct  2    LSDKDKDSIKAFWAKISPKAEDIGADALARMLTVYPQTKTYFSHWKDLSP-----GSAPV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGK V+G+ ++ ++ +D+L     TLSELH  +L VDP NF++L + L+ VLA  F  \n",
            "Sbjct  57   KKHGKTVMGSVAEAVSKIDDLTNGLLTLSELHAFQLRVDPANFKILSHNLLVVLAQQFPN  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP V  +  K ++ ++ +L+ KY \n",
            "Sbjct  117  DFTPEVHVSMDKFLSLLSWSLSEKYR  142\n",
            "\n",
            "\n",
            ">6SVA_A Chain A, Hemoglobin subunit alpha [Equus caballus]\n",
            " 8PUQ_A Chain A, Hemoglobin subunit alpha [Equus caballus]\n",
            "Length=140\n",
            "\n",
            " Score = 106 bits (265),  Expect = 8e-30, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V A W KV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKTNVKAAWSKVGGHAGEYGAEALERMFLGFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVGDALTLAVGHLDDLPGALSNLSDLHAHKLRVDPVNFKLLSHCLLSTLAVHLPN  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K ++ V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLSSVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1O1M_A Chain A, Hemoglobin Alpha chain [Homo sapiens]\n",
            "Length=285\n",
            "\n",
            " Score = 110 bits (275),  Expect = 8e-30, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEAFERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGQGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            " Score = 110 bits (275),  Expect = 8e-30, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  146  LSPADKTNVKAAWGKVGAHAGEYGAEAFERMFLSFPTTKTYFPHF-DLS-----HGSAQV  199\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  200  KGQGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  259\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  260  EFTPAVHASLDKFLASVSTVLTSKY  284\n",
            "\n",
            "\n",
            ">3DHR_A Chain A, Hemoglobin subunit alpha-A [Columba livia]\n",
            " 3DHR_C Chain C, Hemoglobin subunit alpha-A [Columba livia]\n",
            " 3DHR_E Chain E, Hemoglobin subunit alpha-A [Columba livia]\n",
            " 3DHR_G Chain G, Hemoglobin subunit alpha-A [Columba livia]\n",
            "Length=142\n",
            "\n",
            " Score = 106 bits (265),  Expect = 9e-30, Method: Compositional matrix adjust.\n",
            " Identities = 60/145 (41%), Positives = 84/145 (58%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +KS V A++ K+     ++GGEAL RL + YP T+ +F  F DLS      G+ ++\n",
            "Sbjct  3    LSANDKSNVKAVFAKIGGQAGDLGGEALERLFITYPQTKTYFPHF-DLS-----HGSAQI  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A  +   H+D++ G  + LS+LH  KL VDP NF+LLG+  + V+A HF  \n",
            "Sbjct  57   KGHGKKVAEALVEAANHIDDIAGALSKLSDLHAQKLRVDPVNFKLLGHCFLVVVAVHFPS  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K V  V   L  KY\n",
            "Sbjct  117  LLTPEVHASLDKFVLAVGTVLTAKY  141\n",
            "\n",
            "\n",
            ">4H2L_A Chain A, Alpha-globin [Peromyscus maniculatus]\n",
            "Length=141\n",
            "\n",
            " Score = 106 bits (265),  Expect = 9e-30, Method: Compositional matrix adjust.\n",
            " Identities = 58/145 (40%), Positives = 82/145 (57%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+ + A WGK+  +  E G EAL R+   +P T+ +F  F      D   G+ +V\n",
            "Sbjct  2    LSADDKANIKAAWGKIGGHGAEYGAEALERMFCSFPTTKTYFPHF------DVSHGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHG KV  A +    HLD+L    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGGKVADALATAAGHLDDLPAALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">2R80_A Chain A, Hemoglobin subunit alpha-A [Columba livia]\n",
            " 2R80_C Chain C, Hemoglobin subunit alpha-A [Columba livia]\n",
            " 3MJU_A Chain A, Hemoglobin subunit alpha-A [Columba livia]\n",
            "Length=141\n",
            "\n",
            " Score = 106 bits (265),  Expect = 9e-30, Method: Compositional matrix adjust.\n",
            " Identities = 60/145 (41%), Positives = 84/145 (58%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +KS V A++ K+     ++GGEAL RL + YP T+ +F  F DLS      G+ ++\n",
            "Sbjct  2    LSANDKSNVKAVFAKIGGQAGDLGGEALERLFITYPQTKTYFPHF-DLS-----HGSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A  +   H+D++ G  + LS+LH  KL VDP NF+LLG+  + V+A HF  \n",
            "Sbjct  56   KGHGKKVAEALVEAANHIDDIAGALSKLSDLHAQKLRVDPVNFKLLGHCFLVVVAVHFPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K V  V   L  KY\n",
            "Sbjct  116  LLTPEVHASLDKFVLAVGTVLTAKY  140\n",
            "\n",
            "\n",
            ">1IWH_A Chain A, Hemoglobin alpha chain [Equus caballus]\n",
            " 2D5X_A Chain A, Hemoglobin alpha subunit [Equus caballus]\n",
            " 5C6E_A Chain A, Hemoglobin subunit alpha [Equus caballus]\n",
            " 6R2O_A Chain A, Hemoglobin subunit alpha [Equus caballus]\n",
            " 6R2O_C Chain C, Hemoglobin subunit alpha [Equus caballus]\n",
            "Length=141\n",
            "\n",
            " Score = 106 bits (265),  Expect = 9e-30, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V A W KV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKTNVKAAWSKVGGHAGEYGAEALERMFLGFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVGDALTLAVGHLDDLPGALSNLSDLHAHKLRVDPVNFKLLSHCLLSTLAVHLPN  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K ++ V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLSSVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1O1L_A Chain A, Hemoglobin Alpha chain [Homo sapiens]\n",
            "Length=283\n",
            "\n",
            " Score = 109 bits (273),  Expect = 2e-29, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSPADKTNVKAAWGKVGAHAGEYGAEAWERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGQGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  EFTPAVHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            " Score = 109 bits (273),  Expect = 2e-29, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 86/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V A WGKV  +  E G EA  R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  144  LSPADKTNVKAAWGKVGAHAGEYGAEAWERMFLSFPTTKTYFPHF-DLS-----HGSAQV  197\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K  GKKV  A ++ +AH+D++    + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  198  KGQGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPA  257\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K +A V+  L  KY\n",
            "Sbjct  258  EFTPAVHASLDKFLASVSTVLTSKY  282\n",
            "\n",
            "\n",
            ">1IBE_A Chain A, HEMOGLOBIN (DEOXY) [Equus caballus]\n",
            " 8PUR_A Chain A, Hemoglobin, alpha 1 [Equus caballus]\n",
            "Length=141\n",
            "\n",
            " Score = 105 bits (262),  Expect = 2e-29, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 85/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V A W KV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKTNVKAAWSKVGGHAGEFGAEALERMFLGFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  + HLD+L G  + LS LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVGDALTLAVGHLDDLPGALSDLSNLHAHKLRVDPVNFKLLSHCLLSTLAVHLPN  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K ++ V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLSSVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1SPG_A Chain A, Hemoglobin [Leiostomus xanthurus]\n",
            "Length=144\n",
            "\n",
            " Score = 105 bits (262),  Expect = 2e-29, Method: Compositional matrix adjust.\n",
            " Identities = 56/146 (38%), Positives = 86/146 (59%), Gaps = 6/146 (4%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V ALW K+     E+G EALGR+LV +P T+ +F  +G    P      P+V\n",
            "Sbjct  3    LSATDKARVKALWDKIEGKSAELGAEALGRMLVSFPQTKIYFSEWGQDLGPQ----TPQV  58\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            + HG  ++ A    +  +DNL G  + LSELH  KL VDP NF++L + ++ V++ +F  \n",
            "Sbjct  59   RNHGAVIMAAVGKAVKSIDNLVGGLSQLSELHAFKLRVDPANFKILAHNIILVISMYFPG  118\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            +FTP V  +  K +A +A AL+ KY \n",
            "Sbjct  119  DFTPEVHLSVDKFLACLALALSEKYR  144\n",
            "\n",
            "\n",
            ">1G0B_A Chain A, HEMOGLOBIN ALPHA CHAIN [Equus caballus]\n",
            " 1NS6_A Chain A, Hemoglobin alpha subunit [Equus caballus]\n",
            " 1NS9_A Chain A, Hemoglobin alpha subunit [Equus caballus]\n",
            " 1Y8H_A Chain A, Hemoglobin alpha chains [Equus caballus]\n",
            " 1Y8H_C Chain C, Hemoglobin alpha chains [Equus caballus]\n",
            " 1Y8I_A Chain A, Hemoglobin alpha chains [Equus caballus]\n",
            " 1Y8I_C Chain C, Hemoglobin alpha chains [Equus caballus]\n",
            " 1Y8K_A Chain A, Hemoglobin alpha chains [Equus caballus]\n",
            " 1Y8K_C Chain C, Hemoglobin alpha chains [Equus caballus]\n",
            " 2MHB_A Chain A, Hemoglobin (aquo Met) (alpha Chain) [Equus caballus]\n",
            " 2ZLT_A Chain A, Hemoglobin subunit alpha [Equus caballus]\n",
            " 2ZLU_A Chain A, Hemoglobin subunit alpha [Equus caballus]\n",
            " 2ZLV_A Chain A, Hemoglobin subunit alpha [Equus caballus]\n",
            " 2ZLW_A Chain A, Hemoglobin subunit alpha [Equus caballus]\n",
            " 2ZLW_C Chain C, Hemoglobin subunit alpha [Equus caballus]\n",
            " 2ZLX_A Chain A, Hemoglobin subunit alpha [Equus caballus]\n",
            " 2ZLX_C Chain C, Hemoglobin subunit alpha [Equus caballus]\n",
            "Length=141\n",
            "\n",
            " Score = 105 bits (262),  Expect = 3e-29, Method: Compositional matrix adjust.\n",
            " Identities = 62/145 (43%), Positives = 85/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V A W KV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKTNVKAAWSKVGGHAGEYGAEALERMFLGFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  + HLD+L G  + LS LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVGDALTLAVGHLDDLPGALSDLSNLHAHKLRVDPVNFKLLSHCLLSTLAVHLPN  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K ++ V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLSSVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">6II1_A Chain A, Hemoglobin subunit alpha [Bos taurus]\n",
            " 6II1_C Chain C, Hemoglobin subunit alpha [Bos taurus]\n",
            "Length=138\n",
            "\n",
            " Score = 104 bits (260),  Expect = 4e-29, Method: Compositional matrix adjust.\n",
            " Identities = 60/141 (43%), Positives = 82/141 (58%), Gaps = 8/141 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K  V A WGKV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKGNVKAAWGKVGGHAAEYGAEALERMFLSFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG KV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KGHGAKVAAALTKAVEHLDDLPGALSELSDLHAHKLRVDPVNFKLLSHSLLVTLASHLPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANAL  142\n",
            "            +FTP V A+  K +A V+  L\n",
            "Sbjct  116  DFTPAVHASLDKFLANVSTVL  136\n",
            "\n",
            "\n",
            ">2DHB_A Chain A, HEMOGLOBIN (DEOXY) (ALPHA CHAIN) [Equus caballus]\n",
            "Length=141\n",
            "\n",
            " Score = 104 bits (260),  Expect = 5e-29, Method: Compositional matrix adjust.\n",
            " Identities = 61/145 (42%), Positives = 84/145 (58%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V A W KV  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSAADKTNVKAAWSKVGGHAGEYGAEALERMFLGFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV    +  + HLD+L G  + LS LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGKKVADGLTLAVGHLDDLPGALSDLSNLHAHKLRVDPVNFKLLSHCLLSTLAVHLPN  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K ++ V+  L  KY\n",
            "Sbjct  116  DFTPAVHASLDKFLSSVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1QPW_A Chain A, PORCINE HEMOGLOBIN (ALPHA SUBUNIT) [Sus scrofa]\n",
            " 1QPW_C Chain C, PORCINE HEMOGLOBIN (ALPHA SUBUNIT) [Sus scrofa]\n",
            " 2PGH_A Chain A, HEMOGLOBIN (AQUO MET) (ALPHA CHAIN) [Sus scrofa]\n",
            " 2PGH_C Chain C, HEMOGLOBIN (AQUO MET) (ALPHA CHAIN) [Sus scrofa]\n",
            " 4F4O_A Chain A, Hemoglobin subunit alpha [Sus scrofa]\n",
            " 4F4O_D Chain D, Hemoglobin subunit alpha [Sus scrofa]\n",
            " 4F4O_G Chain G, Hemoglobin subunit alpha [Sus scrofa]\n",
            " 4F4O_J Chain J, Hemoglobin subunit alpha [Sus scrofa]\n",
            "Length=141\n",
            "\n",
            " Score = 102 bits (255),  Expect = 2e-28, Method: Compositional matrix adjust.\n",
            " Identities = 60/145 (41%), Positives = 84/145 (58%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V A WGKV       G EAL R+ + +P T+ +F  F +LS      G+ +V\n",
            "Sbjct  2    LSAADKANVKAAWGKVGGQAGAHGAEALERMFLGFPTTKTYFPHF-NLS-----HGSDQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHG+KV  A +  + HLD+L G  + LS+LH  KL VDP NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGQKVADALTKAVGHLDDLPGALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHHPD  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F P V A+  K +A V+  L  KY\n",
            "Sbjct  116  DFNPSVHASLDKFLANVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1HDS_A Chain A, HEMOGLOBIN S (DEOXY) (ALPHA CHAIN) [Odocoileus virginianus]\n",
            " 1HDS_C Chain C, HEMOGLOBIN S (DEOXY) (ALPHA CHAIN) [Odocoileus virginianus]\n",
            "Length=141\n",
            "\n",
            " Score = 102 bits (255),  Expect = 3e-28, Method: Compositional matrix adjust.\n",
            " Identities = 60/145 (41%), Positives = 80/145 (55%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+   KS V A WGKV  N    G +AL R+ + +P T+ +F  F DLS      G+ + \n",
            "Sbjct  2    LSAANKSNVKAAWGKVGGNAPAYGAQALQRMFLSFPTTKTYFPHF-DLS-----HGSAQQ  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHG+KV  A +    HL++L GT + LS LH  KL V+P NF+LL + L+  LA H   \n",
            "Sbjct  56   KAHGQKVANALTKAQGHLNDLPGTLSNLSNLHAHKLRVNPVNFKLLSHSLLVTLASHLPT  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "             FTP V A   K +A  +  L  KY\n",
            "Sbjct  116  NFTPAVHANLNKFLANDSTVLTSKY  140\n",
            "\n",
            "\n",
            ">3MJP_A Chain A, Hemoglobin subunit alpha-A [Coturnix japonica]\n",
            " 3MJP_C Chain C, Hemoglobin subunit alpha-A [Coturnix japonica]\n",
            "Length=141\n",
            "\n",
            " Score = 102 bits (254),  Expect = 4e-28, Method: Compositional matrix adjust.\n",
            " Identities = 54/145 (37%), Positives = 79/145 (54%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V  ++ K+  + +E G EAL R+   YP T+ +F  F      D   G+ ++\n",
            "Sbjct  2    LSAADKTNVKGIFAKIAGHAEEYGAEALDRMFTTYPQTKTYFPHF------DVSHGSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A  +   H+D++ GT + LS+LH  KL VDP NF+LLG   + V+A H   \n",
            "Sbjct  56   KGHGKKVAAALVEAANHIDDIAGTLSKLSDLHAQKLRVDPVNFKLLGQCFLVVVAIHHPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K +  V   L  KY\n",
            "Sbjct  116  ALTPEVHASLDKFLCAVGTVLTAKY  140\n",
            "\n",
            "\n",
            ">3AT5_A Chain A, AlphaA-globin [Podocnemis unifilis]\n",
            " 3AT6_A Chain A, AlphaA-globin [Podocnemis unifilis]\n",
            "Length=141\n",
            "\n",
            " Score = 102 bits (254),  Expect = 4e-28, Method: Compositional matrix adjust.\n",
            " Identities = 54/145 (37%), Positives = 81/145 (56%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVN--VDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+P +K+ V  +W KV+  V++ G E L RL  VYP T+ +F  F      D    + ++\n",
            "Sbjct  2    LSPGDKANVKTVWSKVSGHVEDYGAETLERLFRVYPSTKTYFPHF------DLHHDSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            + HGKKVL A  + ++H+D++    + LS+LH   L VDP NF+LL +  + VLA H   \n",
            "Sbjct  56   RTHGKKVLTAIGEAVSHIDDIASALSKLSDLHAQTLRVDPVNFKLLSHSFLVVLAVHAPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V  +  K +  V+N L  KY\n",
            "Sbjct  116  LLTPEVHVSLDKFLVAVSNVLTSKY  140\n",
            "\n",
            "\n",
            ">2ZFB_A Chain A, Hemoglobin subunit alpha [Psittacula krameri]\n",
            "Length=141\n",
            "\n",
            " Score = 101 bits (252),  Expect = 7e-28, Method: Compositional matrix adjust.\n",
            " Identities = 55/146 (38%), Positives = 80/146 (55%), Gaps = 8/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V +++ K+    D+ G EAL R+ V YP T+ +F  F      D   G+ +V\n",
            "Sbjct  2    LSGTDKTNVKSIFSKIGGQADDYGAEALERMFVTYPQTKTYFPHF------DVSPGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV G  S+   H+D++  + + LS+LH  KL VDP NF+LLG   + V+A H   \n",
            "Sbjct  56   KAHGKKVAGGLSEAANHIDDIATSLSKLSDLHAQKLRVDPVNFKLLGQCFLVVVAIHNPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "              TP   A+  K +  V   L  KY \n",
            "Sbjct  116  ALTPEAHASLDKFLCAVGLVLTAKYR  141\n",
            "\n",
            "\n",
            ">3GDJ_A Chain A, Hemoglobin subunit alpha [Camelus dromedarius]\n",
            " 3GDJ_C Chain C, Hemoglobin subunit alpha [Camelus dromedarius]\n",
            "Length=141\n",
            "\n",
            " Score = 100 bits (250),  Expect = 1e-27, Method: Compositional matrix adjust.\n",
            " Identities = 59/145 (41%), Positives = 85/145 (59%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+ V   +GK+  +  E G EAL R+ + +P T+ +F  F DLS      G+ +V\n",
            "Sbjct  2    LSSKDKTNVKTAFGKIGGHAAEYGAEALERMFLGFPTTKTYFPHF-DLS-----HGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +    HLD+L    + LS+LH  KL VDP NF+LL + L+  +A H   \n",
            "Sbjct  56   KAHGKKVGDALTKAADHLDDLPSALSALSDLHAHKLRVDPVNFKLLSHCLLVTVAAHHPG  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP V A+  K +A V+  L  KY\n",
            "Sbjct  116  DFTPSVHASLDKFLANVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">3A59_A Chain A, Hemoglobin subunit alpha-A [Struthio camelus]\n",
            " 3A59_C Chain C, Hemoglobin subunit alpha-A [Struthio camelus]\n",
            " 3A59_E Chain E, Hemoglobin subunit alpha-A [Struthio camelus]\n",
            " 3A59_G Chain G, Hemoglobin subunit alpha-A [Struthio camelus]\n",
            " 3FS4_A Chain A, Hemoglobin subunit alpha-A [Struthio camelus]\n",
            " 3FS4_C Chain C, Hemoglobin subunit alpha-A [Struthio camelus]\n",
            "Length=141\n",
            "\n",
            " Score = 100 bits (249),  Expect = 2e-27, Method: Compositional matrix adjust.\n",
            " Identities = 53/145 (37%), Positives = 80/145 (55%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVN--VDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V  ++ K++   +E G E L R+ + YP T+ +F  F      D   G+ ++\n",
            "Sbjct  2    LSGTDKTNVKGIFSKISSHAEEYGAETLERMFITYPQTKTYFPHF------DLHHGSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A  + + H+D++ G  + LS+LH  KL VDP NF+LLG   + V+A H   \n",
            "Sbjct  56   KAHGKKVANALIEAVNHIDDISGALSKLSDLHAQKLRVDPVNFKLLGQCFLVVVAIHHPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K +  V   L  KY\n",
            "Sbjct  116  ALTPEVHASLDKFLCAVGAVLTAKY  140\n",
            "\n",
            "\n",
            ">3W4U_A Chain A, Hemoglobin subunit zeta [Homo sapiens]\n",
            " 3W4U_C Chain C, Hemoglobin subunit zeta [Homo sapiens]\n",
            " 3W4U_E Chain E, Hemoglobin subunit zeta [Homo sapiens]\n",
            "Length=142\n",
            "\n",
            " Score = 100 bits (249),  Expect = 2e-27, Method: Compositional matrix adjust.\n",
            " Identities = 52/145 (36%), Positives = 83/145 (57%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNV--DEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            LT  E++ + ++W K++   D +G E L RL + +P T+ +F  F      D   G+ ++\n",
            "Sbjct  3    LTKTERTIIVSMWAKISTQADTIGTETLERLFLSHPQTKTYFPHF------DLHPGSAQL  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHG KV+ A  D +  +D++ G  + LSELH   L VDP NF+LL + L+  LA  F  \n",
            "Sbjct  57   RAHGSKVVAAVGDAVKSIDDIGGALSKLSELHAYILRVDPVNFKLLSHCLLVTLAARFPA  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FT    AA+ K ++ V++ L  KY\n",
            "Sbjct  117  DFTAEAHAAWDKFLSVVSSVLTEKY  141\n",
            "\n",
            "\n",
            ">3WTG_A Chain A, Hemoglobin subunit alpha-A [Dromaius novaehollandiae]\n",
            " 3WTG_C Chain C, Hemoglobin subunit alpha-A [Dromaius novaehollandiae]\n",
            "Length=142\n",
            "\n",
            " Score = 100 bits (249),  Expect = 2e-27, Method: Compositional matrix adjust.\n",
            " Identities = 55/145 (38%), Positives = 79/145 (54%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+   +++ K+  + +E G E L RL   YP T+ +F  F      D   G+ +V\n",
            "Sbjct  3    LSAADKTNTKSVFAKIGPHAEEYGAETLERLFTTYPQTKTYFPHF------DLHHGSAQV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A  +   H+D++    + LS+LH  KL VDP NF+LLG   + V+A H   \n",
            "Sbjct  57   KAHGKKVAAALVEAANHIDDISTALSKLSDLHAQKLRVDPVNFKLLGQCFLVVVAIHHPS  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K +  VAN L  KY\n",
            "Sbjct  117  LLTPEVHASLDKFLCAVANVLTAKY  141\n",
            "\n",
            "\n",
            ">1JEB_A Chain A, HEMOGLOBIN ZETA CHAIN [Homo sapiens]\n",
            " 1JEB_C Chain C, HEMOGLOBIN ZETA CHAIN [Homo sapiens]\n",
            "Length=142\n",
            "\n",
            " Score = 100 bits (248),  Expect = 3e-27, Method: Compositional matrix adjust.\n",
            " Identities = 52/145 (36%), Positives = 83/145 (57%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNV--DEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            LT  E++ + ++W K++   D +G E L RL + +P T+ +F  F      D   G+ ++\n",
            "Sbjct  3    LTKTERTIIVSMWAKISTQADTIGTETLERLFLSHPQTKTYFPHF------DLHPGSAQL  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            +AHG KV+ A  D +  +D++ G  + LSELH   L VDP NF+LL + L+  LA  F  \n",
            "Sbjct  57   RAHGSKVVAAVGDAVKSIDDIGGALSKLSELHAYILRVDPVNFKLLSHCLLVTLAARFPA  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FT    AA+ K ++ V++ L  KY\n",
            "Sbjct  117  DFTAEAHAAWDKFLSVVSSVLTEKY  141\n",
            "\n",
            "\n",
            ">2QMB_A Chain A, Hemoglobin subunit alpha-A [Meleagris gallopavo]\n",
            " 2QMB_C Chain C, Hemoglobin subunit alpha-A [Meleagris gallopavo]\n",
            "Length=142\n",
            "\n",
            " Score = 100 bits (248),  Expect = 4e-27, Method: Compositional matrix adjust.\n",
            " Identities = 55/145 (38%), Positives = 82/145 (57%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V  ++ K+  + +E G E L R+ + YP T+ +F  F DLS      G+ ++\n",
            "Sbjct  3    LSAADKNNVKGIFTKIAGHAEEYGAETLERMFITYPPTKTYFPHF-DLS-----HGSAQI  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV+ A  +   H+D++ GT + LS+LH  KL VDP NF+LLG   + V+A H   \n",
            "Sbjct  57   KGHGKKVVAALIEAANHIDDIAGTLSKLSDLHAHKLRVDPVNFKLLGQCFLVVVAIHHPA  116\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K +  V   L  KY\n",
            "Sbjct  117  ALTPEVHASLDKFLCAVGTVLTAKY  141\n",
            "\n",
            "\n",
            ">3K8B_A Chain A, Hemoglobin subunit alpha-A [Meleagris gallopavo]\n",
            " 3K8B_C Chain C, Hemoglobin subunit alpha-A [Meleagris gallopavo]\n",
            " 6ZMX_A Chain A, Hemoglobin subunit alpha-A [Meleagris gallopavo]\n",
            " 6ZMX_C Chain C, Hemoglobin subunit alpha-A [Meleagris gallopavo]\n",
            " 6ZMY_A Chain A, Hemoglobin subunit alpha-A [Meleagris gallopavo]\n",
            " 6ZMY_C Chain C, Hemoglobin subunit alpha-A [Meleagris gallopavo]\n",
            "Length=141\n",
            "\n",
            " Score = 99.8 bits (247),  Expect = 4e-27, Method: Compositional matrix adjust.\n",
            " Identities = 55/145 (38%), Positives = 82/145 (57%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V  ++ K+  + +E G E L R+ + YP T+ +F  F DLS      G+ ++\n",
            "Sbjct  2    LSAADKNNVKGIFTKIAGHAEEYGAETLERMFITYPPTKTYFPHF-DLS-----HGSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV+ A  +   H+D++ GT + LS+LH  KL VDP NF+LLG   + V+A H   \n",
            "Sbjct  56   KGHGKKVVAALIEAANHIDDIAGTLSKLSDLHAHKLRVDPVNFKLLGQCFLVVVAIHHPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K +  V   L  KY\n",
            "Sbjct  116  ALTPEVHASLDKFLCAVGTVLTAKY  140\n",
            "\n",
            "\n",
            ">3VRE_A Chain A, Hemoglobin subunit alpha [Mammuthus primigenius]\n",
            " 3VRE_C Chain C, Hemoglobin subunit alpha [Mammuthus primigenius]\n",
            " 3VRF_A Chain A, Hemoglobin subunit alpha [Mammuthus primigenius]\n",
            " 3VRG_A Chain A, Hemoglobin subunit alpha [Mammuthus primigenius]\n",
            "Length=141\n",
            "\n",
            " Score = 99.8 bits (247),  Expect = 4e-27, Method: Compositional matrix adjust.\n",
            " Identities = 56/145 (39%), Positives = 80/145 (55%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V A W KV  +  +   EAL R+   +P T+ +F  F      D   G+ +V\n",
            "Sbjct  2    LSDNDKTNVKATWSKVGDHASDYVAEALERMFFSFPTTKTYFPHF------DLSHGSGQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGKKV  A +  + HLD+L    + LS+LH  KL VDP NF+LL + L+  L+ H   \n",
            "Sbjct  56   KGHGKKVGEALTQAVGHLDDLPSALSALSDLHAHKLRVDPVNFKLLSHCLLVTLSSHQPT  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EFTP V A+  K ++ V+  L  KY\n",
            "Sbjct  116  EFTPEVHASLDKFLSNVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">3EOK_A Chain A, Hemoglobin subunit alpha-A [Anas platyrhynchos]\n",
            "Length=141\n",
            "\n",
            " Score = 99.0 bits (245),  Expect = 8e-27, Method: Compositional matrix adjust.\n",
            " Identities = 54/145 (37%), Positives = 82/145 (57%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V  ++ K+  + +E G E L R+ + YP T+ +F  F DLS      G+ ++\n",
            "Sbjct  2    LSAADKTNVKGVFSKIGGHAEEYGAETLERMFIAYPQTKTYFPHF-DLS-----HGSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A  + + H+D++ G  + LS+LH  KL VDP NF+ LG+  + V+A H   \n",
            "Sbjct  56   KAHGKKVAAALVEAVNHVDDIAGALSKLSDLHAQKLRVDPVNFKFLGHCFLVVVAIHHPA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K +  V   L  KY\n",
            "Sbjct  116  ALTPEVHASLDKFMCAVGAVLTAKY  140\n",
            "\n",
            "\n",
            ">3A0G_A Chain A, Hemoglobin subunit alpha [Cavia porcellus]\n",
            " 3HYU_A Chain A, Hemoglobin subunit alpha [Cavia porcellus]\n",
            "Length=141\n",
            "\n",
            " Score = 99.0 bits (245),  Expect = 8e-27, Method: Compositional matrix adjust.\n",
            " Identities = 53/145 (37%), Positives = 77/145 (53%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V   W K+  +  E   E L R+   +P T+ +F         D   G+  +\n",
            "Sbjct  2    LSAADKNNVKTTWDKIGGHAAEYVAEGLTRMFTSFPTTKTYFHHI------DVSPGSGDI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +  + HLD+L    +TLS++H  KL VDP NF+ L + L+  LA H G \n",
            "Sbjct  56   KAHGKKVADALTTAVGHLDDLPTALSTLSDVHAHKLRVDPVNFKFLNHCLLVTLAAHLGA  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP + A+  K  A V+  L  KY\n",
            "Sbjct  116  DFTPSIHASLDKFFASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">1FAW_A Chain A, HEMOGLOBIN (ALPHA SUBUNIT) [Anser anser]\n",
            " 1FAW_C Chain C, HEMOGLOBIN (ALPHA SUBUNIT) [Anser anser]\n",
            "Length=141\n",
            "\n",
            " Score = 98.6 bits (244),  Expect = 1e-26, Method: Compositional matrix adjust.\n",
            " Identities = 52/145 (36%), Positives = 79/145 (54%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V  ++ K+  + +E G E L R+   YP T+ +F  F      D   G+ ++\n",
            "Sbjct  2    LSAADKTNVKGVFSKIGGHAEEYGAETLERMFTAYPQTKTYFPHF------DLQHGSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A  + + H+D++ G  + LS+LH  KL VDP NF+ LG+  + V+A H   \n",
            "Sbjct  56   KAHGKKVAAALVEAVNHIDDIAGALSKLSDLHAQKLRVDPVNFKFLGHCFLVVVAIHHPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K +  V   L  KY\n",
            "Sbjct  116  ALTPEVHASLDKFLCAVGTVLTAKY  140\n",
            "\n",
            "\n",
            ">1GCV_B Chain B, HEMOGLOBIN [Mustelus griseus]\n",
            " 1GCV_D Chain D, HEMOGLOBIN [Mustelus griseus]\n",
            "Length=136\n",
            "\n",
            " Score = 97.8 bits (242),  Expect = 2e-26, Method: Compositional matrix adjust.\n",
            " Identities = 49/146 (34%), Positives = 77/146 (53%), Gaps = 10/146 (7%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH T EE+  ++  +   ++  V  +AL R+  VYPWT R+F+   D  +          \n",
            "Sbjct  1    VHWTQEERDEISKTFQGTDMKTVVTQALDRMFKVYPWTNRYFQKRTDFRSS---------  51\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "              H   V+GA  D + H+D++K  F  LS+ H D LHVDP +F LL + ++  LA+    \n",
            "Sbjct  52   -IHAGIVVGALQDAVKHMDDVKTLFKDLSKKHADDLHVDPGSFHLLTDCIIVELAYLRKD  110\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             FTP +Q  + K    V +A++ +YH\n",
            "Sbjct  111  CFTPHIQGIWDKFFEVVIDAISKQYH  136\n",
            "\n",
            "\n",
            ">3WR1_A Chain A, Hemoglobin subunit alpha-A [Phalacrocorax carbo]\n",
            "Length=141\n",
            "\n",
            " Score = 97.8 bits (242),  Expect = 3e-26, Method: Compositional matrix adjust.\n",
            " Identities = 54/145 (37%), Positives = 79/145 (54%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V  ++ KV  + +  G E L R+   YP T+ +F  F      D   G+ ++\n",
            "Sbjct  2    LSASDKTNVKGVFAKVGGSAEAYGAETLERMFTAYPQTKTYFPHF------DLHHGSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A  +   H+D++ G  + LS+LH  KL VDP NF+LLG+  + V+A H   \n",
            "Sbjct  56   KAHGKKVAAALVEAANHIDDIAGALSKLSDLHAQKLRVDPVNFKLLGHCFLVVVAIHHPT  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              TP V A+  K +  VA  L  KY\n",
            "Sbjct  116  LLTPEVHASLDKFMCAVAKELTAKY  140\n",
            "\n",
            "\n",
            ">1A4F_A Chain A, HEMOGLOBIN (ALPHA CHAIN) [Anser indicus]\n",
            " 1HV4_A Chain A, HEMOGLOBIN ALPHA-A CHAIN [Anser indicus]\n",
            " 1HV4_C Chain C, HEMOGLOBIN ALPHA-A CHAIN [Anser indicus]\n",
            " 1HV4_E Chain E, HEMOGLOBIN ALPHA-A CHAIN [Anser indicus]\n",
            " 1HV4_G Chain G, HEMOGLOBIN ALPHA-A CHAIN [Anser indicus]\n",
            "Length=141\n",
            "\n",
            " Score = 96.7 bits (239),  Expect = 6e-26, Method: Compositional matrix adjust.\n",
            " Identities = 51/145 (35%), Positives = 79/145 (54%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVN--VDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V  ++ K++   +E G E L R+   YP T+ +F  F      D   G+ ++\n",
            "Sbjct  2    LSAADKTNVKGVFSKISGHAEEYGAETLERMFTAYPQTKTYFPHF------DLQHGSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+ A  + + H+D++ G  + LS+LH  KL VDP NF+ LG+  + V+A H   \n",
            "Sbjct  56   KAHGKKVVAALVEAVNHIDDIAGALSKLSDLHAQKLRVDPVNFKFLGHCFLVVVAIHHPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              T  V A+  K +  V   L  KY\n",
            "Sbjct  116  ALTAEVHASLDKFLCAVGTVLTAKY  140\n",
            "\n",
            "\n",
            ">1C40_A Chain A, PROTEIN (HEMOGLOBIN (ALPHA CHAIN)) [Anser indicus]\n",
            "Length=141\n",
            "\n",
            " Score = 96.3 bits (238),  Expect = 1e-25, Method: Compositional matrix adjust.\n",
            " Identities = 51/145 (35%), Positives = 78/145 (54%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVN--VDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  +K+ V  ++ K++   +E G E L R+   YP T+ +F  F      D   G+ ++\n",
            "Sbjct  2    LSAADKTNVKGVFSKISGHAEEYGAETLERMFTAYPQTKTYFPHF------DLQHGSAQI  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV+ A  + + H+D++ G  + LS LH  KL VDP NF+ LG+  + V+A H   \n",
            "Sbjct  56   KAHGKKVVAALVEAVNHIDDIAGALSKLSNLHAQKLRVDPVNFKFLGHCFLVVVAIHHPS  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "              T  V A+  K +  V   L  KY\n",
            "Sbjct  116  ALTAEVHASLDKFLCAVGTVLTAKY  140\n",
            "\n",
            "\n",
            ">6HIT_A Chain A, Hemoglobin alpha 2 chain [Gadus morhua]\n",
            " 6HIT_C Chain C, Hemoglobin alpha 2 chain [Gadus morhua]\n",
            " 6HIT_E Chain E, Hemoglobin alpha 2 chain [Gadus morhua]\n",
            " 6HIT_G Chain G, Hemoglobin alpha 2 chain [Gadus morhua]\n",
            "Length=143\n",
            "\n",
            " Score = 95.9 bits (237),  Expect = 2e-25, Method: Compositional matrix adjust.\n",
            " Identities = 56/146 (38%), Positives = 86/146 (59%), Gaps = 7/146 (5%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNV--DEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+ V   + K++   D++G EAL RL+ VYP T+ +F  + D S      G+  V\n",
            "Sbjct  3    LSSKQKATVKDFFSKMSTRSDDIGAEALSRLVAVYPQTKSYFSHWKDASP-----GSAPV  57\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            + HG  ++G   D +  +D+LKG   +LSELH   L VDP NF+LL + ++  ++  F +\n",
            "Sbjct  58   RKHGITIMGGVYDAVGKIDDLKGGLLSLSELHAFMLRVDPVNFKLLAHCMLVCMSMIFPE  117\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            EFTP V  A  K +A +A ALA KY \n",
            "Sbjct  118  EFTPQVHVAVDKFLAQLALALAEKYR  143\n",
            "\n",
            "\n",
            ">1GCW_B Chain B, PROTEIN (HEMOGLOBIN) [Mustelus griseus]\n",
            " 1GCW_D Chain D, PROTEIN (HEMOGLOBIN) [Mustelus griseus]\n",
            "Length=135\n",
            "\n",
            " Score = 95.5 bits (236),  Expect = 2e-25, Method: Compositional matrix adjust.\n",
            " Identities = 48/145 (33%), Positives = 76/145 (52%), Gaps = 10/145 (7%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH T EE+  ++  +   ++  V  +AL R+  VYPWT R+F+   D  +          \n",
            "Sbjct  1    VHWTQEERDEISKTFQGTDMKTVVTQALDRMFKVYPWTNRYFQKRTDFRSS---------  51\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "              H   V+GA  D + H+D++K  F  LS+ H D LHVDP +F LL + ++  LA+    \n",
            "Sbjct  52   -IHAGIVVGALQDAVKHMDDVKTLFKDLSKKHADDLHVDPGSFHLLTDCIIVELAYLRKD  110\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "             FTP +Q  + K    V +A++ +Y\n",
            "Sbjct  111  CFTPHIQGIWDKFFEVVIDAISKQY  135\n",
            "\n",
            "\n",
            ">3DHT_A Chain A, Hemoglobin subunit alpha-1/2 [Rattus norvegicus]\n",
            " 3HF4_A Chain A, Hemoglobin subunit alpha-1/2 [Rattus norvegicus]\n",
            " 3HF4_E Chain E, Hemoglobin subunit alpha-1/2 [Rattus norvegicus]\n",
            "Length=141\n",
            "\n",
            " Score = 95.1 bits (235),  Expect = 3e-25, Method: Compositional matrix adjust.\n",
            " Identities = 52/145 (36%), Positives = 78/145 (54%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVGGE--ALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ ++K+ +   WGK+          AL R+   +P T+ +F         D   G+ +V\n",
            "Sbjct  2    LSADDKTNIKNCWGKIGGHGGEYGEEALQRMFAAFPTTKTYFSHI------DVSPGSAQV  55\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            KAHGKKV  A +    H+++L G  +TLS+LH  KL VDP NF+ L + L+  LA H   \n",
            "Sbjct  56   KAHGKKVADALAKAADHVEDLPGALSTLSDLHAHKLRVDPVNFKFLSHCLLVTLACHHPG  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +FTP + A+  K +A V+  L  KY\n",
            "Sbjct  116  DFTPAMHASLDKFLASVSTVLTSKY  140\n",
            "\n",
            "\n",
            ">3MKB_B Chain B, Hemoglobin subunit beta [Isurus oxyrinchus]\n",
            " 3MKB_D Chain D, Hemoglobin subunit beta [Isurus oxyrinchus]\n",
            "Length=136\n",
            "\n",
            " Score = 94.7 bits (234),  Expect = 4e-25, Method: Compositional matrix adjust.\n",
            " Identities = 50/146 (34%), Positives = 73/146 (50%), Gaps = 10/146 (7%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            VH T EE+  +   +   N   +G +AL R+ VV+PWT  +F      S           \n",
            "Sbjct  1    VHWTQEERDEIVKTFFSANSSAIGTKALERMFVVFPWTNAYFAKXXXFSAS---------  51\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "              H   V+GA  D + H D++K  F  +S+ H DKLH+DP +F LL +  +  LAH    \n",
            "Sbjct  52   -IHAAIVVGALQDAVKHEDDVKAEFVNISKAHADKLHIDPGSFHLLTDSFIVELAHLKKV  110\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             FTP V A + K    V +A++ +YH\n",
            "Sbjct  111  AFTPFVFAVWIKFFQVVIDAISSQYH  136\n",
            "\n",
            "\n",
            ">1CG5_B Chain B, PROTEIN (HEMOGLOBIN) [Hemitrygon akajei]\n",
            " 1CG8_B Chain B, PROTEIN (HEMOGLOBIN) [Hemitrygon akajei]\n",
            "Length=141\n",
            "\n",
            " Score = 82.8 bits (203),  Expect = 2e-20, Method: Compositional matrix adjust.\n",
            " Identities = 48/146 (33%), Positives = 74/146 (51%), Gaps = 5/146 (3%)\n",
            "\n",
            "Query  2    VHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            V L+ +++  +  +W  V+  ++  +AL R+ VVYPWT R F     L + + +     V\n",
            "Sbjct  1    VKLSEDQEHYIKGVWKDVDHKQITAKALERVFVVYPWTTRLFSKLQGLFSANDI----GV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            + H  KV  A  + +  L  ++  F  LS  H + + VD +NF+LLG   +  LA H+ K\n",
            "Sbjct  57   QQHADKVQRALGEAIDDLKKVEINFQNLSGKHQE-IGVDTQNFKLLGQTFMVELALHYKK  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "             F P   AA  K    VA AL+  YH\n",
            "Sbjct  116  TFRPKEHAAAYKFFRLVAEALSSNYH  141\n",
            "\n",
            "\n",
            ">1GCV_A Chain A, HEMOGLOBIN [Mustelus griseus]\n",
            " 1GCV_C Chain C, HEMOGLOBIN [Mustelus griseus]\n",
            " 1GCW_A Chain A, PROTEIN (HEMOGLOBIN) [Mustelus griseus]\n",
            " 1GCW_C Chain C, PROTEIN (HEMOGLOBIN) [Mustelus griseus]\n",
            "Length=140\n",
            "\n",
            " Score = 75.9 bits (185),  Expect = 8e-18, Method: Compositional matrix adjust.\n",
            " Identities = 43/122 (35%), Positives = 64/122 (52%), Gaps = 7/122 (6%)\n",
            "\n",
            "Query  25   GGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHGKKVLGAFSDGLAHLDNLKG  84\n",
            "            G E L RL V +P ++ +FE + D S   A     KV+ HG KV+ A      H+D+L  \n",
            "Sbjct  25   GAECLARLFVTHPGSKSYFE-YKDYSAAGA-----KVQVHGGKVIRAVVKAAEHVDDLHS  78\n",
            "\n",
            "Query  85   TFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEFTPPVQAAYQKVVAGVANALAH  144\n",
            "               TL+  H  KL VDP+NF +L   ++  LA H   EF+P    A  K+++ +   L+ \n",
            "Sbjct  79   HLETLALTHGKKLLVDPQNFPMLSECIIVTLATHL-TEFSPDTHCAVDKLLSAICQELSS  137\n",
            "\n",
            "Query  145  KY  146\n",
            "            +Y\n",
            "Sbjct  138  RY  139\n",
            "\n",
            "\n",
            ">1UMO_A Chain A, CYTOGLOBIN [Homo sapiens]\n",
            " 1UMO_B Chain B, CYTOGLOBIN [Homo sapiens]\n",
            " 1URV_A Chain A, CYTOGLOBIN [Homo sapiens]\n",
            " 1URV_B Chain B, CYTOGLOBIN [Homo sapiens]\n",
            " 1URY_A Chain A, CYTOGLOBIN [Homo sapiens]\n",
            " 1URY_B Chain B, CYTOGLOBIN [Homo sapiens]\n",
            " 1UT0_A Chain A, CYTOGLOBIN [Homo sapiens]\n",
            " 1UT0_B Chain B, CYTOGLOBIN [Homo sapiens]\n",
            " 1UX9_A Chain A, CYTOGLOBIN [Homo sapiens]\n",
            " 1UX9_B Chain B, CYTOGLOBIN [Homo sapiens]\n",
            "Length=190\n",
            "\n",
            " Score = 73.6 bits (179),  Expect = 2e-16, Method: Compositional matrix adjust.\n",
            " Identities = 38/132 (29%), Positives = 71/132 (54%), Gaps = 5/132 (4%)\n",
            "\n",
            "Query  8    EKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHG  65\n",
            "            E+ AV A+W ++  N ++VG   L R  V +P  +++F  F  +  P  +  +P+++ H \n",
            "Sbjct  23   ERKAVQAMWARLYANSEDVGVAILVRFFVNFPSAKQYFSQFKHMEDPLEMERSPQLRKHA  82\n",
            "\n",
            "Query  66   KKVLGAFSDGLAHL---DNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE  122\n",
            "             +V+GA +  + +L   D +    A + + H  K  V+P  F++L  V++ V+A  F  +\n",
            "Sbjct  83   SRVMGALNTVVENLHDPDKVSSVLALVGKAHALKHKVEPVYFKILSGVILEVVAEEFASD  142\n",
            "\n",
            "Query  123  FTPPVQAAYQKV  134\n",
            "            F P  Q A+ K+\n",
            "Sbjct  143  FPPETQRAWAKL  154\n",
            "\n",
            "\n",
            ">1V5H_A Chain A, Cytoglobin [Homo sapiens]\n",
            " 2DC3_A Chain A, Cytoglobin [Homo sapiens]\n",
            " 2DC3_B Chain B, Cytoglobin [Homo sapiens]\n",
            "Length=193\n",
            "\n",
            " Score = 72.8 bits (177),  Expect = 4e-16, Method: Compositional matrix adjust.\n",
            " Identities = 38/132 (29%), Positives = 71/132 (54%), Gaps = 5/132 (4%)\n",
            "\n",
            "Query  8    EKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHG  65\n",
            "            E+ AV A+W ++  N ++VG   L R  V +P  +++F  F  +  P  +  +P+++ H \n",
            "Sbjct  26   ERKAVQAMWARLYANCEDVGVAILVRFFVNFPSAKQYFSQFKHMEDPLEMERSPQLRKHA  85\n",
            "\n",
            "Query  66   KKVLGAFSDGLAHL---DNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE  122\n",
            "             +V+GA +  + +L   D +    A + + H  K  V+P  F++L  V++ V+A  F  +\n",
            "Sbjct  86   CRVMGALNTVVENLHDPDKVSSVLALVGKAHALKHKVEPVYFKILSGVILEVVAEEFASD  145\n",
            "\n",
            "Query  123  FTPPVQAAYQKV  134\n",
            "            F P  Q A+ K+\n",
            "Sbjct  146  FPPETQRAWAKL  157\n",
            "\n",
            "\n",
            ">3AG0_A Chain A, Cytoglobin [Homo sapiens]\n",
            "Length=192\n",
            "\n",
            " Score = 72.4 bits (176),  Expect = 5e-16, Method: Compositional matrix adjust.\n",
            " Identities = 38/132 (29%), Positives = 71/132 (54%), Gaps = 5/132 (4%)\n",
            "\n",
            "Query  8    EKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHG  65\n",
            "            E+ AV A+W ++  N ++VG   L R  V +P  +++F  F  +  P  +  +P+++ H \n",
            "Sbjct  25   ERKAVQAMWARLYANCEDVGVAILVRFFVNFPSAKQYFSQFKHMEDPLEMERSPQLRKHA  84\n",
            "\n",
            "Query  66   KKVLGAFSDGLAHL---DNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE  122\n",
            "             +V+GA +  + +L   D +    A + + H  K  V+P  F++L  V++ V+A  F  +\n",
            "Sbjct  85   CRVMGALNTVVENLHDPDKVSSVLALVGKAHALKHKVEPVYFKILSGVILEVVAEEFASD  144\n",
            "\n",
            "Query  123  FTPPVQAAYQKV  134\n",
            "            F P  Q A+ K+\n",
            "Sbjct  145  FPPETQRAWAKL  156\n",
            "\n",
            "\n",
            ">3MKB_A Chain A, Hemoglobin subunit alpha [Isurus oxyrinchus]\n",
            " 3MKB_C Chain C, Hemoglobin subunit alpha [Isurus oxyrinchus]\n",
            "Length=140\n",
            "\n",
            " Score = 71.2 bits (173),  Expect = 5e-16, Method: Compositional matrix adjust.\n",
            " Identities = 42/122 (34%), Positives = 62/122 (51%), Gaps = 7/122 (6%)\n",
            "\n",
            "Query  25   GGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHGKKVLGAFSDGLAHLDNLKG  84\n",
            "            G EAL RL   +P  + +F+ + D S   A     KV+ HG KV+ A      H D+L  \n",
            "Sbjct  25   GAEALARLFATHPGAKSYFD-YADYSAAGA-----KVQLHGGKVIRAVVSAAEHDDDLHA  78\n",
            "\n",
            "Query  85   TFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKEFTPPVQAAYQKVVAGVANALAH  144\n",
            "                L+  H  KL VDP NF +L   ++  LA H   EF+P    A  K+++ +++ L+ \n",
            "Sbjct  79   HLMVLAVTHGKKLLVDPSNFPMLSECILVTLATHLA-EFSPATHCAVDKLLSAISSELSS  137\n",
            "\n",
            "Query  145  KY  146\n",
            "            KY\n",
            "Sbjct  138  KY  139\n",
            "\n",
            "\n",
            ">4B3W_A Chain A, CYTOGLOBIN [Homo sapiens]\n",
            " 4B3W_B Chain B, CYTOGLOBIN [Homo sapiens]\n",
            "Length=190\n",
            "\n",
            " Score = 71.2 bits (173),  Expect = 2e-15, Method: Compositional matrix adjust.\n",
            " Identities = 37/132 (28%), Positives = 70/132 (53%), Gaps = 5/132 (4%)\n",
            "\n",
            "Query  8    EKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHG  65\n",
            "            E+ AV A+W ++  N ++VG   L R  V +P  +++F  F  +  P  +  +P+++   \n",
            "Sbjct  23   ERKAVQAMWARLYANSEDVGVAILVRFFVNFPSAKQYFSQFKHMEDPLEMERSPQLRKQA  82\n",
            "\n",
            "Query  66   KKVLGAFSDGLAHL---DNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGKE  122\n",
            "             +V+GA +  + +L   D +    A + + H  K  V+P  F++L  V++ V+A  F  +\n",
            "Sbjct  83   SRVMGALNTVVENLHDPDKVSSVLALVGKAHALKHKVEPVYFKILSGVILEVVAEEFASD  142\n",
            "\n",
            "Query  123  FTPPVQAAYQKV  134\n",
            "            F P  Q A+ K+\n",
            "Sbjct  143  FPPETQRAWAKL  154\n",
            "\n",
            "\n",
            ">6Q6P_A Chain A, Cytoglobin [Dissostichus mawsoni]\n",
            " 6Q6P_B Chain B, Cytoglobin [Dissostichus mawsoni]\n",
            "Length=179\n",
            "\n",
            " Score = 70.1 bits (170),  Expect = 3e-15, Method: Compositional matrix adjust.\n",
            " Identities = 41/149 (28%), Positives = 76/149 (51%), Gaps = 5/149 (3%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKV--NVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            LT +E+  +   W KV  N D+ G   L RL V +P ++++F  F  +  P+ +  + ++\n",
            "Sbjct  19   LTDKERVMIQDSWAKVYENSDDTGVAILVRLFVNFPSSRQYFSQFKHIEEPEELERSAQL  78\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSEL---HCDKLHVDPENFRLLGNVLVCVLAHH  118\n",
            "            + H  +V+   +  +  LDN +   + L  L   H  +  V+P  F++L  V++ VL   \n",
            "Sbjct  79   RKHANRVMNGLNTLVESLDNSEKVASVLKLLGKAHALRHKVEPVYFKILSGVILEVLGEA  138\n",
            "\n",
            "Query  119  FGKEFTPPVQAAYQKVVAGVANALAHKYH  147\n",
            "            F +  TP V AA+ K++A + + +   Y \n",
            "Sbjct  139  FSEVVTPEVAAAWTKLLATIYSGINAVYE  167\n",
            "\n",
            "\n",
            ">1LHS_A Chain A, MYOGLOBIN [Caretta caretta]\n",
            " 1LHT_A Chain A, MYOGLOBIN [Caretta caretta]\n",
            "Length=153\n",
            "\n",
            " Score = 68.9 bits (167),  Expect = 5e-15, Method: Compositional matrix adjust.\n",
            " Identities = 45/145 (31%), Positives = 70/145 (48%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEV--GGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ +E + V  +W KV  D    G E + RL  ++P TQ  F  F +L+T DA+  + +V\n",
            "Sbjct  2    LSDDEWNHVLGIWAKVEPDLSAHGQEVIIRLFQLHPETQERFAKFKNLTTIDALKSSEEV  61\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  VL A    L   +N +     L+E H  K  +  +    +  ++V V+A     \n",
            "Sbjct  62   KKHGTTVLTALGRILKQKNNHEQELKPLAESHATKHKIPVKYLEFICEIIVKVIAEKHPS  121\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F    QAA +K +    N +A KY\n",
            "Sbjct  122  DFGADSQAAMKKALELFRNDMASKY  146\n",
            "\n",
            "\n",
            ">1CG5_A Chain A, PROTEIN (HEMOGLOBIN) [Hemitrygon akajei]\n",
            " 1CG8_A Chain A, PROTEIN (HEMOGLOBIN) [Hemitrygon akajei]\n",
            "Length=141\n",
            "\n",
            " Score = 67.0 bits (162),  Expect = 2e-14, Method: Compositional matrix adjust.\n",
            " Identities = 50/145 (34%), Positives = 71/145 (49%), Gaps = 8/145 (6%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWG--KVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+ + K A+  L    K N +  G +AL RL  ++P T+ +F  F           N +V\n",
            "Sbjct  2    LSSQNKKAIEELGNLIKANAEAWGADALARLFELHPQTKTYFSKFSGFEA-----CNEQV  56\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HGK+V+ A +D   HLDNL      L+  H + L VDP NF L  + +V  LA +  +\n",
            "Sbjct  57   KKHGKRVMNALADATHHLDNLHLHLEDLARKHGENLLVDPHNFHLFADCIVVTLAVNL-Q  115\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "             FTP    A  K +  VA  L+  Y\n",
            "Sbjct  116  AFTPVTHCAVDKFLELVAYELSSCY  140\n",
            "\n",
            "\n",
            ">3QM5_A Chain A, Myoglobin [Thunnus atlanticus]\n",
            " 3QM6_A Chain A, Myoglobin [Thunnus atlanticus]\n",
            " 3QM7_A Chain A, Myoglobin [Thunnus atlanticus]\n",
            " 3QM8_A Chain A, Myoglobin [Thunnus atlanticus]\n",
            " 3QM9_A Chain A, Myoglobin [Thunnus atlanticus]\n",
            " 3QMA_A Chain A, Myoglobin [Thunnus atlanticus]\n",
            "Length=146\n",
            "\n",
            " Score = 55.8 bits (133),  Expect = 4e-10, Method: Compositional matrix adjust.\n",
            " Identities = 37/112 (33%), Positives = 50/112 (45%), Gaps = 3/112 (3%)\n",
            "\n",
            "Query  11   AVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHGKKV  68\n",
            "            AV   WG V  D   +GG  L RL   +P TQ+ F  F  ++  D + GN  V AHG  V\n",
            "Sbjct  6    AVLKCWGPVEADYTTIGGLVLTRLFKEHPETQKLFPKFAGIAQAD-IAGNAAVSAHGATV  64\n",
            "\n",
            "Query  69   LGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG  120\n",
            "            L    + L    +       L+  H  K  +   NF+L+  VLV V+    G\n",
            "Sbjct  65   LKKLGELLKAKGSHAAILKPLANSHATKHKIPINNFKLISEVLVKVMQEKAG  116\n",
            "\n",
            "\n",
            ">2NRM_A Chain A, Myoglobin [Thunnus atlanticus]\n",
            "Length=147\n",
            "\n",
            " Score = 55.8 bits (133),  Expect = 5e-10, Method: Compositional matrix adjust.\n",
            " Identities = 37/112 (33%), Positives = 50/112 (45%), Gaps = 3/112 (3%)\n",
            "\n",
            "Query  11   AVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHGKKV  68\n",
            "            AV   WG V  D   +GG  L RL   +P TQ+ F  F  ++  D + GN  V AHG  V\n",
            "Sbjct  6    AVLKXWGPVEADYTTIGGLVLTRLFKEHPETQKLFPKFAGIAQAD-IAGNAAVSAHGATV  64\n",
            "\n",
            "Query  69   LGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG  120\n",
            "            L    + L    +       L+  H  K  +   NF+L+  VLV V+    G\n",
            "Sbjct  65   LKKLGELLKAKGSHAAILKPLANSHATKHKIPINNFKLISEVLVKVMQEKAG  116\n",
            "\n",
            "\n",
            ">2NRL_A Chain A, Myoglobin [Thunnus atlanticus]\n",
            " 2NX0_A Chain A, Myoglobin [Thunnus atlanticus]\n",
            "Length=147\n",
            "\n",
            " Score = 55.8 bits (133),  Expect = 5e-10, Method: Compositional matrix adjust.\n",
            " Identities = 37/112 (33%), Positives = 50/112 (45%), Gaps = 3/112 (3%)\n",
            "\n",
            "Query  11   AVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHGKKV  68\n",
            "            AV   WG V  D   +GG  L RL   +P TQ+ F  F  ++  D + GN  V AHG  V\n",
            "Sbjct  6    AVLKCWGPVEADYTTIGGLVLTRLFKEHPETQKLFPKFAGIAQAD-IAGNAAVSAHGATV  64\n",
            "\n",
            "Query  69   LGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG  120\n",
            "            L    + L    +       L+  H  K  +   NF+L+  VLV V+    G\n",
            "Sbjct  65   LKKLGELLKAKGSHAAILKPLANSHATKHKIPINNFKLISEVLVKVMQEKAG  116\n",
            "\n",
            "\n",
            ">7DDR_A Chain A, Ancestral myoglobin aMbSp [synthetic construct]\n",
            " 7DDS_A Chain A, Ancestral myoglobin aMbSp [synthetic construct]\n",
            "Length=154\n",
            "\n",
            " Score = 55.5 bits (132),  Expect = 7e-10, Method: Compositional matrix adjust.\n",
            " Identities = 41/145 (28%), Positives = 62/145 (43%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +WGKV  D  G   E L RL   +P T   F+ F  L + D + G+  +\n",
            "Sbjct  3    LSDGEWQLVLNVWGKVEADLAGHGQEVLIRLFKGHPETLEKFDKFKHLKSEDEMKGSEDL  62\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  VL A    L      +     L++ H  K  +  +    + + ++ VL      \n",
            "Sbjct  63   KKHGNTVLTALGGILKKKGQHEAELKPLAQSHATKHKIPVKYLEFISDAIIQVLQSKHPG  122\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F    QAA +K +    N +A KY\n",
            "Sbjct  123  DFGADAQAAMKKALELFRNDIAAKY  147\n",
            "\n",
            "\n",
            ">7DDU_A Chain A, Myoglobin [Mirounga angustirostris]\n",
            " 7DDU_C Chain C, Myoglobin [Mirounga angustirostris]\n",
            "Length=154\n",
            "\n",
            " Score = 53.9 bits (128),  Expect = 2e-09, Method: Compositional matrix adjust.\n",
            " Identities = 40/145 (28%), Positives = 62/145 (43%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +WGKV  D  G   E L RL   +P T   F+ F  L + D +  +  +\n",
            "Sbjct  3    LSDGEWHLVLNIWGKVETDLAGHGQEVLIRLFRSHPETLEKFDKFKHLKSEDDMRRSEDL  62\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            + HG  VL A    L    + +     L++ H  K  +  +    +   ++ VL +    \n",
            "Sbjct  63   RKHGNTVLTALGGILKKKGHHEAELKPLAQSHATKHKIPIKYLEFISEAIIQVLHNKHPG  122\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EF    QAA +K +    N +A KY\n",
            "Sbjct  123  EFGADTQAAMKKALELFRNDIATKY  147\n",
            "\n",
            "\n",
            ">1MYT_A Chain A, MYOGLOBIN [Thunnus albacares]\n",
            "Length=146\n",
            "\n",
            " Score = 53.9 bits (128),  Expect = 3e-09, Method: Compositional matrix adjust.\n",
            " Identities = 35/107 (33%), Positives = 49/107 (46%), Gaps = 3/107 (3%)\n",
            "\n",
            "Query  11   AVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKAHGKKV  68\n",
            "            AV   WG V  D   +GG  L RL   +P TQ+ F  F  ++  D + GN  + AHG  V\n",
            "Sbjct  5    AVLKCWGPVEADYTTMGGLVLTRLFKEHPETQKLFPKFAGIAQAD-IAGNAAISAHGATV  63\n",
            "\n",
            "Query  69   LGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVL  115\n",
            "            L    + L    +       L+  H  K  +   NF+L+  VLV V+\n",
            "Sbjct  64   LKKLGELLKAKGSHAAILKPLANSHATKHKIPINNFKLISEVLVKVM  110\n",
            "\n",
            "\n",
            ">1HLM_A Chain A, HEMOGLOBIN (CYANO MET) [Molpadia arenicola]\n",
            "Length=159\n",
            "\n",
            " Score = 53.9 bits (128),  Expect = 3e-09, Method: Compositional matrix adjust.\n",
            " Identities = 43/147 (29%), Positives = 64/147 (44%), Gaps = 6/147 (4%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            LTP EK  + + W ++     G   +   R+    P  QR F     LS P  +  + ++\n",
            "Sbjct  13   LTPAEKDLIRSTWDQLMTHRTGFVADVFIRIFHNDPTAQRKFPQMAGLS-PAELRTSRQM  71\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDN--LKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHF  119\n",
            "             AH  +V    +  +  +D   L    ATL+  H DK HV  +N+ L G VL+  +    \n",
            "Sbjct  72   HAHAIRVSALMTTYIDEMDTEVLPELLATLTRTH-DKNHVGKKNYDLFGKVLMEAIKAEL  130\n",
            "\n",
            "Query  120  GKEFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            G  FT  V  A+ K  A V   L  K+\n",
            "Sbjct  131  GVGFTKQVHDAWAKTFAIVQGVLITKH  157\n",
            "\n",
            "\n",
            ">1MYI_A Chain A, MYOGLOBIN [Sus scrofa]\n",
            " 1MYI_B Chain B, MYOGLOBIN [Sus scrofa]\n",
            "Length=153\n",
            "\n",
            " Score = 52.8 bits (125),  Expect = 8e-09, Method: Compositional matrix adjust.\n",
            " Identities = 40/145 (28%), Positives = 60/145 (41%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +WGKV  D  G   E L RL   +P T   F+SF  L + D +  +  +\n",
            "Sbjct  2    LSDGEWQLVLNVWGKVEADVAGHGQEVLIRLFKGHPETLEKFDSFKHLKSEDEMKASEDL  61\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  VL A    L    + +     L++ H  K  +  +    +   ++ VL      \n",
            "Sbjct  62   KKHGNTVLTALGGILKKKGHHEAELTPLAQSHATKHKIPVKYLEFISEAIIQVLQSKHPG  121\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F    Q A  K +    N +A KY\n",
            "Sbjct  122  DFGADAQGAMSKALELFRNDMAAKY  146\n",
            "\n",
            "\n",
            ">1MBS_A Chain A, MYOGLOBIN [Phoca vitulina]\n",
            "Length=153\n",
            "\n",
            " Score = 52.4 bits (124),  Expect = 9e-09, Method: Compositional matrix adjust.\n",
            " Identities = 40/145 (28%), Positives = 61/145 (42%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +WGKV  D  G   E L RL   +P T   F+ F  L + D +  +  +\n",
            "Sbjct  2    LSDGEWHLVLNVWGKVETDLAGHGQEVLIRLFKSHPETLEKFDKFKHLKSEDDMRRSEDL  61\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            + HG  VL A    L    + +     L++ H  K  +  +    +   ++ VL      \n",
            "Sbjct  62   RKHGNTVLTALGGILKKKGHHEAELKPLAQSHATKHKIPIKYLEFISEAIIHVLHSKHPA  121\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            EF    QAA +K +    N +A KY\n",
            "Sbjct  122  EFGADAQAAMKKALELFRNDIAAKY  146\n",
            "\n",
            "\n",
            ">7DDT_A Chain A, Ancestral myoglobin aMbSe [synthetic construct]\n",
            "Length=154\n",
            "\n",
            " Score = 52.4 bits (124),  Expect = 1e-08, Method: Compositional matrix adjust.\n",
            " Identities = 40/145 (28%), Positives = 61/145 (42%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +WGKV  D  G   E L RL   +P T   F+ F  L + D +  +  +\n",
            "Sbjct  3    LSDGEWQLVLNVWGKVEADLAGHGQEVLIRLFKGHPETLEKFDKFKHLKSEDEMKRSEDL  62\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  VL A    L    + +     L++ H  K  +  +    +   ++ VL      \n",
            "Sbjct  63   KKHGNTVLTALGGILKKKGHHEAELKPLAQSHATKHKIPIKYLEFISEAIIQVLQSKHPG  122\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F    QAA +K +    N +A KY\n",
            "Sbjct  123  DFGADTQAAMKKALELFRNDIAAKY  147\n",
            "\n",
            "\n",
            ">7DGJ_A Chain A, Myoglobin [Equus caballus]\n",
            " 7DGJ_B Chain B, Myoglobin [Equus caballus]\n",
            " 7DGK_A Chain A, Myoglobin [Equus caballus]\n",
            " 7DGK_B Chain B, Myoglobin [Equus caballus]\n",
            " 7DGL_A Chain A, Myoglobin [Equus caballus]\n",
            " 7DGL_B Chain B, Myoglobin [Equus caballus]\n",
            "Length=153\n",
            "\n",
            " Score = 52.0 bits (123),  Expect = 1e-08, Method: Compositional matrix adjust.\n",
            " Identities = 40/145 (28%), Positives = 60/145 (41%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +WGKV  D  G   E L RL   +P T   F+ F  L T   +  +  +\n",
            "Sbjct  2    LSDGEWQQVLNVWGKVEADIAGHGQEVLIRLFTGHPETLEKFDKFKHLKTEAEMKASEDL  61\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  VL A    L H  + +     L++ H  K  +  +    + + ++ VL      \n",
            "Sbjct  62   KKHGTVVLTALGGILKHKAHAEAELKPLAQSHATKHKIPIKYLEFISDAIIHVLHSKHPG  121\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F    Q A  K +    N +A KY\n",
            "Sbjct  122  DFGADAQGAMTKALELFRNDIAAKY  146\n",
            "\n",
            "\n",
            ">8U5A_A Chain A, Designed myoglobin [synthetic construct]\n",
            " 8U5A_B Chain B, Designed myoglobin [synthetic construct]\n",
            "Length=170\n",
            "\n",
            " Score = 52.4 bits (124),  Expect = 2e-08, Method: Compositional matrix adjust.\n",
            " Identities = 41/131 (31%), Positives = 65/131 (50%), Gaps = 8/131 (6%)\n",
            "\n",
            "Query  7    EEKSA-VTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKVKA  63\n",
            "            EEK+A V AL+ +V  D  E+G   L R    +P T + F  F +L    +   +  +K \n",
            "Sbjct  5    EEKAALVLALFDRVEADREEIGAAVLRRTFEEHPETLKKFPRFLELYKKGSPELDALLKE  64\n",
            "\n",
            "Query  64   HGKKVLGAFSDGLAHL----DNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHF  119\n",
            "            HGK VL A  + +A L    ++ +     L++ H ++  +  E+ R +   L+ VLA  F\n",
            "Sbjct  65   HGKTVLDALIE-IARLRYSGEDYRSLIKELAKSHKEEHKIPIEDLRHIAEALLAVLAERF  123\n",
            "\n",
            "Query  120  GKEFTPPVQAA  130\n",
            "              EF P  +AA\n",
            "Sbjct  124  PDEFGPEARAA  134\n",
            "\n",
            "\n",
            ">4NS2_A Chain A, Myoglobin [Equus caballus]\n",
            " 4TWU_A Chain A, Myoglobin [Equus caballus]\n",
            "Length=154\n",
            "\n",
            " Score = 50.4 bits (119),  Expect = 6e-08, Method: Compositional matrix adjust.\n",
            " Identities = 40/145 (28%), Positives = 60/145 (41%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +WGKV  D  G   E L RL   +P T   F+ F  L T   +  + K+\n",
            "Sbjct  3    LSDGEWQQVLNVWGKVEADIAGHGQEVLIRLFTGHPETLEKFKKFKHLKTEAEMKASEKL  62\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  VL A    L    + +     L++ H  K  +  +    + + ++ VL      \n",
            "Sbjct  63   KKHGTVVLTALGGILKKKGHHEAKLKPLAQSHATKHKIPIKYLEFISDAIIHVLHSKHPG  122\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F    Q A  K +    N +A KY\n",
            "Sbjct  123  DFGADAQGAMTKALELFRNDIAAKY  147\n",
            "\n",
            "\n",
            ">3RJN_B Chain B, Myoglobin [Equus caballus]\n",
            "Length=153\n",
            "\n",
            " Score = 50.1 bits (118),  Expect = 7e-08, Method: Compositional matrix adjust.\n",
            " Identities = 40/145 (28%), Positives = 60/145 (41%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +WGKV  D  G   E L RL   +P T   F+ F  L T   +  + K+\n",
            "Sbjct  2    LSDGEWQQVLNVWGKVEADIAGHGQEVLIRLFTGHPETLEKFKKFKHLKTEAEMKASEKL  61\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  VL A    L    + +     L++ H  K  +  +    + + ++ VL      \n",
            "Sbjct  62   KKHGTVVLTALGGILKKKGHHEAELKPLAQSHATKHKIPIKYLEFISDAIIHVLHSKHPG  121\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F    Q A  K +    N +A KY\n",
            "Sbjct  122  DFGADAQGAMTKALELFRNDIAAKY  146\n",
            "\n",
            "\n",
            ">1YHU_A Chain A, hemoglobin A1 chain [Riftia pachyptila]\n",
            " 1YHU_E Chain E, hemoglobin A1 chain [Riftia pachyptila]\n",
            " 1YHU_I Chain I, hemoglobin A1 chain [Riftia pachyptila]\n",
            " 1YHU_M Chain M, hemoglobin A1 chain [Riftia pachyptila]\n",
            " 1YHU_Q Chain Q, hemoglobin A1 chain [Riftia pachyptila]\n",
            " 1YHU_U Chain U, hemoglobin A1 chain [Riftia pachyptila]\n",
            "Length=145\n",
            "\n",
            " Score = 48.1 bits (113),  Expect = 4e-07, Method: Compositional matrix adjust.\n",
            " Identities = 38/117 (32%), Positives = 56/117 (48%), Gaps = 9/117 (8%)\n",
            "\n",
            "Query  26   GEALGRLLVVY-PWTQRFFESFGDLSTPDAVMGNPKVKAHGKKVLGAFSDGLAHLDNLKG  84\n",
            "            G+AL R +  Y P  +  FES   +++ D  M +P+ KAH  +VLG     ++ LDN   \n",
            "Sbjct  28   GDALWRNVFNYAPNARDIFES---VNSKD--MASPEFKAHIARVLGGLDRVISMLDNQAT  82\n",
            "\n",
            "Query  85   TFATLSELHC--DKLHVDPENFRLLGNVLVCVLAHHFGKEFTPPV-QAAYQKVVAGV  138\n",
            "              A L+ L    D   +DP NF +    L+  +A  FG  F  P  Q  Y  +  G+\n",
            "Sbjct  83   LDADLAHLKSQHDPRTIDPVNFVVFRKALIATVAGTFGVCFDVPAWQGCYNIIAKGI  139\n",
            "\n",
            "\n",
            ">6H6I_A Chain A, Neuroglobin [Mus musculus]\n",
            "Length=151\n",
            "\n",
            " Score = 46.2 bits (108),  Expect = 2e-06, Method: Compositional matrix adjust.\n",
            " Identities = 32/148 (22%), Positives = 63/148 (43%), Gaps = 4/148 (3%)\n",
            "\n",
            "Query  3    HLTPEEKSAVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPK  60\n",
            "            H+   E   +   W  V+    E G     RL  + P     F+  G  S+P+  + +P+\n",
            "Sbjct  1    HMERPESELIRQSWRVVSRSPLEHGTVLFARLFALEPSLLPLFQGGGQFSSPEDSLSSPE  60\n",
            "\n",
            "Query  61   VKAHGKKVLGAFSDGLAHLDNLKGTFATLSEL--HCDKLHVDPENFRLLGNVLVCVLAHH  118\n",
            "               H +KV+      + ++++L      L+ L      + V   +F  +G  L+ +L   \n",
            "Sbjct  61   FLDHIRKVMLVIDAAVTNVEDLSSLEEYLTSLGRKHRAVGVRLSSFSTVGESLLYMLEKS  120\n",
            "\n",
            "Query  119  FGKEFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "             G +FTP  + A+ ++   V  A++  +\n",
            "Sbjct  121  LGPDFTPATRTAWSRLYGAVVQAMSRGW  148\n",
            "\n",
            "\n",
            ">6H6J_A Chain A, Neuroglobin [Mus musculus]\n",
            "Length=150\n",
            "\n",
            " Score = 46.2 bits (108),  Expect = 2e-06, Method: Compositional matrix adjust.\n",
            " Identities = 32/148 (22%), Positives = 63/148 (43%), Gaps = 4/148 (3%)\n",
            "\n",
            "Query  3    HLTPEEKSAVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPK  60\n",
            "            H+   E   +   W  V+    E G     RL  + P     F+  G  S+P+  + +P+\n",
            "Sbjct  1    HMERPESELIRQSWRVVSRSPLEHGTVLFARLFALEPSLLPLFQGGGQFSSPEDSLSSPE  60\n",
            "\n",
            "Query  61   VKAHGKKVLGAFSDGLAHLDNLKGTFATLSEL--HCDKLHVDPENFRLLGNVLVCVLAHH  118\n",
            "               H +KV+      + ++++L      L+ L      + V   +F  +G  L+ +L   \n",
            "Sbjct  61   FLDHIRKVMLVIDAAVTNVEDLSSLEEYLTSLGRKHRAVGVRLSSFSTVGESLLYMLEKS  120\n",
            "\n",
            "Query  119  FGKEFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "             G +FTP  + A+ ++   V  A++  +\n",
            "Sbjct  121  LGPDFTPATRTAWSRLYGAVVQAMSRGW  148\n",
            "\n",
            "\n",
            ">5YCI_A Chain A, Ancestral myoglobin aMbWb' of Basilosaurus relative \n",
            "(polyphyly) [Physeter catodon]\n",
            " 5YCI_B Chain B, Ancestral myoglobin aMbWb' of Basilosaurus relative \n",
            "(polyphyly) [Physeter catodon]\n",
            " 5YCJ_A Chain A, Ancestral myoglobin aMbWb' of Basilosaurus relative \n",
            "(polyphyly) [Physeter catodon]\n",
            " 5YCJ_B Chain B, Ancestral myoglobin aMbWb' of Basilosaurus relative \n",
            "(polyphyly) [Physeter catodon]\n",
            "Length=157\n",
            "\n",
            " Score = 45.8 bits (107),  Expect = 3e-06, Method: Compositional matrix adjust.\n",
            " Identities = 37/145 (26%), Positives = 58/145 (40%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEV--GGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +WGKV  D    G + L RL   +P T   F+ F  L T   +  +  +\n",
            "Sbjct  6    LSDGEWQLVLNIWGKVEADVAGHGQDVLIRLFKGHPETLEKFDKFKHLKTEAEMKASEDL  65\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  VL A    L    + +     L++ H  K  +  +    + + ++ VL      \n",
            "Sbjct  66   KKHGNTVLTALGGILKKKGHHEAELKPLAQSHATKHKIPIKYLEFISDAIIHVLHSRHPG  125\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F    Q A  K +      +A KY\n",
            "Sbjct  126  DFGADAQGAMNKALELFRKDIAAKY  150\n",
            "\n",
            "\n",
            ">4OOD_A Chain A, Myoglobin [Physeter catodon]\n",
            "Length=154\n",
            "\n",
            " Score = 44.3 bits (103),  Expect = 1e-05, Method: Compositional matrix adjust.\n",
            " Identities = 36/145 (25%), Positives = 57/145 (39%), Gaps = 2/145 (1%)\n",
            "\n",
            "Query  4    LTPEEKSAVTALWGKVNVDEVG--GEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPKV  61\n",
            "            L+  E   V  +W KV  D  G   + L RL   +P T  +F+ F  L T   +  +  +\n",
            "Sbjct  3    LSEGEWQLVLHVWAKVEADVAGHGQDILIRLFKSHPETLEYFDRFKHLKTEAEMKASEDL  62\n",
            "\n",
            "Query  62   KAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFGK  121\n",
            "            K HG  VL A    L    + +     L++ H  K  +  +    +   ++ VL      \n",
            "Sbjct  63   KKHGVTVLTALGAILKKKGHHEAELKPLAQSHATKHKIPIKYLEFISEAIIHVLHSRHPG  122\n",
            "\n",
            "Query  122  EFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "            +F    Q A  K +      +A KY\n",
            "Sbjct  123  DFGADAQGAMNKALELFRKDIAAKY  147\n",
            "\n",
            "\n",
            ">6RA6_A Chain A, Neuroglobin,Murine neuroglobin mutant,Neuroglobin [Mus \n",
            "musculus]\n",
            "Length=148\n",
            "\n",
            " Score = 42.7 bits (99),  Expect = 3e-05, Method: Compositional matrix adjust.\n",
            " Identities = 31/148 (21%), Positives = 62/148 (42%), Gaps = 4/148 (3%)\n",
            "\n",
            "Query  3    HLTPEEKSAVTALWGKVNVD--EVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPK  60\n",
            "            H+   E   +   W  V+    E G     RL  + P     F+  G  S+P+  + +P+\n",
            "Sbjct  1    HMERPESELIRQSWRVVSRSPLEHGTVLFARLFALEPSLLPLFQGGGQFSSPEDSLSSPE  60\n",
            "\n",
            "Query  61   VKAHGKKVLGAFSDGLAHLDNLKGTFATLSEL--HCDKLHVDPENFRLLGNVLVCVLAHH  118\n",
            "               H +KV+      + ++++L      L+ L      + V   +   +G  L+ +L   \n",
            "Sbjct  61   FLDHIRKVMLVIDAAVTNVEDLSSLEEYLTSLGRKHRAVGVRLSSASTVGESLLYMLEKS  120\n",
            "\n",
            "Query  119  FGKEFTPPVQAAYQKVVAGVANALAHKY  146\n",
            "             G +FTP  + A+ ++   V  A++  +\n",
            "Sbjct  121  LGPDFTPATRTAWSRLYGAVVQAMSRGW  148\n",
            "\n",
            "\n",
            "\n",
            "Lambda      K        H        a         alpha\n",
            "   0.320    0.137    0.422    0.792     4.96 \n",
            "\n",
            "Gapped\n",
            "Lambda      K        H        a         alpha    sigma\n",
            "   0.267   0.0410    0.140     1.90     42.6     43.6 \n",
            "\n",
            "Effective search space used: 1458313700\n",
            "\n",
            "\n",
            "  Database: PDB protein database\n",
            "    Posted date:  May 9, 2024  3:49 AM\n",
            "  Number of letters in database: 44,659,502\n",
            "  Number of sequences in database:  159,724\n",
            "\n",
            "\n",
            "\n",
            "Matrix: BLOSUM62\n",
            "Gap Penalties: Existence: 11, Extension: 1\n",
            "Neighboring words threshold: 11\n",
            "Window for multiple hits: 40\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#MSA"
      ],
      "metadata": {
        "id": "2-vFZhGsIEQA"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "MAFTT\n",
        "\n"
      ],
      "metadata": {
        "id": "sTktGvlLfPEQ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "\n",
        "# Download the BLAST+ software package\n",
        "os.system('wget https://mafft.cbrc.jp/alignment/software/mafft-7.525-with-extensions-src.tgz')\n",
        "\n",
        "# Extract the downloaded tar.gz file\n",
        "os.system('tar xvzf /content/mafft-7.525-with-extensions-src.tgz')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "btcCzUl4IDYR",
        "outputId": "28906dd5-71ba-4d38-976e-fb46e5bc6210"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0"
            ]
          },
          "metadata": {},
          "execution_count": 1
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%cd /content/mafft-7.525-with-extensions/core"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YRXn3xI0YTfC",
        "outputId": "07f0d2a8-2de2-431d-8272-6704b18910f8"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/mafft-7.525-with-extensions/core\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!make clean\n",
        "!make\n",
        "#!su\n",
        "!make install"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "wnndeAKJIQ87",
        "outputId": "743bb769-bdaf-43ba-c209-504bd87f2556"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "rm -f *.o *.a *.exe *~ mafftash_premafft.pl seekquencer_premafft.pl dvtditr dndfast7 dndblast sextet5 mafft-distance pairlocalalign multi2hat3s pairash addsingle maffttext2hex hex2maffttext splittbfast disttbfast tbfast nodepair mafft-profile f2cl mccaskillwrap contrafoldwrap countlen seq2regtable regtable2seq score getlag dndpre setcore filter replaceu restoreu setdirection makedirectionlist version  mafft mafft-homologs.rb mafft-sparsecore.rb libdisttbfast.so libdisttbfast.dylib libdisttbfast.dll *.gcda *.gcno \n",
            "cp mafftash_premafft.tmpl mafftash_premafft.pl\n",
            "cp seekquencer_premafft.tmpl seekquencer_premafft.pl\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c mtxutl.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c io.c\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KPreRead\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:1120:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1120 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K; *locnjob = atoi( b );\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:1125:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1125 |                 \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:1129:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1129 |                         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K; nleni = atoi( b );\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KFRead\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:1165:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1165 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:1255:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1255 |                 \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K; nlen[i] = atoi( b );      /* seq i no nagasa */\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadhat2_doublehalf_part_pointer\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:2708:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2708 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2710:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2710 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K; nseq0 = atoi( b );  // 2020/Oct/23\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2716:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2716 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadhat2_doublehalf_pointer\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:2746:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2746 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2747:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2747 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K; b[5] = 0; nseq0 = atoi( b ); if( nseq != nseq0 )\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2752:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2752 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadhat2_doublehalf\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:2780:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2780 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2781:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2781 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K; b[5] = 0; nseq0 = atoi( b ); if( nseq != nseq0 ) ErrorExit( \"hat2 is wrong.\" );\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2782:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2782 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadhat2_double\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:2810:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2810 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2811:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2811 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K; b[5] = 0; nseq0 = atoi( b ); if( nseq != nseq0 ) ErrorExit( \"hat2 is wrong.\" );\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2812:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2812 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadhat2_int\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:2840:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2840 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2841:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2841 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K; b[5] = 0; nseq0 = atoi( b ); if( nseq != nseq0 ) ErrorExit( \"hat2 is wrong.\" );\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2842:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2842 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadhat2_pointer\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:2871:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2871 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2872:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2872 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K; b[5] = 0; nseq0 = atoi( b ); if( nseq != nseq0 ) ErrorExit( \"hat2 is wrong.\" );\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2873:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2873 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadhat2\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:2901:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2901 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2902:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2902 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K; b[5] = 0; nseq0 = atoi( b ); if( nseq != nseq0 ) ErrorExit( \"hat2 is wrong.\" );\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:2903:5:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2903 |     \u001b[01;35m\u001b[Kfgets( b, B, fp )\u001b[m\u001b[K;\n",
            "      |     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta_sub\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3158:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3158 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:3173:13:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3173 |             \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |             \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadSsearch\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3192:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3192 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadBlastm7_avscore\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3320:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3320 |                 \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:3327:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3327 |                         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:3328:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3328 |                         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadBlastm7_scoreonly\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3443:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3443 |                 \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:3450:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3450 |                         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:3451:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3451 |                         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadBlastm7\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3561:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3561 |                 \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:3568:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3568 |                         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:3569:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3569 |                         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta34noalign\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3593:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3593 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta34m10_nuc\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3630:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3630 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:3656:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3656 |                         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta34m10\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3769:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3769 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:3793:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3793 |                         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta34m10_scoreonly_nucbk\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3896:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3896 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta34m10_scoreonly_nuc\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:3946:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3946 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta34m10_scoreonly\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:4007:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4007 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta34\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:4063:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4063 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:4089:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4089 |                 \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K; // initn:\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:4092:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4092 |                 \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K; // Smith-Waterman score\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:4104:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4104 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta3\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:4126:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4126 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadFasta\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:4151:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4151 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadOpt\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:4188:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4188 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KReadOpt2\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:4209:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4209 |         \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[KreadOtherOptions\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:4305:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4305 |                 \u001b[01;35m\u001b[Kfgets( b, B-1, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kloadaamtx\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:5564:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5564 |                 \u001b[01;35m\u001b[Kfgets( inorder, 999, mf )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:5596:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5596 |                 \u001b[01;35m\u001b[Kfgets( line, 999, mf )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:5619:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5619 |                 \u001b[01;35m\u001b[Kfgets( line, 999, mf )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadsubalignmentstable\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:5775:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5775 |                         \u001b[01;35m\u001b[Kfgets( line, linelen-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:5796:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5796 |                         \u001b[01;35m\u001b[Kfgets( line, linelen-1, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadmccaskill\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:5850:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5850 |         \u001b[01;35m\u001b[Kfgets( gett, 999, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:5860:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5860 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadpairfoldalign\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:5919:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5919 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:5932:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5932 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:5939:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5939 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:5953:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5953 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:5959:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 5959 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kio.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadexternalanchors\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kio.c:6397:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6397 |                 \u001b[01;35m\u001b[Kfgets( buf, 9999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c mltaln9.c\n",
            "\u001b[01m\u001b[Kmltaln9.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcheck_guidetreefile\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kmltaln9.c:2850:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 2850 |         \u001b[01;35m\u001b[Kfgets( string, sizeof(string) / sizeof(char), fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmltaln9.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadlocalhomtable2_single_bin_noseek\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kmltaln9.c:15688:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "15688 |         \u001b[01;35m\u001b[Kfread( &size, sizeof( int ), 1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmltaln9.c:15689:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "15689 |         \u001b[01;35m\u001b[Kfread( &opt, sizeof( double ), 1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmltaln9.c:15692:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "15692 |         \u001b[01;35m\u001b[Kfread( m, size*sizeof( int ), 3, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmltaln9.c:15728:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfread\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "15728 |         \u001b[01;35m\u001b[Kfread( &c, sizeof( char ), 1, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c tddis.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c constants.c\n",
            "In file included from \u001b[01m\u001b[Kconstants.c:7\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Kblosum.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Koverridematrix\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kblosum.c:372:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  372 |                 \u001b[01;35m\u001b[Kfgets( buf, 499, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c nj.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c partSalignmm.c  \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c Lalignmm.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c rna.c \n",
            "\u001b[01m\u001b[Krna.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Krnaalifoldcall\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Krna.c:225:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  225 |         \u001b[01;35m\u001b[Ksystem( cmd )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Krna.c:245:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  245 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Krna.c:285:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  285 |         \u001b[01;35m\u001b[Ksystem( cmd )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c Salignmm.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c Dalignmm.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c Falign.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c Falign_localhom.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c Galign11.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c Lalign11.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c genalign11.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c MSalignmm.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c fftFunctions.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c fft.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c tditeration.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c dvtditr.c\n",
            "\u001b[01m\u001b[Kdvtditr.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kdvtditr.c:1020:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1020 |                                 \u001b[01;35m\u001b[Ksystem( \"cp infile.tree GuideTree\" )\u001b[m\u001b[K; // tekitou\n",
            "      |                                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c defs.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c SAalignmm.c -o SAalignmm.o\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c treeOperation.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c addfunctions.c\n",
            "gcc -o dvtditr mtxutl.o io.o mltaln9.o tddis.o constants.o nj.o partSalignmm.o Lalignmm.o rna.o Salignmm.o Dalignmm.o Falign.o Falign_localhom.o Galign11.o Lalign11.o genalign11.o MSalignmm.o fftFunctions.o fft.o tditeration.o dvtditr.o defs.o SAalignmm.o treeOperation.o addfunctions.o   -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c dndfast7.c\n",
            "\u001b[01m\u001b[Kdndfast7.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kdndfast7.c:326:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  326 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o dndfast7 dndfast7.o io.o constants.o mtxutl.o mltaln9.o tddis.o defs.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc -O3   -c -o dndblast.o dndblast.c\n",
            "\u001b[01m\u001b[Kdndblast.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kdndblast.c:201:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  201 |                         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kdndblast.c:382:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  382 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o dndblast dndblast.o io.o constants.o mtxutl.o mltaln9.o tddis.o defs.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c sextet5.c\n",
            "gcc -o sextet5 io.o constants.o mtxutl.o mltaln9.o tddis.o sextet5.o defs.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c mafft-distance.c\n",
            "gcc -o mafft-distance io.o constants.o mtxutl.o mltaln9.o tddis.o mafft-distance.o defs.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c MSalign11.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c suboptalign11.c \n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c pairlocalalignmain.c\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c pairlocalalign.c\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcalllast_once\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1056:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1056 |                 \u001b[01;35m\u001b[Ksystem( command )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1092:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1092 |                 \u001b[01;35m\u001b[Ksystem( command )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadlastresx\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:608:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  608 |                 \u001b[01;35m\u001b[Kfgets( gett, 9999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcalllast_fast\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:981:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  981 |                 \u001b[01;35m\u001b[Ksystem( command )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Klastcallthread\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:872:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  872 |                                 \u001b[01;35m\u001b[Ksystem( command )\u001b[m\u001b[K;\n",
            "      |                                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadlastresx_singleq\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:374:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  374 |                 \u001b[01;35m\u001b[Kfgets( gett, 9999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadhat4\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1532:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1532 |         \u001b[01;35m\u001b[Kfgets( oneline, 999, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1544:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1544 |                 \u001b[01;35m\u001b[Kfgets( oneline, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Krecalllara\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1245:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1245 |         \u001b[01;35m\u001b[Kfgets( com, 999, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcallmxscarna_giving_bpp\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1408:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1408 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1410:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1410 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1430:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1430 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K; // for cygwin, wakaran\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1451:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1451 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K; // for cygwin, wakaran\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1465:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1465 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K; // for cygwin, wakaran\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1485:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1485 |         \u001b[01;35m\u001b[Kfgets( com, 999, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1487:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1487 |         \u001b[01;35m\u001b[Kfgets( com, 999, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcalldafs_giving_bpp\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1291:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1291 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1293:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1293 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1313:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1313 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K; // for cygwin, wakaran\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1334:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1334 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K; // for cygwin, wakaran\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1344:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1344 |         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K; // for cygwin, wakaran\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1364:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1364 |         \u001b[01;35m\u001b[Kfgets( com, 999, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1366:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1366 |         \u001b[01;35m\u001b[Kfgets( com, 999, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:1368:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1368 |         \u001b[01;35m\u001b[Kfgets( com, 999, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kpairalign_node\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:3468:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3468 |         \u001b[01;35m\u001b[Ksystem( \"rm -rf hat3dir\" )\u001b[m\u001b[K; // toriaezu\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:3469:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3469 |         \u001b[01;35m\u001b[Ksystem( \"mkdir hat3dir\" )\u001b[m\u001b[K; // toriaezu\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairlocalalign.c:3474:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3474 |                 \u001b[01;35m\u001b[Ksystem( fn )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o pairlocalalign mtxutl.o io.o mltaln9.o tddis.o constants.o partSalignmm.o Lalignmm.o rna.o Salignmm.o Dalignmm.o Falign.o MSalignmm.o Galign11.o Lalign11.o genalign11.o MSalign11.o suboptalign11.o SAalignmm.o pairlocalalignmain.o pairlocalalign.o defs.o fft.o fftFunctions.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c multi2hat3s.c\n",
            "gcc -o multi2hat3s mtxutl.o io.o mltaln9.o tddis.o constants.o multi2hat3s.o defs.o fft.o fftFunctions.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c pairash.c\n",
            "\u001b[01m\u001b[Kpairash.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kprepareash\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairash.c:639:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  639 |                 \u001b[01;35m\u001b[Kfgets( line, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:681:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  681 |                 \u001b[01;35m\u001b[Kfgets( line, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcallrash\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairash.c:485:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  485 |                 \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadrash\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairash.c:277:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  277 |                 \u001b[01;35m\u001b[Kfgets( line, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:284:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  284 |                         \u001b[01;35m\u001b[Kfgets( line, 999, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:289:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  289 |                         \u001b[01;35m\u001b[Kfgets( line, 999, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:294:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  294 |                         \u001b[01;35m\u001b[Kfgets( line, 999, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kpreparetmalign\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairash.c:530:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  530 |                 \u001b[01;35m\u001b[Kfgets( line, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:576:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  576 |                 \u001b[01;35m\u001b[Kfgets( line, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:587:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  587 |                         \u001b[01;35m\u001b[Ksystem( command )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcheckcbeta\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairash.c:321:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  321 |                 \u001b[01;35m\u001b[Kfgets( linec, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcalltmalign\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairash.c:347:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  347 |                 \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:356:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  356 |                         \u001b[01;35m\u001b[Kfgets( com, 100, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadtmalign\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairash.c:228:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  228 |                 \u001b[01;35m\u001b[Kfgets( line, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:232:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  232 |         \u001b[01;35m\u001b[Kfgets( line, nlenmax*2, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:235:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  235 |         \u001b[01;35m\u001b[Kfgets( line, nlenmax*2, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:238:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  238 |         \u001b[01;35m\u001b[Kfgets( line, nlenmax*2, fp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcallrash.constprop\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairash.c:428:43:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K.\u001b[m\u001b[K’ directive writing 1 byte into a region of size between 0 and 9999 [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  428 |                 sprintf( cachefile, \"%s/%s\u001b[01;35m\u001b[K.\u001b[m\u001b[K%s.rash\", cachedir, fname1, fname2 );\n",
            "      |                                           \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "In file included from \u001b[01m\u001b[K/usr/include/stdio.h:894\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kmltaln.h:12\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kpairash.c:1\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:10:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[K‘\u001b[01m\u001b[K__builtin___sprintf_chk\u001b[m\u001b[K’ output 8 or more bytes (assuming 10007) into a destination of size 10000\n",
            "   38 |   return \u001b[01;36m\u001b[K__builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,\u001b[m\u001b[K\n",
            "      |          \u001b[01;36m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   39 | \u001b[01;36m\u001b[K                                  __glibc_objsize (__s), __fmt,\u001b[m\u001b[K\n",
            "      |                                   \u001b[01;36m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   40 | \u001b[01;36m\u001b[K                                  __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                                   \u001b[01;36m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:484:36:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive writing up to 9999 bytes into a region of size 9996 [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  484 |                 sprintf( com, \"cat \u001b[01;35m\u001b[K%s\u001b[m\u001b[K > %s.rashout\", \u001b[32m\u001b[Kcachefile\u001b[m\u001b[K, pairid );\n",
            "      |                                    \u001b[01;35m\u001b[K^~\u001b[m\u001b[K                \u001b[32m\u001b[K~~~~~~~~~\u001b[m\u001b[K\n",
            "In file included from \u001b[01m\u001b[K/usr/include/stdio.h:894\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kmltaln.h:12\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kpairash.c:1\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:10:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[K‘\u001b[01m\u001b[K__builtin___sprintf_chk\u001b[m\u001b[K’ output between 16 and 11014 bytes into a destination of size 10000\n",
            "   38 |   return \u001b[01;36m\u001b[K__builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,\u001b[m\u001b[K\n",
            "      |          \u001b[01;36m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   39 | \u001b[01;36m\u001b[K                                  __glibc_objsize (__s), __fmt,\u001b[m\u001b[K\n",
            "      |                                   \u001b[01;36m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   40 | \u001b[01;36m\u001b[K                                  __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                                   \u001b[01;36m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kcalltmalign.constprop\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kpairash.c:346:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive writing up to 9999 bytes into a region of size 9991 [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  346 |                 sprintf( com, \"mkdir -p \u001b[01;35m\u001b[K%s\u001b[m\u001b[K\", \u001b[32m\u001b[Kcachedir\u001b[m\u001b[K );\n",
            "      |                                         \u001b[01;35m\u001b[K^~\u001b[m\u001b[K   \u001b[32m\u001b[K~~~~~~~~\u001b[m\u001b[K\n",
            "In file included from \u001b[01m\u001b[K/usr/include/stdio.h:894\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kmltaln.h:12\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kpairash.c:1\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:10:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[K‘\u001b[01m\u001b[K__builtin___sprintf_chk\u001b[m\u001b[K’ output between 10 and 10009 bytes into a destination of size 10000\n",
            "   38 |   return \u001b[01;36m\u001b[K__builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,\u001b[m\u001b[K\n",
            "      |          \u001b[01;36m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   39 | \u001b[01;36m\u001b[K                                  __glibc_objsize (__s), __fmt,\u001b[m\u001b[K\n",
            "      |                                   \u001b[01;36m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   40 | \u001b[01;36m\u001b[K                                  __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                                   \u001b[01;36m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kpairash.c:349:45:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K-\u001b[m\u001b[K’ directive writing 1 byte into a region of size between 0 and 9999 [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  349 |                 sprintf( cachefile, \"%s/%s%s\u001b[01;35m\u001b[K-\u001b[m\u001b[K%s%s\", cachedir, fname1, chain1, fname2, chain2 );\n",
            "      |                                             \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "In file included from \u001b[01m\u001b[K/usr/include/stdio.h:894\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kmltaln.h:12\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kpairash.c:1\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:10:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[K‘\u001b[01m\u001b[K__builtin___sprintf_chk\u001b[m\u001b[K’ output 3 or more bytes (assuming 10002) into a destination of size 10000\n",
            "   38 |   return \u001b[01;36m\u001b[K__builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,\u001b[m\u001b[K\n",
            "      |          \u001b[01;36m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   39 | \u001b[01;36m\u001b[K                                  __glibc_objsize (__s), __fmt,\u001b[m\u001b[K\n",
            "      |                                   \u001b[01;36m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   40 | \u001b[01;36m\u001b[K                                  __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                                   \u001b[01;36m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o pairash mtxutl.o io.o mltaln9.o tddis.o constants.o partSalignmm.o Lalignmm.o rna.o Salignmm.o Dalignmm.o Falign.o MSalignmm.o Galign11.o Lalign11.o genalign11.o MSalign11.o suboptalign11.o SAalignmm.o pairash.o defs.o fft.o fftFunctions.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c addsingle.c\n",
            "gcc -o addsingle mtxutl.o io.o mltaln9.o tddis.o constants.o MSalignmm.o partSalignmm.o Lalignmm.o rna.o Salignmm.o Dalignmm.o Falign.o Falign_localhom.o Galign11.o Lalign11.o genalign11.o SAalignmm.o addsingle.o defs.o fft.o fftFunctions.o addfunctions.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc -o maffttext2hex maffttext2hex.c  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "\u001b[01m\u001b[Kmaffttext2hex.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kmaffttext2hex.c:55:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   55 |                         \u001b[01;35m\u001b[Kfgets( (char *)buf, 999, fp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o hex2maffttext hex2maffttext.c  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c splittbfast.c\n",
            "\u001b[01m\u001b[Ksplittbfast.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksplittbfast.c:3236:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3236 |                 \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o splittbfast mtxutl.o io.o mltaln9.o tddis.o constants.o partSalignmm.o Lalignmm.o rna.o Salignmm.o  Dalignmm.o Falign.o Falign_localhom.o Galign11.o Lalign11.o genalign11.o SAalignmm.o MSalignmm.o splittbfast.o defs.o fft.o fftFunctions.o   -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c disttbfast.c\n",
            "\u001b[01m\u001b[Kdisttbfast.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kdisttbfast\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kdisttbfast.c:4387:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 4387 |                                         \u001b[01;35m\u001b[Ksystem( \"cp infile.tree GuideTree\" )\u001b[m\u001b[K; // tekitou\n",
            "      |                                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o disttbfast mtxutl.o io.o mltaln9.o tddis.o constants.o partSalignmm.o Lalignmm.o rna.o Salignmm.o Dalignmm.o Falign.o Falign_localhom.o Galign11.o Lalign11.o genalign11.o SAalignmm.o MSalignmm.o disttbfast.o defs.o fft.o fftFunctions.o addfunctions.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c tbfast.c\n",
            "\u001b[01m\u001b[Ktbfast.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ktbfast.c:3260:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 3260 |                                 \u001b[01;35m\u001b[Ksystem( \"cp infile.tree GuideTree\" )\u001b[m\u001b[K; // tekitou\n",
            "      |                                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o tbfast mtxutl.o io.o mltaln9.o tddis.o constants.o MSalignmm.o partSalignmm.o Lalignmm.o rna.o Salignmm.o Dalignmm.o Falign.o Falign_localhom.o Galign11.o Lalign11.o genalign11.o SAalignmm.o tbfast.o defs.o fft.o fftFunctions.o addfunctions.o pairlocalalign.o MSalign11.o   -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c nodepair.c\n",
            "gcc -o nodepair mtxutl.o io.o mltaln9.o tddis.o constants.o MSalignmm.o partSalignmm.o Lalignmm.o rna.o Salignmm.o Dalignmm.o Falign.o Falign_localhom.o Galign11.o Lalign11.o genalign11.o SAalignmm.o nodepair.o defs.o fft.o fftFunctions.o addfunctions.o pairlocalalign.o MSalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c mafft-profile.c\n",
            "gcc -o mafft-profile io.o mtxutl.o mltaln9.o tddis.o constants.o partSalignmm.o MSalignmm.o Lalignmm.o rna.o Salignmm.o Dalignmm.o SAalignmm.o Galign11.o Lalign11.o genalign11.o Falign.o Falign_localhom.o fftFunctions.o fft.o mafft-profile.o defs.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c f2cl.c\n",
            "\u001b[01m\u001b[Kf2cl.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kf2cl.c:297:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  297 |                         \u001b[01;35m\u001b[Kfgets( gett, B-1, orderfp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o f2cl mtxutl.o io.o f2cl.o constants.o defs.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c mccaskillwrap.c\n",
            "\u001b[01m\u001b[Kmccaskillwrap.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kmccaskillwrap.c:372:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  372 |                         \u001b[01;35m\u001b[Ksystem( \"tr -d '\\\\r' < _mccaskillinorg > _mccaskillin\" )\u001b[m\u001b[K; // for cygwin, wakaran\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmccaskillwrap.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kathread\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kmccaskillwrap.c:115:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  115 |                 \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmccaskillwrap.c:117:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  117 |                 \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmccaskillwrap.c:129:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  129 |                 \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K; // for cygwin, wakaran\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmccaskillwrap.c:155:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksystem\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  155 |                         \u001b[01;35m\u001b[Ksystem( com )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmccaskillwrap.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadrawmccaskill\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kmccaskillwrap.c:46:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   46 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o mccaskillwrap mtxutl.o io.o mccaskillwrap.o constants.o defs.o mltaln9.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c contrafoldwrap.c\n",
            "\u001b[01m\u001b[Kcontrafoldwrap.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadcontrafold\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kcontrafoldwrap.c:57:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   57 |                 \u001b[01;35m\u001b[Kfgets( gett, 9999, fp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o contrafoldwrap mtxutl.o io.o contrafoldwrap.o constants.o defs.o mltaln9.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c countlen.c\n",
            "gcc -o countlen mtxutl.o io.o countlen.o defs.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c seq2regtable.c\n",
            "gcc -o seq2regtable mtxutl.o io.o seq2regtable.o defs.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c regtable2seq.c\n",
            "\u001b[01m\u001b[Kregtable2seq.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadereg\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kregtable2seq.c:69:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   69 |         \u001b[01;35m\u001b[Kfgets( gett, 999, regfp )\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kregtable2seq.c:97:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   97 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, regfp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kregtable2seq.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kreadreg\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kregtable2seq.c:120:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  120 |                 \u001b[01;35m\u001b[Kfgets( gett, 999, regfp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o regtable2seq mtxutl.o io.o regtable2seq.o defs.o mltaln9.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c score.c\n",
            "gcc -o score io.o mtxutl.o mltaln9.o score.o constants.o defs.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c getlag.c\n",
            "gcc -o getlag mtxutl.o io.o mltaln9.o tddis.o constants.o partSalignmm.o Lalignmm.o rna.o Salignmm.o Falign.o Falign_localhom.o Galign11.o Lalign11.o genalign11.o SAalignmm.o MSalignmm.o Dalignmm.o getlag.o defs.o fft.o fftFunctions.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c dndpre.c\n",
            "gcc -o dndpre dndpre.o io.o constants.o mtxutl.o mltaln9.o defs.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c setcore.c\n",
            "gcc -o setcore mtxutl.o io.o mltaln9.o tddis.o constants.o partSalignmm.o Lalignmm.o rna.o Salignmm.o Dalignmm.o Falign.o Falign_localhom.o Galign11.o Lalign11.o genalign11.o SAalignmm.o MSalignmm.o setcore.o defs.o fft.o fftFunctions.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc -O3   -c -o filter.o filter.c\n",
            "gcc -o filter mtxutl.o io.o filter.o defs.o mltaln9.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c replaceu.c\n",
            "gcc -o replaceu mtxutl.o io.o replaceu.o defs.o mltaln9.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c restoreu.c\n",
            "\u001b[01m\u001b[Krestoreu.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Krestoreu.c:185:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  185 |                         \u001b[01;35m\u001b[Kfgets( gett, 999, dlfp )\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o restoreu mtxutl.o io.o restoreu.o defs.o mltaln9.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c setdirection.c\n",
            "\u001b[01m\u001b[Ksetdirection.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmain\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksetdirection.c:132:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kfgets\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  132 |                 \u001b[01;35m\u001b[Kfgets( line, 99, difp )\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -o setdirection mtxutl.o io.o setdirection.o defs.o mltaln9.o Galign11.o Lalign11.o genalign11.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc  -Denablemultithread  -std=c99 -O3 -c makedirectionlist.c\n",
            "gcc -o makedirectionlist mtxutl.o io.o mltaln9.o tddis.o constants.o partSalignmm.o Lalignmm.o rna.o Salignmm.o  Dalignmm.o Falign.o Falign_localhom.o Galign11.o Lalign11.o genalign11.o SAalignmm.o MSalignmm.o makedirectionlist.o defs.o fft.o fftFunctions.o addfunctions.o  -Denablemultithread  -std=c99 -O3  -lm  -lpthread\n",
            "gcc -o version version.c  -Denablemultithread  -std=c99 -O3  -lm  -lpthread \n",
            "sed \"s:_LIBDIR:/usr/local/libexec/mafft:\" mafft.tmpl  > mafft\n",
            "sed \"s:_BINDIR:/usr/local/bin:\" mafft-homologs.tmpl  > mafft-homologs.rb\n",
            "sed \"s:_BINDIR:/usr/local/bin:\" mafft-sparsecore.tmpl  > mafft-sparsecore.rb\n",
            "cp mafft mafft-homologs.rb mafft-sparsecore.rb ../scripts\n",
            "chmod 755 ../scripts/*\n",
            "cp mafftash_premafft.pl seekquencer_premafft.pl dvtditr dndfast7 dndblast sextet5 mafft-distance pairlocalalign multi2hat3s pairash addsingle maffttext2hex hex2maffttext splittbfast disttbfast tbfast nodepair mafft-profile f2cl mccaskillwrap contrafoldwrap countlen seq2regtable regtable2seq score getlag dndpre setcore filter replaceu restoreu setdirection makedirectionlist version  ../binaries\n",
            "chmod 755 ../binaries/*\n",
            "done.\n",
            "cp mafft mafft-homologs.rb mafft-sparsecore.rb ../scripts\n",
            "chmod 755 ../scripts/*\n",
            "cp mafftash_premafft.pl seekquencer_premafft.pl dvtditr dndfast7 dndblast sextet5 mafft-distance pairlocalalign multi2hat3s pairash addsingle maffttext2hex hex2maffttext splittbfast disttbfast tbfast nodepair mafft-profile f2cl mccaskillwrap contrafoldwrap countlen seq2regtable regtable2seq score getlag dndpre setcore filter replaceu restoreu setdirection makedirectionlist version  ../binaries\n",
            "chmod 755 ../binaries/*\n",
            "done.\n",
            "mkdir -p /usr/local/libexec/mafft\n",
            "chmod 755 /usr/local/libexec/mafft\n",
            "mkdir -p /usr/local/bin\n",
            "chmod 755 /usr/local/bin\n",
            "chmod 755 mafft mafft-homologs.rb mafft-sparsecore.rb\n",
            "install mafft mafft-homologs.rb mafft-sparsecore.rb  /usr/local/bin\n",
            "chmod 755 dvtditr dndfast7 dndblast sextet5 mafft-distance pairlocalalign multi2hat3s pairash addsingle maffttext2hex hex2maffttext splittbfast disttbfast tbfast nodepair mafft-profile f2cl mccaskillwrap contrafoldwrap countlen seq2regtable regtable2seq score getlag dndpre setcore filter replaceu restoreu setdirection makedirectionlist version  ||:     # in MinGW, it's ok if this fails\n",
            "strip dvtditr dndfast7 dndblast sextet5 mafft-distance pairlocalalign multi2hat3s pairash addsingle maffttext2hex hex2maffttext splittbfast disttbfast tbfast nodepair mafft-profile f2cl mccaskillwrap contrafoldwrap countlen seq2regtable regtable2seq score getlag dndpre setcore filter replaceu restoreu setdirection makedirectionlist version  ||: # may fail for dash_client on mac.\n",
            "install dvtditr dndfast7 dndblast sextet5 mafft-distance pairlocalalign multi2hat3s pairash addsingle maffttext2hex hex2maffttext splittbfast disttbfast tbfast nodepair mafft-profile f2cl mccaskillwrap contrafoldwrap countlen seq2regtable regtable2seq score getlag dndpre setcore filter replaceu restoreu setdirection makedirectionlist version  /usr/local/libexec/mafft\n",
            "install mafftash_premafft.pl seekquencer_premafft.pl /usr/local/libexec/mafft\n",
            "( cd /usr/local/bin; \\\n",
            "rm -f linsi ginsi einsi fftns fftnsi nwns nwnsi xinsi qinsi; \\\n",
            "rm -f mafft-linsi mafft-ginsi mafft-einsi mafft-fftns mafft-fftnsi mafft-nwns mafft-nwnsi mafft-xinsi mafft-qinsi mafft-randomcore.rb ; \\\n",
            "ln -s mafft linsi; ln -s mafft ginsi; ln -s mafft fftns; \\\n",
            "ln -s mafft fftnsi; ln -s mafft nwns; ln -s mafft nwnsi; \\\n",
            "ln -s mafft einsi; \\\n",
            "ln -s mafft mafft-linsi; ln -s mafft mafft-ginsi; ln -s mafft mafft-fftns; \\\n",
            "ln -s mafft mafft-fftnsi; ln -s mafft mafft-nwns; ln -s mafft mafft-nwnsi; \\\n",
            "ln -s mafft mafft-einsi; ln -s mafft mafft-xinsi; ln -s mafft mafft-qinsi;\\\n",
            "rm -f mafft-profile mafft-profile.exe; ln -s /usr/local/libexec/mafft/mafft-profile .; \\\n",
            "rm -f mafft-distance mafft-distance.exe; ln -s /usr/local/libexec/mafft/mafft-distance . )\n",
            "mkdir -p /usr/local/share/man/man1\n",
            "chmod 755 /usr/local/share/man/man1\n",
            "install -m 644 mafft.1 mafft-homologs.1 /usr/local/share/man/man1\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!mafft -h\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "NRDFt1umY2Ju",
        "outputId": "98032750-c2e2-457a-c42c-90aa7b7792bf"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/bin/mafft: Cannot open -h.\n",
            "\n",
            "------------------------------------------------------------------------------\n",
            "  MAFFT v7.525 (2024/Mar/13)\n",
            "  https://mafft.cbrc.jp/alignment/software/\n",
            "  MBE 30:772-780 (2013), NAR 30:3059-3066 (2002)\n",
            "------------------------------------------------------------------------------\n",
            "High speed:\n",
            "  % mafft in > out\n",
            "  % mafft --retree 1 in > out (fast)\n",
            "\n",
            "High accuracy (for <~200 sequences x <~2,000 aa/nt):\n",
            "  % mafft --maxiterate 1000 --localpair  in > out (% linsi in > out is also ok)\n",
            "  % mafft --maxiterate 1000 --genafpair  in > out (% einsi in > out)\n",
            "  % mafft --maxiterate 1000 --globalpair in > out (% ginsi in > out)\n",
            "\n",
            "If unsure which option to use:\n",
            "  % mafft --auto in > out\n",
            "\n",
            "--op # :         Gap opening penalty, default: 1.53\n",
            "--ep # :         Offset (works like gap extension penalty), default: 0.0\n",
            "--maxiterate # : Maximum number of iterative refinement, default: 0\n",
            "--clustalout :   Output: clustal format, default: fasta\n",
            "--reorder :      Outorder: aligned, default: input order\n",
            "--quiet :        Do not report progress\n",
            "--thread # :     Number of threads (if unsure, --thread -1)\n",
            "--dash :         Add structural information (Rozewicki et al, submitted)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!wget https://github.com/etemadism/Courses/raw/main/Bioinformatics/Alignment/MSA_hemoglobin.fasta"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-YRGgWDTY7Be",
        "outputId": "f3a1d69b-7d14-4439-b410-6c5d1b937c11"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-04-29 20:05:49--  https://github.com/etemadism/Courses/raw/main/Bioinformatics/Alignment/MSA_hemoglobin.fasta\n",
            "Resolving github.com (github.com)... 140.82.116.4\n",
            "Connecting to github.com (github.com)|140.82.116.4|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://raw.githubusercontent.com/etemadism/Courses/main/Bioinformatics/Alignment/MSA_hemoglobin.fasta [following]\n",
            "--2024-04-29 20:05:49--  https://raw.githubusercontent.com/etemadism/Courses/main/Bioinformatics/Alignment/MSA_hemoglobin.fasta\n",
            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.111.133, 185.199.110.133, 185.199.108.133, ...\n",
            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 1024 (1.0K) [text/plain]\n",
            "Saving to: ‘MSA_hemoglobin.fasta’\n",
            "\n",
            "MSA_hemoglobin.fast 100%[===================>]   1.00K  --.-KB/s    in 0s      \n",
            "\n",
            "2024-04-29 20:05:50 (70.5 MB/s) - ‘MSA_hemoglobin.fasta’ saved [1024/1024]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!mafft --auto /content/mafft-7.525-with-extensions/core/MSA_hemoglobin.fasta"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "KfT0J8y2YkkI",
        "outputId": "2b604a6a-4fba-4fed-f839-fd92ea94c9ca"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "outputhat23=16\n",
            "treein = 0\n",
            "compacttree = 0\n",
            "stacksize: 8192 kb\n",
            "rescale = 1\n",
            "All-to-all alignment.\n",
            "    0 / 5\r    1 / 5\r    2 / 5\r    3 / 5\rtbfast-pair (aa) Version 7.525\n",
            "alg=L, model=BLOSUM62, 2.00, -0.10, +0.10, noshift, amax=0.0\n",
            "0 thread(s)\n",
            "\n",
            "outputhat23=16\n",
            "Loading 'hat3.seed' ... \n",
            "done.\n",
            "Writing hat3 for iterative refinement\n",
            "rescale = 1\n",
            "Gap Penalty = -1.53, +0.00, +0.00\n",
            "tbutree = 1, compacttree = 0\n",
            "Constructing a UPGMA tree ... \n",
            "\r    0 / 5\n",
            "done.\n",
            "\n",
            "Progressive alignment ... \n",
            "\rSTEP     1 /4 \rSTEP     2 /4 \rSTEP     3 /4 \rSTEP     4 /4 \n",
            "done.\n",
            "tbfast (aa) Version 7.525\n",
            "alg=A, model=BLOSUM62, 1.53, -0.00, -0.00, noshift, amax=0.0\n",
            "1 thread(s)\n",
            "\n",
            "minimumweight = 0.000010\n",
            "autosubalignment = 0.000000\n",
            "nthread = 0\n",
            "randomseed = 0\n",
            "blosum 62 / kimura 200\n",
            "poffset = 0\n",
            "niter = 16\n",
            "sueff_global = 0.100000\n",
            "nadd = 16\n",
            "Loading 'hat3' ... done.\n",
            "rescale = 1\n",
            "\n",
            "\r    0 / 5\n",
            "Segment   1/  1    1- 148\n",
            "STEP 001-001-0  identical.   \rSTEP 001-001-1  identical.   \rSTEP 001-002-0  identical.   \rSTEP 001-002-1  identical.   \rSTEP 001-003-0  identical.   \rSTEP 001-003-1  identical.   \rSTEP 001-004-1  identical.   \rSTEP 002-004-1  identical.   \rSTEP 002-003-0  identical.   \rSTEP 002-003-1  identical.   \r\n",
            "Converged.\n",
            "\n",
            "done\n",
            "dvtditr (aa) Version 7.525\n",
            "alg=A, model=BLOSUM62, 1.53, -0.00, -0.00, noshift, amax=0.0\n",
            "0 thread(s)\n",
            "\n",
            "\n",
            "Strategy:\n",
            " L-INS-i (Probably most accurate, very slow)\n",
            " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
            "\n",
            "If unsure which option to use, try 'mafft --auto input > output'.\n",
            "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
            "\n",
            "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
            "It tends to insert more gaps into gap-rich regions than previous versions.\n",
            "To disable this change, add the --leavegappyregion option.\n",
            "\n",
            ">NP_990820.1 hemoglobin subunit beta [Gallus gallus]\n",
            "MVHWTAEEKQLITGLWGKVNVAECGAEALARLLIVYPWTQ----------------GNPM\n",
            "VRAHGKKVLTSFGDAVKNLDNIKNTFSQLSELHCDKLHVDPENFRLLGDILIIVLAAHFS\n",
            "KDFTPECQAAWQKLVRVVAHALARKYH\n",
            ">NP_058652.1 hemoglobin subunit beta-2 [Mus musculus]\n",
            "MVHLTDAEKSAVSCLWAKVNPDEVGGEALGRLLVV----------------ASAIMGNPK\n",
            "VKAHGKKVITAFNEGLKNLDNLKGTFASLSELHCDKLHVDPENFRLLGNAIVIVLGHHLG\n",
            "KDFTPAAQAAFQKVVAGVATALAHKYH\n",
            ">NP_001257813.1 hemoglobin subunit beta-like [Canis lupus familiaris]\n",
            "MVHLTAEEKSLVSGLWGK-----------GRLLIVYPWTQRFFDSFGDLSTPDAVMSNAK\n",
            "VKAHGKKVLNSFSDGLKNLDNLKGTFAKLSELHCDKLHVDPENFKLLGNVLVCVLAHHFG\n",
            "KEFTPQVQAAYQKVVAGVANALAHKYH\n",
            ">XP_508242.1 hemoglobin subunit beta [Pan troglodytes]\n",
            "MVHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPK\n",
            "VKAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG\n",
            "KEFTPPVQAAYQKVVAGVANALAHKYH\n",
            ">NP_000509.1 hemoglobin subunit beta [Homo sapiens]\n",
            "MVHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGN--\n",
            "-----------FSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG\n",
            "KEFTPPVQAAYQKVVAGVANALAHKYH\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#Project"
      ],
      "metadata": {
        "id": "hCo33-9YjmJD"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "Consistency-Based approaches:\n",
        "ProbCons and T-COFFEE"
      ],
      "metadata": {
        "id": "vYqrzn0pkBrU"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "Install t-cofee here."
      ],
      "metadata": {
        "id": "bVOr3TmOjtaQ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "https://tcoffee.org/Projects/tcoffee/index.html#DOWNLOAD"
      ],
      "metadata": {
        "id": "18AzpJFZjrTv"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "Structure-Based Methods\n",
        "\n",
        "PRALINE and the T-COFFEE module Expresso"
      ],
      "metadata": {
        "id": "oTgsJ_8Uke9s"
      }
    }
  ]
}